# Publications [ ](# )

 * [Key Publications](#key_publications)
 * [Methods](#methods)
 * [Others](#others)

## Key Publications

*The Complete Genome Sequence of Haloferax volcanii DS2, a Model Archaeon*<br/>
Hartman, Amber L.; Norais, CÃ©dric; Badger, Jonathan H.; Delmas, StÃ©phane; Haldenby, Sam; Madupu, Ramana; Robinson, Jeffrey; Khouri, Hoda; Ren, Qinghu; Lowe, Todd M.; Maupin-Furlow, Julie; Pohlschroder, Mecky; Daniels, Charles; Pfeiffer, Friedhelm; Allers, Thorsten; Eisen, Jonathan A.<br/>
**2010**<br/>
PLOS ONE 5(3)<br/>
10.1371/journal.pone.0009605
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://doi.org/10.1371/journal.pone.0009605)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Key_Publications/Hatman2010PlosOne.ris)<br/>
Background Haloferax volcanii is an easily culturable moderate halophile that grows on simple defined media, is readily transformable, and has a relatively stable genome. This, in combination with its biochemical and genetic tractability, has made Hfx. volcanii a key model organism, not only for the study of halophilicity, but also for archaeal biology in general.  Methodology/Principal Findings We report here the sequencing and analysis of the genome of Hfx. volcanii DS2, the type strain of this species. The genome contains a main 2.848 Mb chromosome, three smaller chromosomes pHV1, 3, 4 (85, 438, 636 kb, respectively) and the pHV2 plasmid (6.4 kb).  Conclusions/Significance The completed genome sequence, presented here, provides an invaluable tool for further in vivo and in vitro studies of Hfx. volcanii.<br/>
</details><br/>
---

[Go to top of page](# )<br/>
----
## Methods

*In vitro Analysis of Ubiquitin-like Protein Modification in Archaea*<br/>
Fu, Xian; Adams, Zachary; Maupin-Furlow, Julie A.<br/>
**2018**<br/>
Bio-protocol 8(10)<br/>
10.21769/BioProtoc.2845
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://doi.org/10.21769/BioProtoc.2845)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Methods/Fu2018Bioprotocol.ris)<br/>
The ubiquitin-like (Ubl) protein is widely distributed in Archaea and involved in many cellular pathways. A well-established method to reconstitute archaeal Ubl protein conjugation in vitro is important to better understand the process of archaeal Ubl protein modification. This protocol describes the in vitro reconstitution of Ubl protein modification and following analysis of this modification in Haloferax volcanii, a halophilic archaeon serving as the model organism.<br/>
</details><br/>
---

*Development of Additional Selectable Markers for the Halophilic Archaeon Haloferax volcanii Based on the leuB and trpA Genes*<br/>
Allers, Thorsten; Ngo, Hien-Ping; Mevarech, Moshe; Lloyd, Robert G.<br/>
**2004**<br/>
Appl. Environ. Microbiol. 70(2)<br/>
10.1128/AEM.70.2.943-953.2004
<details>
<summary>Abstract and Links</summary>

[Link to Publication](http://aem.asm.org/content/70/2/943.abstract)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Methods/Allers2003JBact.ris)<br/>
Since most archaea are extremophilic and difficult to cultivate, our current knowledge of their biology is confined largely to comparative genomics and biochemistry. Haloferax volcanii offers great promise as a model organism for archaeal genetics, but until now there has been a lack of a wide variety of selectable markers for this organism. We describe here isolation of H. volcanii leuB and trpA genes encoding 3-isopropylmalate dehydrogenase and tryptophan synthase, respectively, and development of these genes as a positive selection system. Î”leuB and Î”trpA mutants were constructed in a variety of genetic backgrounds and were shown to be auxotrophic for leucine and tryptophan, respectively. We constructed both integrative and replicative plasmids carrying the leuB or trpA gene under control of a constitutive promoter. The use of these selectable markers in deletion of the lhr gene of H. volcanii is described.<br/>
</details><br/>
---

*Improved shuttle vectors for Haloferax volcanii including a dual-resistance plasmid*<br/>
Holmes, Melissa; Pfeifer, Felicitas; Dyall-Smith, Michael<br/>
**1994**<br/>
Gene 146(1)<br/>
10.1016/0378-1119(94)90844-3
<details>
<summary>Abstract and Links</summary>

[Link to Publication](http://www.sciencedirect.com/science/article/pii/0378111994908443)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Methods/Homes1994Gene.ris)<br/>
Two new Haloferax-Escherichia shuttle vectors are described, pMDS20 and pMLH3. These vectors contain the E. coli ColE1 plasmid ori region and ampicillin-resistance(ApR)-conferring bla gene, and the Haloferax pHK2 replicon region and novobiocin-resistance (NbR)-encoding gyrB gene, enabling maintenance and selection in both hosts. Plasmid pMLH3 has, in addition, a H. volcanii mevinolin-resistance (MvR) determinant and restriction sites allowing insertional inactivation of either marker, to facilitate the identification of Haloferax transformants harbouring cloned sequences. Sequencing of gyrA, within the NbR determinant, and the pHK2 ori region has been completed so the complete sequence of both pMDS20 and pMLH3 is now known.<br/>
</details><br/>
---

[Go to top of page](# )<br/>
----
## Others

*The Primary Antisense Transcriptome of Halobacterium salinarum NRC-1*<br/>
de Almeida, P. JoÃ£o; VÃªncio, Z. N. Ricardo; Lorenzetti, P. R. Alan; ten-Caten, Felipe; Gomes-Filho, V. JosÃ©; Koide, Tie<br/>
**2019**<br/>
Genes 10(4)<br/>
10.3390/genes10040280
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://www.mdpi.com/2073-4425/10/4/280)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/deAlmeida2019Genes.ris)<br/>
Antisense RNAs (asRNAs) are present in diverse organisms and play important roles in gene regulation. In this work, we mapped the primary antisense transcriptome in the halophilic archaeon Halobacterium salinarum NRC-1. By reanalyzing publicly available data, we mapped antisense transcription start sites (aTSSs) and inferred the probable 3&prime; ends of these transcripts. We analyzed the resulting asRNAs according to the size, location, function of genes on the opposite strand, expression levels and conservation. We show that at least 21% of the genes contain asRNAs in H. salinarum. Most of these asRNAs are expressed at low levels. They are located antisense to genes related to distinctive characteristics of H. salinarum, such as bacteriorhodopsin, gas vesicles, transposases and other important biological processes such as translation. We provide evidence to support asRNAs in type II toxin&ndash;antitoxin systems in archaea. We also analyzed public Ribosome profiling (Ribo-seq) data and found that ~10% of the asRNAs are ribosome-associated non-coding RNAs (rancRNAs), with asRNAs from transposases overrepresented. Using a comparative transcriptomics approach, we found that ~19% of the asRNAs annotated in H. salinarum belong to genes with an ortholog in Haloferax volcanii, in which an aTSS could be identified with positional equivalence. This shows that most asRNAs are not conserved between these halophilic archaea.<br/>
</details><br/>
---

*Bacterioruberin extracts from a genetically modified hyperpigmented Haloferax volcanii strain: antioxidant activity and bioactive properties on sperm cells.*<br/>
Zalazar L; Pagola P; Miró MV; Churio MS; Cerletti M; Martínez C; Iniesta-Cuerda M; Soler AJ; Cesari A; De Castro R<br/>
**2019**<br/>
J. Appl. Microbiol. 126(3)<br/>
10.1111/jam.14160
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30472814)<br/>
[Citation]()<br/>
AIMS: To examine the antioxidant activity of Bacterioruberin (Bctr)-rich extracts isolated from a hyperpigmented, genetically modified Haloferax volcanii strain (HVLON3) and to investigate the effect on cold-sensitive ram sperm cells.
METHODS AND RESULTS: The strain HVLON3 produces higher Bctr amounts than most haloarchaea (220 ± 13 mg g
CONCLUSIONS: HVLON3 is an excellent source of natural microbial C
SIGNIFICANCE AND IMPACT OF THE STUDY: This study provides an insight on the bioactive properties of a bioproduct derived from haloarchaea (carotenoids) which are so far underexploited.<br/>
</details><br/>
---

*tRNA Modification Profiles and Codon-Decoding Strategies in Methanocaldococcus jannaschii.*<br/>
Yu N; Jora M; Solivio B; Thakur P; Acevedo-Rocha CG; Randau L; de Crécy-Lagard V; Addepalli B; Limbach PA<br/>
**2019**<br/>
J. Bacteriol. 201(9)<br/>
10.1128/JB.00690-18
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30745370)<br/>
[Citation]()<br/>
tRNAs play a critical role in mRNA decoding, and posttranscriptional modifications within tRNAs drive decoding efficiency and accuracy. The types and positions of tRNA modifications in model bacteria have been extensively studied, and tRNA modifications in a few eukaryotic organisms have also been characterized and localized to particular tRNA sequences. However, far less is known regarding tRNA modifications in archaea. While the identities of modifications have been determined for multiple archaeal organisms, <br/>
</details><br/>
---

*Division plane placement in pleomorphic archaea is dynamically coupled to cell shape.*<br/>
Walsh JC; Angstmann CN; Bisson-Filho AW; Garner EC; Duggin IG; Curmi PMG<br/>
**2019**<br/>
Mol. Microbiol. 112(3)<br/>
10.1111/mmi.14316
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31136034)<br/>
[Citation]()<br/>
One mechanism for achieving accurate placement of the cell division machinery is via Turing patterns, where nonlinear molecular interactions spontaneously produce spatiotemporal concentration gradients. The resulting patterns are dictated by cell shape. For example, the Min system of Escherichia coli shows spatiotemporal oscillation between cell poles, leaving a mid-cell zone for division. The universality of pattern-forming mechanisms in divisome placement is currently unclear. We examined the location of the division plane in two pleomorphic archaea, Haloferax volcanii and Haloarcula japonica, and showed that it correlates with the predictions of Turing patterning. Time-lapse analysis of H. volcanii shows that divisome locations after successive rounds of division are dynamically determined by daughter cell shape. For H. volcanii, we show that the location of DNA does not influence division plane location, ruling out nucleoid occlusion. Triangular cells provide a stringent test for Turing patterning, where there is a bifurcation in division plane orientation. For the two archaea examined, most triangular cells divide as predicted by a Turing mechanism; however, in some cases multiple division planes are observed resulting in cells dividing into three viable progeny. Our results suggest that the division site placement is consistent with a Turing patterning system in these archaea.<br/>
</details><br/>
---

*Pervasive acquisition of CRISPR memory driven by inter-species mating of archaea can limit gene transfer and influence speciation.*<br/>
Turgeman-Grott I; Joseph S; Marton S; Eizenshtein K; Naor A; Soucy SM; Stachler AE; Shalev Y; Zarkor M; Reshef L; Altman-Price N; Marchfelder A; Gophna U<br/>
**2019**<br/>
Nat Microbiol 4(1)<br/>
10.1038/s41564-018-0302-8
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30478289)<br/>
[Citation]()<br/>
CRISPR-Cas systems provide prokaryotes with sequence-specific immunity against viruses and plasmids based on DNA acquired from these invaders, known as spacers. Surprisingly, many archaea possess spacers that match chromosomal genes of related species, including those encoding core housekeeping genes. By sequencing genomes of environmental archaea isolated from a single site, we demonstrate that inter-species spacers are common. We show experimentally, by mating Haloferax volcanii and Haloferax mediterranei, that spacers are indeed acquired chromosome-wide, although a preference for integrated mobile elements and nearby regions of the chromosome exists. Inter-species mating induces increased spacer acquisition and may result in interactions between the acquisition machinery of the two species. Surprisingly, many of the spacers acquired following inter-species mating target self-replicons along with those originating from the mating partner, indicating that the acquisition machinery cannot distinguish self from non-self under these conditions. Engineering the chromosome of one species to be targeted by the other's CRISPR-Cas reduces gene exchange between them substantially. Thus, spacers acquired during inter-species mating could limit future gene transfer, resulting in a role for CRISPR-Cas systems in microbial speciation.<br/>
</details><br/>
---

*Denitrifying haloarchaea within the genus Haloferax display divergent respiratory phenotypes, with implications for their release of nitrogenous gases.*<br/>
Torregrosa-Crespo J; Pire C; Martínez-Espinosa RM; Bergaust L<br/>
**2019**<br/>
Environ. Microbiol. 21(1)<br/>
10.1111/1462-2920.14474
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30421557)<br/>
[Citation]()<br/>
Haloarchaea are extremophiles, generally thriving at high temperatures and salt concentrations, thus, with limited access to oxygen. As a strategy to maintain a respiratory metabolism, many halophilic archaea are capable of denitrification. Among them are members of the genus Haloferax, which are abundant in saline/hypersaline environments. Three reported haloarchaeal denitrifiers, Haloferax mediterranei, Haloferax denitrificans and Haloferax volcanii, were characterized with respect to their denitrification phenotype. A semi-automatic incubation system was used to monitor the depletion of electron acceptors and accumulation of gaseous intermediates in batch cultures under a range of conditions. Out of the species tested, only H. mediterranei was able to consistently reduce all available N-oxyanions to N<br/>
</details><br/>
---

*Homologs of aquifex aeolicus protein-only RNase P are not the major RNase P activities in the archaea haloferax volcanii and methanosarcina mazei.*<br/>
Schwarz TS; Wäber NB; Feyh R; Weidenbach K; Schmitz RA; Marchfelder A; Hartmann RK<br/>
**2019**<br/>
IUBMB Life 71(8)<br/>
10.1002/iub.2122
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31283101)<br/>
[Citation]()<br/>
The mature 5'-ends of tRNAs are generated by RNase P in all domains of life. The ancient form of the enzyme is a ribonucleoprotein consisting of a catalytic RNA and one or more protein subunits. However, in the hyperthermophilic bacterium Aquifex aeolicus and close relatives, RNase P is a protein-only enzyme consisting of a single type of polypeptide (Aq_880, 23 kDa). In many archaea, homologs of Aq_880 were identified (termed HARPs for Homologs of Aquifex RNase P) in addition to the RNA-based RNase P, raising the question about the functions of HARP and the classical RNase P in these archaea. Here we investigated HARPs from two euryarchaeotes, Haloferax volcanii and Methanosarcina mazei. Archaeal strains with HARP gene knockouts showed no growth phenotypes under standard conditions, temperature and salt stress (H. volcanii) or nitrogen deficiency (M. mazei). Recombinant H. volcanii and M. mazei HARPs were basically able to catalyse specific tRNA 5'-end maturation in vitro. Furthermore, M. mazei HARP was able to rescue growth of an Escherichia coli RNase P depletion strain with comparable efficiency as Aq_880, while H. volcanii HARP was unable to do so. In conclusion, both archaeal HARPs showed the capacity (in at least one functional assay) to act as RNases P. However, the ease to obtain knockouts of the singular HARP genes and the lack of growth phenotypes upon HARP gene deletion contrasts with the findings that the canonical RNase P RNA gene cannot be deleted in H. volcanii, and a knockdown of RNase P RNA in H. volcanii results in severe tRNA processing defects. We conclude that archaeal HARPs do not make a major contribution to global tRNA 5'-end maturation in archaea, but may well exert a specialised, yet unknown function in (t)RNA metabolism. © 2019 IUBMB Life, 2019 © 2019 IUBMB Life, 71(8):1109-1116, 2019.<br/>
</details><br/>
---

*CRISPRi as an efficient tool for gene repression in archaea.*<br/>
Stachler AE; Schwarz TS; Schreiber S; Marchfelder A<br/>
**2019**<br/>
Methods None(None)<br/>
10.1016/j.ymeth.2019.05.023
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31150759)<br/>
[Citation]()<br/>
In the years following its discovery and characterization, the CRISPR-Cas system has been modified and converted into a multitude of applications for eukaryotes and bacteria, such as genome editing and gene regulation. Since no such method has been available for archaea, we developed a tool for gene repression in the haloarchaeon Haloferax volcanii by repurposing its endogenous type I-B CRISPR-Cas system. Here, we present the two possible approaches for gene repression as well as our workflow to achieve and assess gene knockdown, offer recommendations on protospacer selection and give some examples of genes we have successfully silenced.<br/>
</details><br/>
---

*Catalytic linkage between caspase activity and proteostasis in Archaea.*<br/>
Seth-Pasricha M; Senn S; Sanman LE; Bogyo M; Nanda V; Bidle KA; Bidle KD<br/>
**2019**<br/>
Environ. Microbiol. 21(1)<br/>
10.1111/1462-2920.14456
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30370585)<br/>
[Citation]()<br/>
The model haloarchaeon, Haloferax volcanii possess an extremely high, and highly specific, basal caspase activity in exponentially growing cells that closely resembles caspase-4. This activity is specifically inhibited by the pan-caspase inhibitor, z-VAD-FMK, and has no cross-reactivity with other known protease families. Although it is one of the dominant cellular proteolytic activities in exponentially growing H. volcanii cells, the interactive cellular roles remain unknown and the protein(s) responsible for this activity remain elusive. Here, biochemical purification and in situ trapping with caspase targeted covalent inhibitors combined with genome-enabled proteomics, structural analysis, targeted gene knockouts and treatment with canavanine demonstrated a catalytic linkage between caspase activity and thermosomes, proteasomes and cdc48b, a cell division protein and proteasomal degradation facilitating ATPase, as part of an 'interactase' of stress-related protein complexes with an established link to the unfolded protein response (UPR). Our findings provide novel cellular and biochemical context for the observed caspase activity in Archaea and add new insight to understanding the role of this activity, implicating their possible role in the establishment of protein stress and ER associated degradation pathways in Eukarya.<br/>
</details><br/>
---

*Environmental factors influence the Haloferax volcanii S-layer protein structure.*<br/>
Rodrigues-Oliveira T; Souza AA; Kruger R; Schuster B; Maria de Freitas S; Kyaw CM<br/>
**2019**<br/>
PLoS ONE 14(5)<br/>
10.1371/journal.pone.0216863
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31075115)<br/>
[Citation]()<br/>
S-layers commonly cover archaeal cell envelopes and are composed of proteins that self-assemble into a paracrystalline surface structure. Despite their detection in almost all archaea, there are few reports investigating the structural properties of these proteins, with no reports exploring this topic for halophilic S-layers. The objective of the present study was to investigate the secondary and tertiary organization of the Haloferax volcanii S-layer protein. Such investigations were performed using circular dichroism, fluorescence spectroscopy, dynamic light scattering and transmission electron microscopy. The protein secondary structure is centered on β-sheets and is affected by environmental pH, with higher disorder in more alkaline conditions. The pH can also affect the protein's tertiary structure, with higher tryptophan side-chain exposure to the medium under the same conditions. The concentrations of Na, Mg and Ca ions in the environment also affect the protein structures, with small changes in α-helix and β-sheet content, as well as changes in tryptophan side chain exposure. These changes in turn influence the protein's functional properties, with cell envelope preparations revealing striking differences when in different salt conditions. Thermal denaturation assays revealed that the protein is stable. It has been reported that the S-layer protein N-glycosylation process is affected by external factors and the present study indicates for the first time changes in the protein structure.<br/>
</details><br/>
---

*l-Rhamnose catabolism in archaea.*<br/>
Reinhardt A; Johnsen U; Schönheit P<br/>
**2019**<br/>
Mol. Microbiol. 111(4)<br/>
10.1111/mmi.14213
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30707467)<br/>
[Citation]()<br/>
The halophilic archaeon Haloferax volcanii utilizes l-rhamnose as a sole carbon and energy source. It is shown that l-rhamnose is taken up by an ABC transporter and is oxidatively degraded to pyruvate and l-lactate via the diketo-hydrolase pathway. The genes involved in l-rhamnose uptake and degradation form a l-rhamnose catabolism (rhc) gene cluster. The rhc cluster also contains a gene, rhcR, that encodes the transcriptional regulator RhcR which was characterized as an activator of all rhc genes. 2-keto-3-deoxy-l-rhamnonate, a metabolic intermediate of l-rhamnose degradation, was identified as inducer molecule of RhcR. The essential function of rhc genes for uptake and degradation of l-rhamnose was proven by the respective knockout mutants. Enzymes of the diketo-hydrolase pathway, including l-rhamnose dehydrogenase, l-rhamnonolactonase, l-rhamnonate dehydratase, 2-keto-3-deoxy-l-rhamnonate dehydrogenase and 2,4-diketo-3-deoxy-l-rhamnonate hydrolase, were characterized. Further, genes of the diketo-hydrolase pathway were also identified in the hyperthermophilic crenarchaeota Vulcanisaeta distributa and Sulfolobus solfataricus and selected enzymes were characterized, indicating the presence of the diketo-hydrolase pathway in these archaea. Together, this is the first comprehensive description of l-rhamnose catabolism in the domain of archaea.<br/>
</details><br/>
---

*Haloferax volcanii.*<br/>
Pohlschroder M; Schulze S<br/>
**2019**<br/>
Trends Microbiol. 27(1)<br/>
10.1016/j.tim.2018.10.004
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30459094)<br/>
[Citation]()<br/>
In this infographic we present the main tools available for the halophilic archaeon Haloferax volcanii, which have enabled successful research on its biology, including its genetics, proteostasis, cell surface structures, metabolic pathways, and adaptation to high salt environments. Isolated from the Dead Sea in 1975, Haloferax volcanii thrives in high salt environments and has emerged as an important archaeal model system. An extensive repertoire of genetic, molecular biological, and biochemical tools has been developed for this fast-growing, easily cultivated haloarchaeon, including expression vectors and gene-deletion strategies, including CRISPR. Its low mutation rate and ability to grow on defined media allow straightforward application of methods such as metabolic labeling, and the sequenced genome laid the foundation for transcriptomics and proteomics studies. These tools have allowed examination of key pathways such as transcription, noncoding RNAs, protein synthesis and degradation, protein glycosylation, motility, and biofilm formation. With the collaborative spirit of the H. volcanii community, this model system has become invaluable not only for enhancing our understanding of archaea but also for improving the development of biotech applications.<br/>
</details><br/>
---

*Several One-Domain Zinc Finger µ-Proteins of Haloferax Volcanii Are Important for Stress Adaptation, Biofilm Formation, and Swarming.*<br/>
Nagel C; Machulla A; Zahn S; Soppa J<br/>
**2019**<br/>
Genes (Basel) 10(5)<br/>
10.3390/genes10050361
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31083437)<br/>
[Citation]()<br/>
Zinc finger domains are highly structured and can mediate interactions to DNA, RNA, proteins, lipids, and small molecules. Accordingly, zinc finger proteins are very versatile and involved in many biological functions. Eukaryotes contain a wealth of zinc finger proteins, but zinc finger proteins have also been found in archaea and bacteria. Large zinc finger proteins have been well studied, however, in stark contrast, single domain zinc finger µ-proteins of less than 70 amino acids have not been studied at all, with one single exception. Therefore, 16 zinc finger µ-proteins of the haloarchaeon <br/>
</details><br/>
---

*It's all about the T: transcription termination in archaea.*<br/>
Maier LK; Marchfelder A<br/>
**2019**<br/>
Biochem. Soc. Trans. 47(1)<br/>
10.1042/BST20180557
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30783016)<br/>
[Citation]()<br/>
One of the most fundamental biological processes driving all life on earth is transcription. The, at first glance, relatively simple cycle is divided into three stages: initiation at the promoter site, elongation throughout the open reading frame, and finally termination and product release at the terminator. In all three processes, motifs of the template DNA and protein factors of the transcription machinery including the multisubunit polymerase itself as well as a broad range of associated transcription factors work together and mutually influence each other. Despite several decades of research, this interplay holds delicate mechanistic and structural details as well as interconnections yet to be explored. One of the surprising characteristics of archaeal biology is the use of eukaryotic-like information processing systems against a backdrop of a bacterial-like genome. Archaeal genomes usually comprise main chromosomes alongside chromosomal plasmids, and the genetic information is encoded in single transcriptional units as well as in multicistronic operons alike their bacterial counterparts. Moreover, archaeal genomes are densely packed and this necessitates a tight regulation of transcription and especially assured termination events in order to prevent read-through into downstream coding regions and the accumulation of antisense transcripts.<br/>
</details><br/>
---

*The nuts and bolts of the Haloferax CRISPR-Cas system I-B.*<br/>
Maier LK; Stachler AE; Brendel J; Stoll B; Fischer S; Haas KA; Schwarz TS; Alkhnbashi OS; Sharma K; Urlaub H; Backofen R; Gophna U; Marchfelder A<br/>
**2019**<br/>
RNA Biol 16(4)<br/>
10.1080/15476286.2018.1460994
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29649958)<br/>
[Citation]()<br/>
Invading genetic elements pose a constant threat to prokaryotic survival, requiring an effective defence. Eleven years ago, the arsenal of known defence mechanisms was expanded by the discovery of the CRISPR-Cas system. Although CRISPR-Cas is present in the majority of archaea, research often focuses on bacterial models. Here, we provide a perspective based on insights gained studying CRISPR-Cas system I-B of the archaeon Haloferax volcanii. The system relies on more than 50 different crRNAs, whose stability and maintenance critically depend on the proteins Cas5 and Cas7, which bind the crRNA and form the Cascade complex. The interference machinery requires a seed sequence and can interact with multiple PAM sequences. H. volcanii stands out as the first example of an organism that can tolerate autoimmunity via the CRISPR-Cas system while maintaining a constitutively active system. In addition, the H. volcanii system was successfully developed into a tool for gene regulation.<br/>
</details><br/>
---

*Limited Cross-Complementation Between Haloferax volcanii PilB1-C1 and PilB3-C3 Paralogs*<br/>
Legerme, Georgio; Pohlschroder, Mechthild<br/>
**2019**<br/>
Frontiers in Microbiology 10()<br/>
10.3389/fmicb.2019.00700
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://www.frontiersin.org/article/10.3389/fmicb.2019.00700)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Legerme2019FrontMicrobiol.ris)<br/>
Type IV pili are evolutionarily conserved cell surface filaments that promote surface adhesion and cell aggregation providing bacteria and archaea protection from a variety of stress conditions. In fact, prokaryotic genomes frequently contain several copies of the core biosynthesis genes, pilB and pilC, encoding an ATPase and membrane anchor, respectively. Recent phylogenetic analyses suggest that in haloarchaea, a subset of pilB-C paralogs, such as the Haloferax volcanii pilB1-C1, were gained via horizontal transfer from the crenarchaea, while the co-regulated type IV pilus subunits, the pilins, evolved by duplication, followed by diversification of the ancestral euryarchaeal pilins. Here, we report the identification of an H. volcanii pilB1 transposon mutant that exhibits an adhesion defect in defined media. A similar defect observed in an H. volcanii âˆ†pilB1-C1 strain can be rescued by expressing pilB1-C1Â in trans. However, these proteins cannot rescue the severe adhesion defect of a previously reported âˆ†pilB3-C3 strain. Conversely, pilB3-C3, which are not predicted to have been laterally transferred, expressed in trans can rescue the adhesion defect of a âˆ†pilB1-C1 strain. This cross-complementation supports the proposed hybrid origin of the operon containing pilB1-C1 and shows that at least certain euryarchaeal PilB paralogs can work with different pilin sets. Efficient recognition of the euryarchaeal pilins by the crenarchaeal PilB1-C1 may have required some degree of pilin modification, but perhaps the modifications were minor enough that PilB3 recognition of these pilins was not precluded, resulting in modular evolution and an extensive combinatorial diversity that allows for adaptation to a variety of stress conditions and attachment to varied surfaces.<br/>
</details><br/>
---

*Polyploidy in halophilic archaea: regulation, evolutionary advantages, and gene conversion.*<br/>
Ludt K; Soppa J<br/>
**2019**<br/>
Biochem. Soc. Trans. 47(3)<br/>
10.1042/BST20190256
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31189733)<br/>
[Citation]()<br/>
All analyzed haloarachea are polyploid. In addition, haloarchaea contain more than one type of chromosome, and thus the gene dosage can be regulated independently on different replicons. Haloarchaea and several additional archaea have more than one replication origin on their major chromosome, in stark contrast with bacteria, which have a single replication origin. Two of these replication origins of <br/>
</details><br/>
---

*Positioning of the Motility Machinery in Halophilic Archaea.*<br/>
Li Z; Kinosita Y; Rodriguez-Franco M; Nußbaum P; Braun F; Delpech F; Quax TEF; Albers SV<br/>
**2019**<br/>
MBio 10(3)<br/>
10.1128/mBio.00377-19
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31064826)<br/>
[Citation]()<br/>
Bacteria and archaea exhibit tactical behavior and can move up and down chemical gradients. This tactical behavior relies on a motility structure, which is guided by a chemosensory system. Environmental signals are sensed by membrane-inserted chemosensory receptors that are organized in large ordered arrays. While the cellular positioning of the chemotaxis machinery and that of the flagellum have been studied in detail in bacteria, we have little knowledge about the localization of such macromolecular assemblies in archaea. Although the archaeal motility structure, the archaellum, is fundamentally different from the flagellum, archaea have received the chemosensory machinery from bacteria and have connected this system with the archaellum. Here, we applied a combination of time-lapse imaging and fluorescence and electron microscopy using the model euryarchaeon <br/>
</details><br/>
---

*Characterization of the transcriptome of Haloferax volcanii, grown under four different conditions, with mixed RNA-Seq.*<br/>
Laass S; Monzon VA; Kliemt J; Hammelmann M; Pfeiffer F; Förstner KU; Soppa J<br/>
**2019**<br/>
PLoS ONE 14(4)<br/>
10.1371/journal.pone.0215986
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31039177)<br/>
[Citation]()<br/>
Haloferax volcanii is a well-established model species for haloarchaea. Small scale RNomics and bioinformatics predictions were used to identify small non-coding RNAs (sRNAs), and deletion mutants revealed that sRNAs have important regulatory functions. A recent dRNA-Seq study was used to characterize the primary transcriptome. Unexpectedly, it was revealed that, under optimal conditions, H. volcanii contains more non-coding sRNAs than protein-encoding mRNAs. However, the dRNA-Seq approach did not contain any length information. Therefore, a mixed RNA-Seq approach was used to determine transcript length and to identify additional transcripts, which are not present under optimal conditions. In total, 50 million paired end reads of 150 nt length were obtained. 1861 protein-coding RNAs (cdRNAs) were detected, which encoded 3092 proteins. This nearly doubled the coverage of cdRNAs, compared to the previous dRNA-Seq study. About 2/3 of the cdRNAs were monocistronic, and 1/3 covered more than one gene. In addition, 1635 non-coding sRNAs were identified. The highest fraction of non-coding RNAs were cis antisense RNAs (asRNAs). Analysis of the length distribution revealed that sRNAs have a median length of about 150 nt. Based on the RNA-Seq and dRNA-Seq results, genes were chosen to exemplify characteristics of the H. volcanii transcriptome by Northern blot analyses, e.g. 1) the transcript patterns of gene clusters can be straightforward, but also very complex, 2) many transcripts differ in expression level under the four analyzed conditions, 3) some genes are transcribed into RNA isoforms of different length, which can be differentially regulated, 4) transcripts with very long 5'-UTRs and with very long 3'-UTRs exist, and 5) about 30% of all cdRNAs have overlapping 3'-ends, which indicates, together with the asRNAs, that H. volcanii makes ample use of sense-antisense interactions. Taken together, this RNA-Seq study, together with a previous dRNA-Seq study, enabled an unprecedented view on the H. volcanii transcriptome.<br/>
</details><br/>
---

*A Haloarchaeal Small Regulatory RNA (sRNA) Is Essential for Rapid Adaptation to Phosphate Starvation Conditions.*<br/>
Kliemt J; Jaschinski K; Soppa J<br/>
**2019**<br/>
Front Microbiol 10(None)<br/>
10.3389/fmicb.2019.01219
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31231327)<br/>
[Citation]()<br/>
The haloarchaeon <br/>
</details><br/>
---

*Solution structure and dynamics of the small protein HVO_2922 from Haloferax volcanii.*<br/>
Kubatova N; Jonker HRA; Saxena K; Richter C; Vogel V; Schreiber S; Marchfelder A; Schwalbe H<br/>
**2019**<br/>
Chembiochem None(None)<br/>
10.1002/cbic.201900085
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31161645)<br/>
[Citation]()<br/>
Past sequencing campaigns overlooked small proteins as they seemed to be irrelevant due to their small size. However, their occurrence is widespread and there is growing evidence that these small proteins are in fact functionally very important in organisms found in all kingdoms of life. Within a global proteome analysis for small proteins of the archaeal model organism Haloferax volcanii, the HVO_2922 protein has been identified. It is differentially expressed in response to changes in iron and salt concentrations, suggesting that its expression is stress-regulated. The protein is conserved among Haloarchaea and contains an uncharacterized domain of unknown function (DUF1508, UPF0339 family protein). We elucidated the NMR solution structure, which shows that the isolated protein forms a symmetrical dimer. The dimerization is found to be concentration-dependent and essential for the protein stability and most likely for its functionality as mutagenesis at the dimer interface leads to a decrease of stability and protein aggregation.<br/>
</details><br/>
---

*Characterisation of a solvent-tolerant haloarchaeal (R)-selective transaminase isolated from a Triassic period salt mine.*<br/>
Kelly SA; Magill DJ; Megaw J; Skvortsov T; Allers T; McGrath JW; Allen CCR; Moody TS; Gilmore BF<br/>
**2019**<br/>
Appl. Microbiol. Biotechnol. 103(14)<br/>
10.1007/s00253-019-09806-y
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31123770)<br/>
[Citation]()<br/>
Transaminase enzymes (TAms) are becoming increasingly valuable in the chemist's toolbox as a biocatalytic route to chiral amines. Despite high profile successes, the lack of (R)-selective TAms and robustness under harsh industrial conditions continue to prove problematic. Herein, we report the isolation of the first haloarchaeal TAm (BC61-TAm) to be characterised for the purposes of pharmaceutical biocatalysis. BC61-TAm is an (R)-selective enzyme, cloned from an extremely halophilic archaeon, isolated from a Triassic period salt mine. Produced using a Haloferax volcanii-based expression model, the resulting protein displays a classic halophilic activity profile, as well as thermotolerance (optimum 50 °C) and organic solvent tolerance. Molecular modelling predicts the putative active site residues of haloarchaeal TAms, with molecular dynamics simulations providing insights on the basis of BC61-TAm's organic solvent tolerance. These results represent an exciting advance in the study of transaminases from extremophiles, providing a possible scaffold for future discovery of biocatalytic enzymes with robust properties.<br/>
</details><br/>
---

*The Response of Haloferax Volcanii to Salt and Temperature Stress: A Proteome Study by Label-Free Mass Spectrometry.*<br/>
Jevtic Z; Stoll B; Pfeiffer F; Sharma K; Urlaub H; Marchfelder A; Lenz C<br/>
**2019**<br/>
Proteomics None(None)<br/>
10.1002/pmic.201800491
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31502396)<br/>
[Citation]()<br/>
We performed an in-depth proteome analysis of the haloarchaeal model organism Haloferax volcanii under standard, low/high salt and low/high temperature conditions using label-free mass spectrometry. Qualitative analysis of protein identification data from high-pH/reversed phase fractionated samples indicated 61.1% proteome coverage (2,509 proteins), which is close to the maximum recorded values in archaea. Identified proteins matched to the predicted proteome in their physicochemical properties, with only a small bias against low-molecular-weight and membrane-associated proteins. Cells grown under low and high salt stress as well as low and high temperature stress were quantitatively compared to standard cultures by SWATH mass spectrometry. 2,244 proteins, or 54.7% of the predicted proteome, were quantified across all conditions at high reproducibility, which allowed for global analysis of protein expression changes under these stresses. Of these, 2,034 were significantly regulated under at least one stress condition. KEGG pathway enrichment analysis showed that several major cellular pathways are part of H. volcanii's universal stress response. In addition, specific pathways (purine, cobalamin, tryptophan) were affected by temperature stress. The most strongly down-regulated proteins under all stress conditions, zinc finger protein HVO_2753 and ribosomal protein S14, were found oppositely regulated to their immediate genetic neighbors from the same operon. This article is protected by copyright. All rights reserved.<br/>
</details><br/>
---

*Uptake of D-xylose and L-arabinose in Haloferax volcanii involves an ABC transporter of the CUT1 subfamily.*<br/>
Johnsen U; Ortjohann M; Sutter JM; Geweke S; Schönheit P<br/>
**2019**<br/>
FEMS Microbiol. Lett. 366(8)<br/>
femsle/fnz089
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31089701)<br/>
[Citation]()<br/>
Haloferax volcanii degrades D-xylose and L-arabinose via an oxidative pathway to α-ketoglutarate as an intermediate. The enzymes of this pathway are encoded by the xac gene cluster (xylose and arabinose catabolism) which also contains genes (xacGHIJK) that encode all components of a putative ABC transporter. The xacGHIJK genes encode one substrate binding protein, two transmembrane domains and two nucleotide binding domains. It is shown here, that xacGHIJK is upregulated by both D-xylose and L-arabinose mediated by the transcriptional regulator XacR, the general regulator of xac genes. Knock-out mutants of xacG and of xacGHIJK resulted in a reduced growth rate on both pentoses; wild type growth could be recovered by complementation in trans. Together, the data indicate that uptake of xylose and arabinose in H. volcanii is mediated by this ABC transporter. Pentose specific ABC transporters, homologous to that of H. volcanii, were identified in other haloarchaea suggesting a similar function in pentose uptake in these archaea. Sequence analyses attribute the haloarchaeal pentose ABC transporter to the CUT1 (carbohydrate uptake transporter 1) subfamily.<br/>
</details><br/>
---

*Translational coupling via termination-reinitiation in archaea and bacteria.*<br/>
Huber M; Faure G; Laass S; Kolbe E; Seitz K; Wehrheim C; Wolf YI; Koonin EV; Soppa J<br/>
**2019**<br/>
Nat Commun 10(1)<br/>
10.1038/s41467-019-11999-9
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31488843)<br/>
[Citation]()<br/>
The genomes of many prokaryotes contain substantial fractions of gene pairs with overlapping stop and start codons (ATGA or TGATG). A potential benefit of overlapping gene pairs is translational coupling. In 720 genomes of archaea and bacteria representing all major phyla, we identify substantial, albeit highly variable, fractions of co-directed overlapping gene pairs. Various patterns are observed for the utilization of the SD motif for de novo initiation at upstream genes versus reinitiation at overlapping gene pairs. We experimentally test the predicted coupling in 9 gene pairs from the archaeon Haloferax volcanii and 5 gene pairs from the bacterium Escherichia coli. In 13 of 14 cases, translation of both genes is strictly coupled. Mutational analysis of SD motifs located upstream of the downstream genes indicate that the contribution of the SD to translational coupling widely varies from gene to gene. The nearly universal, abundant occurrence of overlapping gene pairs suggests that tight translational coupling is widespread in archaea and bacteria.<br/>
</details><br/>
---

*Rhodanese-Like Domain Protein UbaC and Its Role in Ubiquitin-Like Protein Modification and Sulfur Mobilization in Archaea.*<br/>
Hepowit NL; Maupin-Furlow JA<br/>
**2019**<br/>
J. Bacteriol. 201(15)<br/>
10.1128/JB.00254-19
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/31085691)<br/>
[Citation]()<br/>
Ubiquitin-like protein (Ubl) modification targets proteins for transient inactivation and/or proteasome-mediated degradation in archaea. Here the rhodanese-like domain (RHD) protein UbaC (HVO_1947) was found to copurify with the E1-like enzyme (UbaA) of the Ubl modification machinery in the archaeon <br/>
</details><br/>
---

*Haloferax volcanii as immobilised whole cell biocatalyst: new applications for halophilic systems.*<br/>
Haque RU; Paradisi F; Allers T<br/>
**2019**<br/>
Appl. Microbiol. Biotechnol. 103(9)<br/>
10.1007/s00253-019-09725-y
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30877354)<br/>
[Citation]()<br/>
Enzyme-mediated synthesis of pharmaceutical compounds is a 'green' alternative to traditional synthetic chemistry, and microbial engineering opens up the possibility of using whole cells as mini-factories. Whole-cell biocatalysis reduces cost by eliminating expensive enzyme purification and cofactor addition steps, as well as resulting in increased enzyme stability. Haloferax volcanii is a model halophilic archaeon encoding highly salt and organic solvent tolerant enzymes such as alcohol dehydrogenase (HvADH2), which catalyses the reduction of aldehydes and ketone in the presence of NADPH/NADH cofactor. A H. volcanii strain for constitutive HvADH2 expression was generated using a strong synthetic promoter (p.syn). The strain was immobilised in calcium alginate beads and repeatedly used as a whole-cell biocatalyst. The reduction of acetophenone, used as test substrate, was very successful and high yields were detected from immobilised whole cells over repeated biotransformation cycles. The immobilised H. volcanii retained stability and high product yields after 1 month of storage at room temperature. This newly developed system offers halophilic enzyme expression in its native environment, high product yield, stability and reusability without the addition of any expensive NADPH/NADH cofactor. This is the first report of whole cell-mediated biocatalysis by the halophilic archaeon H. volcanii.<br/>
</details><br/>
---

*Assays for ubiquitin-like protein ligation and proteasome function in archaea.*<br/>
Fu X; Adams Z; Maupin-Furlow J<br/>
**2019**<br/>
Meth. Enzymol. 619(None)<br/>
10.1016/bs.mie.2018.12.036
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30910020)<br/>
[Citation]()<br/>
Ubiquitin-like protein (Ubl) ligation is common to diverse archaea and targets many cellular pathways, including those associated with sulfur mobilization, and also tags proteins as substrates for degradation by the proteasome. Here we highlight protocols to assay proteasome function and Ubl ligation in archaea. A chase assay is described to monitor the impact of proteasome function on the stability of Ubl-modified proteins in the cell. A method to reconstitute Ubl ligation using a purified E1-like enzyme (UbaA), Ubl (SAMP2), methionine sulfoxide reductase A (MsrA), and cell lysate of an ΔmsrA ΔubaA Δsamp1-3 mutant is also described. MsrA is found to have the surprising ability to stimulate the formation of Ubl bonds. Haloferax volcanii, a halophilic archaeon originally isolated from the Dead Sea, serves as the model organism for these protocols.<br/>
</details><br/>
---

*Structural Characterization of the Xi Class Glutathione Transferase From the Haloalkaliphilic Archaeon Natrialba magadii*<br/>
Di Matteo, Adele; Federici, Luca; Masulli, Michele; Carletti, Erminia; Santorelli, Daniele; Cassidy, Jennifer; Paradisi, Francesca; Di Ilio, Carmine; Allocati, Nerino<br/>
**2019**<br/>
Frontiers in Microbiology 10()<br/>
10.3389/fmicb.2019.00009
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://www.frontiersin.org/article/10.3389/fmicb.2019.00009)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/DiMatteo2019FrontMicrobiol.ris)<br/>
Xi class glutathione transferases (GSTs) are a recently identified group, within this large superfamily of enzymes, specifically endowed with glutathione-dependent reductase activity on glutathionyl-hydroquinone. Enzymes belonging to this group are widely distributed in bacteria, fungi, and plants but not in higher eukaryotes. Xi class GSTs are also frequently found in archaea and here we focus on the enzyme produced by the extreme haloalkaliphilic archaeon Natrialba magadii (NmGHR). We investigated its function and stability and determined its 3D structure in the apo form by X-ray crystallography. NmGHR displays the same fold of its mesophilic counterparts, is enriched in negatively charged residues, which are evenly distributed along the surface of the protein, and is characterized by a peculiar distribution of hydrophobic residues. A distinctive feature of haloalkaliphilic archaea is their preference for Î³-glutamyl-cysteine over glutathione as a reducing thiol. Indeed we found that the N. magadii genome lacks a gene coding for glutathione synthase. Analysis of NmGHR structure suggests that the thiol binding site (G-site) of the enzyme is well suited for hosting Î³-glutamyl-cysteine.<br/>
</details><br/>
---

*Cyclic nucleotides in archaea: Cyclic di-AMP in the archaeon Haloferax volcanii and its putative role.*<br/>
Braun F; Thomalla L; van der Does C; Quax TEF; Allers T; Kaever V; Albers SV<br/>
**2019**<br/>
Microbiologyopen None(None)<br/>
10.1002/mbo3.829
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30884174)<br/>
[Citation]()<br/>
The role of cyclic nucleotides as second messengers for intracellular signal transduction has been well described in bacteria. One recently discovered bacterial second messenger is cyclic di-adenylate monophosphate (c-di-AMP), which has been demonstrated to be essential in bacteria. Compared to bacteria, significantly less is known about second messengers in archaea. This study presents the first evidence of in vivo presence of c-di-AMP in an archaeon. The model organism Haloferax volcanii was demonstrated to produce c-di-AMP. Its genome encodes one diadenylate cyclase (DacZ) which was shown to produce c-di-AMP in vitro. Similar to bacteria, the dacZ gene is essential and homologous overexpression of DacZ leads to cell death, suggesting the need for tight regulation of c-di-AMP levels. Such tight regulation often indicates the control of important regulatory processes. A central target of c-di-AMP signaling in bacteria is cellular osmohomeostasis. The results presented here suggest a comparable function in H. volcanii. A strain with decreased c-di-AMP levels exhibited an increased cell area in hypo-salt medium, implying impaired osmoregulation. In summary, this study expands the field of research on c-di-AMP and its physiological function to archaea and indicates that osmoregulation is likely to be a common function of c-di-AMP in bacteria and archaea.<br/>
</details><br/>
---

*Evolutionary insights into Trm112-methyltransferase holoenzymes involved in translation between archaea and eukaryotes.*<br/>
van Tran N; Muller L; Ross RL; Lestini R; Létoquart J; Ulryck N; Limbach PA; de Crécy-Lagard V; Cianférani S; Graille M<br/>
**2018**<br/>
Nucleic Acids Res. 46(16)<br/>
nar/gky638
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30010922)<br/>
[Citation]()<br/>
Protein synthesis is a complex and highly coordinated process requiring many different protein factors as well as various types of nucleic acids. All translation machinery components require multiple maturation events to be functional. These include post-transcriptional and post-translational modification steps and methylations are the most frequent among these events. In eukaryotes, Trm112, a small protein (COG2835) conserved in all three domains of life, interacts and activates four methyltransferases (Bud23, Trm9, Trm11 and Mtq2) that target different components of the translation machinery (rRNA, tRNAs, release factors). To clarify the function of Trm112 in archaea, we have characterized functionally and structurally its interaction network using Haloferax volcanii as model system. This led us to unravel that methyltransferases are also privileged Trm112 partners in archaea and that this Trm112 network is much more complex than anticipated from eukaryotic studies. Interestingly, among the identified enzymes, some are functionally orthologous to eukaryotic Trm112 partners, emphasizing again the similarity between eukaryotic and archaeal translation machineries. Other partners display some similarities with bacterial methyltransferases, suggesting that Trm112 is a general partner for methyltransferases in all living organisms.<br/>
</details><br/>
---

*Sialic Acid-Like Sugars in Archaea: Legionaminic Acid Biosynthesis in the Halophile Halorubrum sp. PV6*<br/>
Zaretsky, Marianna; Roine, Elina; Eichler, Jerry<br/>
**2018**<br/>
Frontiers in Microbiology 9()<br/>
10.3389/fmicb.2018.02133
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://www.frontiersin.org/article/10.3389/fmicb.2018.02133)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Zaretsky2018FrontMicrobiol.ris)<br/>
N-glycosylation is a post-translational modification that occurs in all three domains. In Archaea, however, N-linked glycans present a degree of compositional diversity not observed in either Eukarya or Bacteria. As such, it is surprising that nonulosonic acids (NulOs), nine-carbon sugars that include sialic acids, pseudaminic acids, and legionaminic acids, are routinely detected as components of protein-linked glycans in Eukarya and Bacteria but not in Archaea. In the following, we report that the N-linked glycan attached to the S-layer glycoprotein of the haloarchaea Halorubrum sp. PV6 includes an N-formylated legionaminic acid. Analysis of the Halorubrum sp. PV6 genome led to the identification of sequences predicted to comprise the legionaminic acid biosynthesis pathway. The transcription of pathway genes was confirmed, as was the co-transcription of several of these genes. In addition, the activities of LegI, which catalyzes the condensation of 2,4-di-N-acetyl-6-deoxymannose and phosphoenolpyruvate to generate legionaminic acid, and LegF, which catalyzes the addition of cytidine monophosphate (CMP) to legionaminic acid, both heterologously expressed in Haloferax volcanii, were demonstrated. Further genome analysis predicts that the genes encoding enzymes of the legionaminic acid biosynthetic pathway are clustered together with sequences seemingly encoding components of the N-glycosylation pathway in this organism. In defining the first example of a legionaminic acid biosynthesis pathway in Archaea, the findings reported here expand our insight into archaeal N-glycosylation, an almost universal post-translational modification in this domain of life.<br/>
</details><br/>
---

*mRNA-specific translation regulation by a ribosome-associated ncRNA in Haloferax volcanii.*<br/>
Wyss L; Waser M; Gebetsberger J; Zywicki M; Polacek N<br/>
**2018**<br/>
Sci Rep 8(1)<br/>
10.1038/s41598-018-30332-w
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30131517)<br/>
[Citation]()<br/>
Regulation of gene expression at the translational level allows rapid adaptation of cellular proteomes to quickly changing environmental conditions and is thus central for prokaryotic organisms. Small non-coding RNAs (sRNAs) have been reported to effectively orchestrate translation control in bacteria and archaea mainly by targeting mRNAs by partial base complementarity. Here we report an unprecedented mechanism how sRNAs are capable of modulating protein biosynthesis in the halophilic archaeon Haloferax volcanii. By analyzing the ribosome-associated ncRNAs (rancRNAs) under different stress conditions we identified an intergenic sRNA, termed rancRNA_s194, that is primarily expressed during exponential growth under all tested conditions. By interaction with the ribosome rancRNA_s194 inhibits peptide bond formation and protein synthesis in vitro but appears to target a specific mRNA in vivo. The respective knock-out strain shows a reduced lag phase in media containing xylose as sole carbon source and outcompetes the wildtype cells under these conditions. Mass spectrometry, polysome profiling and mRNA binding competition experiments suggest that rancRNA_s194 prevents the cstA mRNA from being efficiently translated by H. volcanii ribosomes. These findings enlarge the regulatory repertoire of archaeal sRNAs in modulating post-transcriptional gene expression.<br/>
</details><br/>
---

*Everyone loves an underdog: metabolic engineering of the xylose oxidative pathway in recombinant microorganisms.*<br/>
Valdehuesa KNG; Ramos KRM; Nisola GM; Bañares AB; Cabulong RB; Lee WK; Liu H; Chung WJ<br/>
**2018**<br/>
Appl. Microbiol. Biotechnol. 102(18)<br/>
10.1007/s00253-018-9186-z
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30003296)<br/>
[Citation]()<br/>
The D-xylose oxidative pathway (XOP) has recently been employed in several recombinant microorganisms for growth or for the production of several valuable compounds. The XOP is initiated by D-xylose oxidation to D-xylonolactone, which is then hydrolyzed into D-xylonic acid. D-Xylonic acid is then dehydrated to form 2-keto-3-deoxy-D-xylonic acid, which may be further dehydrated then oxidized into α-ketoglutarate or undergo aldol cleavage to form pyruvate and glycolaldehyde. This review introduces a brief discussion about XOP and its discovery in bacteria and archaea, such as Caulobacter crescentus and Haloferax volcanii. Furthermore, the current advances in the metabolic engineering of recombinant strains employing the XOP are discussed. This includes utilization of XOP for the production of diols, triols, and short-chain organic acids in Escherichia coli, Saccharomyces cerevisiae, and Corynebacterium glutamicum. Improving the D-xylose uptake, growth yields, and product titer through several metabolic engineering techniques bring some of these recombinant strains close to industrial viability. However, more developments are still needed to optimize the XOP pathway in the host strains, particularly in the minimization of by-product formation.<br/>
</details><br/>
---

*Two distinct glyceraldehyde-3-phosphate dehydrogenases in glycolysis and gluconeogenesis in the archaeon Haloferax volcanii.*<br/>
Tästensen JB; Schönheit P<br/>
**2018**<br/>
FEBS Lett. 592(9)<br/>
10.1002/1873-3468.13037
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29572819)<br/>
[Citation]()<br/>
The halophilic archaeon Haloferax volcanii degrades glucose via the semiphosphorylative Entner-Doudoroff pathway and can also grow on gluconeogenic substrates. Here, the enzymes catalysing the conversion of glyceraldehyde-3-phosphate (GAP) to 3-phosphoglycerate were analysed. The genome contains the genes gapI and gapII encoding two putative GAP dehydrogenases, and pgk encoding phosphoglycerate kinase (PGK). We show that gapI is functionally involved in sugar catabolism, whereas gapII is involved in gluconeogenesis. For pgk, an amphibolic function is indicated. This is the first report of the functional involvement of a phosphorylating glyceraldehyde-3-phosphate dehydrogenase and PGK in sugar catabolism in archaea. Phylogenetic analyses indicate that the catabolic gapI from H. volcanii is acquired from bacteria via lateral genetransfer, whereas the anabolic gapII as well as pgk are of archaeal origin.<br/>
</details><br/>
---

*A Cobalamin Activity-Based Probe Enables Microbial Cell Growth and Finds New Cobalamin-Protein Interactions across Domains.*<br/>
Rosnow JJ; Hwang S; Killinger BJ; Kim YM; Moore RJ; Lindemann SR; Maupin-Furlow JA; Wright AT<br/>
**2018**<br/>
Appl. Environ. Microbiol. 84(18)<br/>
10.1128/AEM.00955-18
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30006406)<br/>
[Citation]()<br/>
Understanding the factors that regulate microbe function and microbial community assembly, function, and fitness is a grand challenge. A critical factor and an important enzyme cofactor and regulator of gene expression is cobalamin (vitamin B<br/>
</details><br/>
---

*Characterizing the DNA Methyltransferases of Haloferax volcanii via Bioinformatics, Gene Deletion, and SMRT Sequencing.*<br/>
Ouellette M; Gogarten JP; Lajoie J; Makkay AM; Papke RT<br/>
**2018**<br/>
Genes (Basel) 9(3)<br/>
10.3390/genes9030129
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29495512)<br/>
[Citation]()<br/>
DNA methyltransferases (MTases), which catalyze the methylation of adenine and cytosine bases in DNA, can occur in bacteria and archaea alongside cognate restriction endonucleases (REases) in restriction-modification (RM) systems or independently as orphan MTases. Although DNA methylation and MTases have been well-characterized in bacteria, research into archaeal MTases has been limited. A previous study examined the genomic DNA methylation patterns (methylome) of the halophilic archaeon <br/>
</details><br/>
---

*GlpR Is a Direct Transcriptional Repressor of Fructose Metabolic Genes in &lt;span class=&quot;named-content genus-species&quot; id=&quot;named-content-1&quot;&gt;Haloferax volcanii&lt;/span&gt;*<br/>
Martin, Jonathan H.; Sherwood Rawls, Katherine; Chan, Jou Chin; Hwang, Sungmin; Martinez-Pastor, Mar; McMillan, Lana J.; Prunetti, Laurence; Schmid, Amy K.; Maupin-Furlow, Julie A.<br/>
**2018**<br/>
J. Bacteriol. 200(17)<br/>
10.1128/JB.00244-18
<details>
<summary>Abstract and Links</summary>

[Link to Publication](http://jb.asm.org/content/200/17/e00244-18.abstract)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Martin2019JBact.ris)<br/>
DeoR-type helix-turn-helix (HTH) domain proteins are transcriptional regulators of sugar and nucleoside metabolism in diverse bacteria and also occur in select archaea. In the model archaeon Haloferax volcanii, previous work implicated GlpR, a DeoR-type transcriptional regulator, in the transcriptional repression of glpR and the gene encoding the fructose-specific phosphofructokinase (pfkB) during growth on glycerol. However, the global regulon governed by GlpR remained unclear. Here, we compared transcriptomes of wild-type and Î”glpR mutant strains grown on glycerol and glucose to detect significant transcript level differences for nearly 50 new genes regulated by GlpR. By coupling computational prediction of GlpR binding sequences with in vivo and in vitro DNA binding experiments, we determined that GlpR directly controls genes encoding enzymes involved in fructose degradation, including fructose bisphosphate aldolase, a central control point in glycolysis. GlpR also directly controls other transcription factors. In contrast, other metabolic pathways appear to be under the indirect influence of GlpR. In vitro experiments demonstrated that GlpR purifies to function as a tetramer that binds the effector molecule fructose-1-phosphate (F1P). These results suggest that H. volcanii GlpR functions as a direct negative regulator of fructose degradation during growth on carbon sources other than fructose, such as glucose and glycerol, and that GlpR bears striking functional similarity to bacterial DeoR-type regulators.IMPORTANCE Many archaea are extremophiles, able to thrive in habitats of extreme salinity, pH and temperature. These biological properties are ideal for applications in biotechnology. However, limited knowledge of archaeal metabolism is a bottleneck that prevents the broad use of archaea as microbial factories for industrial products. Here, we characterize how sugar uptake and use are regulated in a species that lives in high salinity. We demonstrate that a key sugar regulatory protein in this archaeal species functions using molecular mechanisms conserved with distantly related bacterial species.<br/>
</details><br/>
---

*Characterization of Copy Number Control of Two &lt;span class=&quot;named-content genus-species&quot; id=&quot;named-content-1&quot;&gt;Haloferax volcanii&lt;/span&gt; Replication Origins Using Deletion Mutants and Haloarchaeal Artificial Chromosomes*<br/>
Maurer, Sandy; Ludt, Katharina; Soppa, JÃ¶rg<br/>
**2018**<br/>
J. Bacteriol. 200(1)<br/>
10.1128/JB.00517-17
<details>
<summary>Abstract and Links</summary>

[Link to Publication](http://jb.asm.org/content/200/1/e00517-17.abstract)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Maurer2017JBact.ris)<br/>
Haloferax volcanii is polyploid and contains about 20 genome copies under optimal conditions. However, the chromosome copy number is highly regulated and ranges from two during phosphate starvation to more than 40 under conditions of phosphate surplus. The aim of this study was the characterization of the influence of two replication origins on the genome copy number. The origin repeats and the genes encoding origin recognition complex (ORC) proteins were deleted. The core origin oriC1-orc1 (ori1) deletion mutant had a lower genome copy number and a higher level of fitness than the wild type, in stark contrast to the oriC2-orc5 (ori2) deletion mutant. The genes adjacent to ori1 could not be deleted, and thus, at least two of them are probably essential, while deletion of the genes adjacent to ori2 was possible. Various fragments of and around the origins were cloned into a suicide plasmid to generate haloarchaeal artificial chromosomes (HACs). The copy number of the oriC1-orc1 HAC was much higher than that of the oriC2-orc5 HAC. The addition of adjacent genes influenced both the HAC copy number and the chromosome copy number. The results indicate that the origins of H. volcanii are not independent but that the copy number is regulated via a network of genes around the origins.IMPORTANCE Several species of archaea have more than one origin of replication on their major chromosome and are thus the only known prokaryotic species that allow the analysis of the evolution of multiorigin replication. The widely studied Haloferax volcanii H26 strain has a major chromosome with four origins of replication. Two origins, ori1 and ori2, were chosen for an in-depth analysis using deletion mutants and haloarchaeal artificial chromosomes. The analysis was not restricted to the core origin regions; origin-adjacent genes were also included. Because H. volcanii is polyploid, the effects on the chromosome copy number were of specific importance. The results revealed extreme differences between the two origins.<br/>
</details><br/>
---

*Multiplex quantitative SILAC for analysis of archaeal proteomes: a case study of oxidative stress responses.*<br/>
McMillan LJ; Hwang S; Farah RE; Koh J; Chen S; Maupin-Furlow JA<br/>
**2018**<br/>
Environ. Microbiol. 20(1)<br/>
10.1111/1462-2920.14014
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29194950)<br/>
[Citation]()<br/>
Stable isotope labelling of amino acids in cell culture (SILAC) is a quantitative proteomic method that can illuminate new pathways used by cells to adapt to different lifestyles and niches. Archaea, while thriving in extreme environments and accounting for ∼20%-40% of the Earth's biomass, have not been analyzed with the full potential of SILAC. Here, we report SILAC for quantitative comparison of archaeal proteomes, using Haloferax volcanii as a model. A double auxotroph was generated that allowed for complete incorporation of <br/>
</details><br/>
---

*Influence of Origin Recognition Complex Proteins on the Copy Numbers of Three Chromosomes in Haloferax volcanii.*<br/>
Ludt K; Soppa J<br/>
**2018**<br/>
J. Bacteriol. 200(17)<br/>
10.1128/JB.00161-18
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29941422)<br/>
[Citation]()<br/>
Replication initiation in archaea involves a protein named ORC, Cdc6, or ORC1/Cdc6, which is homologous to the eukaryotic origin recognition complex (ORC) proteins and to the eukaryotic Cdc6. Archaeal replication origins are comprised of origin repeat regions and adjacent <br/>
</details><br/>
---

*Mechanism of salt-induced activity enhancement of a marine-derived laccase, Lac15.*<br/>
Li J; Xie Y; Wang R; Fang Z; Fang W; Zhang X; Xiao Y<br/>
**2018**<br/>
Eur. Biophys. J. 47(3)<br/>
10.1007/s00249-017-1251-5
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28875401)<br/>
[Citation]()<br/>
Laccase (benzenediol: oxygen oxidoreductases, EC1.10.3.2) is a multi-copper oxidase capable of oxidizing a variety of phenolic and other aromatic organic compounds. The catalytic power of laccase makes it an attractive candidate for potential applications in many areas of industry including biodegradation of organic pollutants and synthesis of novel drugs. Most laccases are vulnerable to high salt and have limited applications. However, some laccases are not only tolerant to but also activated by certain concentrations of salt and thus have great application potential. The mechanisms of salt-induced activity enhancement of laccases are unclear as yet. In this study, we used dynamic light scattering, size exclusion chromatography, analytical ultracentrifugation, intrinsic fluorescence emission, circular dichroism, ultraviolet-visible light absorption, and an enzymatic assay to investigate the potential correlation between the structure and activity of the marine-derived laccase, Lac15, whose activity is promoted by low concentrations of NaCl. The results showed that low concentrations of NaCl exert little influence on the protein structure, which was partially folded in the absence of the salt; moreover, the partially folded rather than the fully folded state seemed to be favorable for enzyme activity, and this partially folded state was distinctive from the so-called 'molten globule' occasionally observed in active enzymes. More data indicated that salt might promote laccase activity through mechanisms involving perturbation of specific local sites rather than a change in global structure. Potential binding sites for chloride ions and their roles in enzyme activity promotion are proposed.<br/>
</details><br/>
---

*Genome sequence and description of Haloferax massiliense sp. nov., a new halophilic archaeon isolated from the human gut.*<br/>
Khelaifia S; Caputo A; Andrieu C; Cadoret F; Armstrong N; Michelle C; Lagier JC; Djossou F; Fournier PE; Raoult D<br/>
**2018**<br/>
Extremophiles 22(3)<br/>
10.1007/s00792-018-1011-1
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29435649)<br/>
[Citation]()<br/>
By applying the culturomics concept and using culture conditions containing a high salt concentration, we herein isolated the first known halophilic archaeon colonizing the human gut. Here we described its phenotypic and biochemical characterization as well as its genome annotation. Strain Arc-Hr<br/>
</details><br/>
---

*Insights into the evolutionary conserved regulation of Rio ATPase activity.*<br/>
Knüppel R; Christensen RH; Gray FC; Esser D; Strauß D; Medenbach J; Siebers B; MacNeill SA; LaRonde N; Ferreira-Cerca S<br/>
**2018**<br/>
Nucleic Acids Res. 46(3)<br/>
nar/gkx1236
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29237037)<br/>
[Citation]()<br/>
Eukaryotic ribosome biogenesis is a complex dynamic process which requires the action of numerous ribosome assembly factors. Among them, the eukaryotic Rio protein family members (Rio1, Rio2 and Rio3) belong to an ancient conserved atypical protein kinase/ ATPase family required for the maturation of the small ribosomal subunit (SSU). Recent structure-function analyses suggested an ATPase-dependent role of the Rio proteins to regulate their dynamic association with the nascent pre-SSU. However, the evolutionary origin of this feature and the detailed molecular mechanism that allows controlled activation of the catalytic activity remained to be determined. In this work we provide functional evidence showing a conserved role of the archaeal Rio proteins for the synthesis of the SSU in archaea. Moreover, we unravel a conserved RNA-dependent regulation of the Rio ATPases, which in the case of Rio2 involves, at least, helix 30 of the SSU rRNA and the P-loop lysine within the shared RIO domain. Together, our study suggests a ribosomal RNA-mediated regulatory mechanism enabling the appropriate stimulation of Rio2 catalytic activity and subsequent release of Rio2 from the nascent pre-40S particle. Based on our findings we propose a unified release mechanism for the Rio proteins.<br/>
</details><br/>
---

*In vitro evaluation of archaeosome vehicles for transdermal vaccine delivery.*<br/>
Jia Y; McCluskie MJ; Zhang D; Monette R; Iqbal U; Moreno M; Sauvageau J; Williams D; Deschatelets L; Jakubek ZJ; Krishnan L<br/>
**2018**<br/>
J Liposome Res 28(4)<br/>
10.1080/08982104.2017.1376683
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28948854)<br/>
[Citation]()<br/>
Archaeosomes composed of archaeal total polar lipids (TPL) or semi-synthetic analog vesicles have been used as vaccine adjuvants and delivery systems in animal models for many years. Typically administered by intramuscular or subcutaneous injections, archaeosomes can induce robust, long-lasting humoral and cell-mediated immune responses against entrapped antigens and provide protection in murine models of infectious disease and cancer. Herein, we evaluated various archaeosomes for transdermal delivery, since this route may help eliminate needle-stick injuries and needle re-use, and therefore increase patient compliance. Archaeosomes composed of TPL from different archaea (Halobacterium salinarum, Methanobrevibacter smithii, Haloferax volcanii) and various semi-synthetic glycolipid combinations were evaluated for their ability to diffuse across the skin barrier using an ex vivo pig skin model and the results were compared to conventional synthetic ester liposomes. Physicochemical characteristics were determined for selected formulations including vesicle size, size distribution, zeta potential, fluidity, antigen (ovalbumin) incorporation efficiency and release. Archaeosomes, in particular those composed of M. smithii TPL or the synthetic glycolipid sulfated S-lactosylarchaeol (SLA) mixed with uncharged glycolipid lactosyl archaeol (LA), appeared to be effective carriers for ovalbumin, achieving much better antigen distribution and vesicle accumulation in the skin epidermis than conventional liposomes. The enhanced skin permeation of archaeosomes may be attributed to their chemical structure and physicochemical properties such as particle size, surface charge, stability, and fluidity of their lipid bilayer.<br/>
</details><br/>
---

*In Vitro Antioxidant, Antihemolytic, and Anticancer Activity of the Carotenoids from Halophilic Archaea.*<br/>
Hou J; Cui HL<br/>
**2018**<br/>
Curr. Microbiol. 75(3)<br/>
10.1007/s00284-017-1374-z
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29038844)<br/>
[Citation]()<br/>
Halophilic archaea represent a promising natural source of carotenoids. However, little information is available about the biological effects of carotenoids from halophilic archaea. In this study, the carotenoids produced by seven halophilic archaeal strains Halogeometricum rufum, Halogeometricum limi, Haladaptatus litoreus, Haloplanus vescus, Halopelagius inordinatus, Halogranum rubrum, and Haloferax volcanii were identified by ultraviolet/visible spectroscopy, thin-layer chromatography, and high-performance liquid chromatography-tandem mass spectrometry. The C<br/>
</details><br/>
---

*Molecular Factors of Hypochlorite Tolerance in the Hypersaline Archaeon Haloferax volcanii.*<br/>
Gomez M; Leung W; Dantuluri S; Pillai A; Gani Z; Hwang S; McMillan LJ; Kiljunen S; Savilahti H; Maupin-Furlow JA<br/>
**2018**<br/>
Genes (Basel) 9(11)<br/>
10.3390/genes9110562
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30463375)<br/>
[Citation]()<br/>
Halophilic archaea thrive in hypersaline conditions associated with desiccation, ultraviolet (UV) irradiation and redox active compounds, and thus are naturally tolerant to a variety of stresses. Here, we identified mutations that promote enhanced tolerance of halophilic archaea to redox-active compounds using <br/>
</details><br/>
---

*Transcriptional Landscape and Regulatory Roles of Small Noncoding RNAs in the Oxidative Stress Response of the Haloarchaeon Haloferax volcanii.*<br/>
Gelsinger DR; DiRuggiero J<br/>
**2018**<br/>
J. Bacteriol. 200(9)<br/>
10.1128/JB.00779-17
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29463600)<br/>
[Citation]()<br/>
Haloarchaea in their natural environment are exposed to hypersalinity, intense solar radiation, and desiccation, all of which generate high levels of oxidative stress. Previous work has shown that haloarchaea are an order of magnitude more resistant to oxidative stress than most mesophilic organisms. Despite this resistance, the pathways haloarchaea use to respond to oxidative stress damage are similar to those of nonresistant organisms, suggesting that regulatory processes might be key to their robustness. Recently, small regulatory noncoding RNAs (sRNAs) were discovered in <br/>
</details><br/>
---

*The trimeric Hef-associated nuclease HAN is a 3'→5' exonuclease and is probably involved in DNA repair.*<br/>
Feng L; Chang CC; Song D; Jiang C; Song Y; Wang CF; Deng W; Zou YJ; Chen HF; Xiao X; Wang FP; Liu XP<br/>
**2018**<br/>
Nucleic Acids Res. 46(17)<br/>
nar/gky707
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30102394)<br/>
[Citation]()<br/>
Nucleases play important roles in nucleic acid metabolism. Some archaea encode a conserved protein known as Hef-associated nuclease (HAN). In addition to its C-terminal DHH nuclease domain, HAN also has three N-terminal domains, including a DnaJ-Zinc-finger, ribosomal protein S1-like, and oligonucleotide/oligosaccharide-binding fold. To further understand HAN's function, we biochemically characterized the enzymatic properties of HAN from Pyrococcus furiosus (PfuHAN), solved the crystal structure of its DHH nuclease domain, and examined its role in DNA repair. Our results show that PfuHAN is a Mn2+-dependent 3'-exonuclease specific to ssDNA and ssRNA with no activity on blunt and 3'-recessive double-stranded DNA. Domain truncation confirmed that the intrinsic nuclease activity is dependent on the C-terminal DHH nuclease domain. The crystal structure of the DHH nuclease domain adopts a trimeric topology, with each subunit adopting a classical DHH phosphoesterase fold. Yeast two hybrid assay confirmed that the DHH domain interacts with the IDR peptide of Hef nuclease. Knockout of the han gene or its C-terminal DHH nuclease domain in Haloferax volcanii resulted in increased sensitivity to the DNA damage reagent MMS. Our results imply that HAN nuclease might be involved in repairing stalled replication forks in archaea.<br/>
</details><br/>
---

*Divergent Roles of RPA Homologs of the Model Archaeon Halobacterium salinarum in Survival of DNA Damage*<br/>
Evans, J. Jessica; Gygli, E. Patrick; McCaskill, Julienne; DeVeaux, C. Linda<br/>
**2018**<br/>
Genes 9(4)<br/>
10.3390/genes9040223
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://www.mdpi.com/2073-4425/9/4/223)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Evans2018Genes.ris)<br/>
The haloarchaea are unusual in possessing genes for multiple homologs to the ubiquitous single-stranded DNA binding protein (SSB or replication protein A, RPA) found in all three domains of life. Halobacterium salinarum contains five homologs: two are eukaryotic in organization, two are prokaryotic and are encoded on the minichromosomes, and one is uniquely euryarchaeal. Radiation-resistant mutants previously isolated show upregulation of one of the eukaryotic-type RPA genes. Here, we have created deletions in the five RPA operons. These deletion mutants were exposed to DNA-damaging conditions: ionizing radiation, UV radiation, and mitomycin C. Deletion of the euryarchaeal homolog, although not lethal as in Haloferax volcanii, causes severe sensitivity to all of these agents. Deletion of the other RPA/SSB homologs imparts a variable sensitivity to these DNA-damaging agents, suggesting that the different RPA homologs have specialized roles depending on the type of genomic insult encountered.<br/>
</details><br/>
---

*Snapshots of archaeal DNA replication and repair in living cells using super-resolution imaging.*<br/>
Delpech F; Collien Y; Mahou P; Beaurepaire E; Myllykallio H; Lestini R<br/>
**2018**<br/>
Nucleic Acids Res. 46(20)<br/>
nar/gky829
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/30212908)<br/>
[Citation]()<br/>
Using the haloarchaeon Haloferax volcanii as a model, we developed nascent DNA labeling and the functional GFP-labeled single-stranded binding protein RPA2 as novel tools to gain new insight into DNA replication and repair in live haloarchaeal cells. Our quantitative fluorescence microscopy data revealed that RPA2 forms distinct replication structures that dynamically responded to replication stress and DNA damaging agents. The number of the RPA2 foci per cell followed a probabilistic Poisson distribution, implying hitherto unnoticed stochastic cell-to-cell variation in haloarchaeal DNA replication and repair processes. The size range of haloarchaeal replication structures is very similar to those observed earlier in eukaryotic cells. The improved lateral resolution of 3D-SIM fluorescence microscopy allowed proposing that inhibition of DNA synthesis results in localized replication foci clustering and facilitated observation of RPA2 complexes brought about by chemical agents creating DNA double-strand breaks. Altogether our in vivo observations are compatible with earlier in vitro studies on archaeal single-stranded DNA binding proteins. Our work thus underlines the great potential of live cell imaging for unraveling the dynamic nature of transient molecular interactions that underpin fundamental molecular processes in the Third domain of life.<br/>
</details><br/>
---

*Significant Enhancement of Structural Stability of the Hyperhalophilic ADH from Haloferax volcanii via Entrapment on Metal Organic Framework Support.*<br/>
Carucci C; Bruen L; Gascón V; Paradisi F; Magner E<br/>
**2018**<br/>
Langmuir 34(28)<br/>
10.1021/acs.langmuir.8b01037
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29920206)<br/>
[Citation]()<br/>
The use of an in situ immobilization procedure for the immobilization of hyperhalophilic alcohol dehydrogenase in a metal organic framework material is described. The easy and rapid in situ immobilization process enables retention of activity over a broad range of pH and temperature together with a decrease in the halophilicity of the enzyme. The catalytic activity of the immobilized enzyme was studied in nonaqueous solvent mixtures with the highest retention of activity in aqueous solutions of methanol and acetonitrile. The approach demonstrates that this immobilization method can be extended to hyperhalophilic enzymes with enhancements in activity and stability.<br/>
</details><br/>
---

*Proteomic Study of the Exponential-Stationary Growth Phase Transition in the Haloarchaea Natrialba magadii and Haloferax volcanii.*<br/>
Cerletti M; Giménez MI; Tröetschel C; D' Alessandro C; Poetsch A; De Castro RE; Paggi RA<br/>
**2018**<br/>
Proteomics 18(14)<br/>
10.1002/pmic.201800116
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29888524)<br/>
[Citation]()<br/>
The dynamic changes that take place along the phases of microbial growth (lag, exponential, stationary, and death) have been widely studied in bacteria at the molecular and cellular levels, but little is known for archaea. In this study, a high-throughput approach was used to analyze and compare the proteomes of two haloarchaea during exponential and stationary growth: the neutrophilic Haloferax volcanii and the alkaliphilic Natrialba magadii. Almost 2000 proteins were identified in each species (≈50% of the predicted proteome). Among them, 532 and 432 were found to be differential between growth phases in H. volcanii and N. magadii, respectively. Changes upon entrance into stationary phase included an overall increase in proteins involved in the transport of small molecules and ions, stress response, and fatty acid catabolism. Proteins related to genetic processes and cell division showed a notorious decrease in amount. The data reported in this study not only contributes to our understanding of the exponential-stationary growth phase transition in extremophilic archaea but also provides the first comprehensive analysis of the proteome composition of N. magadii. The MS proteomics data have been deposited in the ProteomeXchange Consortium with the dataset identifier JPST000395.<br/>
</details><br/>
---

*Haloquadratum walsbyi Yields a Versatile, NAD*<br/>
Cassidy J; Paradisi F<br/>
**2018**<br/>
Mol. Biotechnol. 60(6)<br/>
10.1007/s12033-018-0083-6
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29654471)<br/>
[Citation]()<br/>
This study presents the first example of an alcohol dehydrogenase (ADH) from the halophilic archaeum Haloquadratum walsbyi (HwADH). A hexahistidine-tagged recombinant HwADH was heterologously overexpressed in Haloferax volcanii. HwADH was purified in one step and was found to be thermophilic with optimal activity at 65 °C. HwADH was active in the presence of 10% (v/v) organic solvent. The enzyme displayed dual cofactor specificity and a broad substrate scope, and maximum activity was detected with benzyl alcohol and 2-phenyl-1-propanol. HwADH accepted aromatic ketones, acetophenone and phenylacetone as substrates. The enzyme also accepted cyclohexanol and aromatic secondary alcohols, 1-phenylethanol and 4-phenyl-2-butanol. H. walsbyi may offer an excellent alternative to other archaeal sources to expand the toolbox of halophilic biocatalysts.<br/>
</details><br/>
---

*LonB Protease Is a Novel Regulator of Carotenogenesis Controlling Degradation of Phytoene Synthase in Haloferax volcanii.*<br/>
Cerletti M; Paggi R; Troetschel C; Ferrari MC; Guevara CR; Albaum S; Poetsch A; De Castro R<br/>
**2018**<br/>
J. Proteome Res. 17(3)<br/>
10.1021/acs.jproteome.7b00809
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29411617)<br/>
[Citation]()<br/>
The membrane protease LonB is an essential protein in the archaeon Haloferax volcanii and globally impacts its physiology. However, natural substrates of the archaeal Lon protease have not been identified. The whole proteome turnover was examined in a H. volcanii LonB mutant under reduced and physiological protease levels. LC-MS/MS combined with stable isotope labeling was applied for the identification/quantitation of membrane and cytoplasm proteins. Differential synthesis and degradation rates were evidenced for 414 proteins in response to Lon expression. A total of 58 proteins involved in diverse cellular processes showed a degradation pattern (none/very little degradation in the absence of Lon and increased degradation in the presence of Lon) consistent with a LonB substrate, which was further substantiated for several of these candidates by pull-down assays. The most notable was phytoene synthase (PSY), the rate-limiting enzyme in carotenoid biosynthesis. The rapid degradation of PSY upon LonB induction in addition to the remarkable stabilization of this protein and hyperpigmentation phenotype in the Lon mutant strongly suggest that PSY is a LonB substrate. This work identifies for the first time candidate targets of the archaeal Lon protease and establishes proteolysis by Lon as a novel post-translational regulatory mechanism of carotenogenesis.<br/>
</details><br/>
---

*Haloferax volcanii Proteome Response to Deletion of a Rhomboid Protease Gene.*<br/>
Costa MI; Cerletti M; Paggi RA; Trötschel C; De Castro RE; Poetsch A; Giménez MI<br/>
**2018**<br/>
J. Proteome Res. 17(3)<br/>
10.1021/acs.jproteome.7b00530
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29301397)<br/>
[Citation]()<br/>
Rhomboids are conserved intramembrane serine proteases involved in cell signaling processes. Their role in prokaryotes is scarcely known and remains to be investigated in Archaea. We previously constructed a rhomboid homologue deletion mutant (ΔrhoII) in Haloferax volcanii, which showed reduced motility, increased novobiocin sensitivity, and an N- glycosylation defect. To address the impact of rhoII deletion on H. volcanii physiology, the proteomes of mutant and parental strains were compared by shotgun proteomics. A total of 1847 proteins were identified (45.8% of H. volcanii predicted proteome), from which 103 differed in amount. Additionally, the mutant strain evidenced 99 proteins with altered electrophoretic migration, which suggested differential post-translational processing/modification. Integral membrane proteins that evidenced variations in concentration, electrophoretic migration, or semitryptic cleavage in the mutant were considered as potential RhoII targets. These included a PrsW protease homologue (which was less stable in the mutant strain), a predicted halocyanin, and six integral membrane proteins potentially related to the mutant glycosylation (S-layer glycoprotein, Agl15) and cell adhesion/motility (flagellin1, HVO_1153, PilA1, and PibD) defects. This study investigated for the first time the impact of a rhomboid protease on the whole proteome of an organism.<br/>
</details><br/>
---

*Structural and functional adaptation of Haloferax volcanii TFEα/β.*<br/>
Blombach F; Ausiannikava D; Figueiredo AM; Soloviev Z; Prentice T; Zhang M; Zhou N; Thalassinos K; Allers T; Werner F<br/>
**2018**<br/>
Nucleic Acids Res. 46(5)<br/>
nar/gkx1302
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29309690)<br/>
[Citation]()<br/>
The basal transcription factor TFE enhances transcription initiation by catalysing DNA strand-separation, a process that varies with temperature and ionic strength. Canonical TFE forms a heterodimeric complex whose integrity and function critically relies on a cubane iron-sulphur cluster residing in the TFEβ subunit. Halophilic archaea such as Haloferax volcanii have highly divergent putative TFEβ homologues with unknown properties. Here, we demonstrate that Haloferax TFEβ lacks the prototypical iron-sulphur cluster yet still forms a stable complex with TFEα. A second metal cluster contained in the zinc ribbon domain in TFEα is highly degenerate but retains low binding affinity for zinc, which contributes to protein folding and stability. The deletion of the tfeB gene in H. volcanii results in the aberrant expression of approximately one third of all genes, consistent with its function as a basal transcription initiation factor. Interestingly, tfeB deletion particularly affects foreign genes including a prophage region. Our results reveal the loss of metal centres in Hvo transcription factors, and confirm the dual function of TFE as basal factor and regulator of transcription.<br/>
</details><br/>
---

*Evolution of Genome Architecture in Archaea: Spontaneous Generation of a New Chromosome in Haloferax volcanii.*<br/>
Ausiannikava D; Mitchell L; Marriott H; Smith V; Hawkins M; Makarova KS; Koonin EV; Nieduszynski CA; Allers T<br/>
**2018**<br/>
Mol. Biol. Evol. 35(8)<br/>
molbev/msy075
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29668953)<br/>
[Citation]()<br/>
The common ancestry of archaea and eukaryotes is evident in their genome architecture. All eukaryotic and several archaeal genomes consist of multiple chromosomes, each replicated from multiple origins. Three scenarios have been proposed for the evolution of this genome architecture: 1) mutational diversification of a multi-copy chromosome; 2) capture of a new chromosome by horizontal transfer; 3) acquisition of new origins and splitting into two replication-competent chromosomes. We report an example of the third scenario: the multi-origin chromosome of the archaeon Haloferax volcanii has split into two elements via homologous recombination. The newly generated elements are bona fide chromosomes, because each bears "chromosomal" replication origins, rRNA loci, and essential genes. The new chromosomes were stable during routine growth but additional genetic manipulation, which involves selective bottlenecks, provoked further rearrangements. To the best of our knowledge, rearrangement of a naturally evolved prokaryotic genome to generate two new chromosomes has not been described previously.<br/>
</details><br/>
---

*Conserved residues are critical for Haloferax volcanii archaeosortase catalytic activity: Implications for convergent evolution of the catalytic mechanisms of non-homologous sortases from archaea and bacteria.*<br/>
Abdul Halim MF; Rodriguez R; Stoltzfus JD; Duggin IG; Pohlschroder M<br/>
**2018**<br/>
Mol. Microbiol. 108(3)<br/>
10.1111/mmi.13935
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29465796)<br/>
[Citation]()<br/>
Proper protein anchoring is key to the biogenesis of prokaryotic cell surfaces, dynamic, resilient structures that play crucial roles in various cell processes. A novel surface protein anchoring mechanism in Haloferax volcanii depends upon the peptidase archaeosortase A (ArtA) processing C-termini of substrates containing C-terminal tripartite structures and anchoring mature substrates to the cell membrane via intercalation of lipid-modified C-terminal amino acid residues. While this membrane protein lacks clear homology to soluble sortase transpeptidases of Gram-positive bacteria, which also process C-termini of substrates whose C-terminal tripartite structures resemble those of ArtA substrates, archaeosortases do contain conserved cysteine, arginine and arginine/histidine/asparagine residues, reminiscent of His-Cys-Arg residues of sortase catalytic sites. The study presented here shows that ArtA<br/>
</details><br/>
---

*RadB acts in homologous recombination in the archaeon Haloferax volcanii, consistent with a role as recombination mediator.*<br/>
Wardell K; Haldenby S; Jones N; Liddell S; Ngo GHP; Allers T<br/>
**2017**<br/>
DNA Repair (Amst.) 55(None)<br/>
10.1016/j.dnarep.2017.04.005
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28501701)<br/>
[Citation]()<br/>
Homologous recombination plays a central role in the repair of double-strand DNA breaks, the restart of stalled replication forks and the generation of genetic diversity. Regulation of recombination is essential since defects can lead to genome instability and chromosomal rearrangements. Strand exchange is a key step of recombination - it is catalysed by RecA in bacteria, Rad51/Dmc1 in eukaryotes and RadA in archaea. RadB, a paralogue of RadA, is present in many archaeal species. RadB has previously been proposed to function as a recombination mediator, assisting in RadA-mediated strand exchange. In this study, we use the archaeon Haloferax volcanii to provide evidence to support this hypothesis. We show that RadB is required for efficient recombination and survival following treatment with DNA-damaging agents, and we identify two point mutations in radA that suppress the ΔradB phenotype. Analysis of these point mutations leads us to propose that the role of RadB is to act as a recombination mediator, which it does by inducing a conformational change in RadA and thereby promoting its polymerisation on DNA.<br/>
</details><br/>
---

*N-Glycosylation Is Important for Proper Haloferax volcanii S-Layer Stability and Function.*<br/>
Tamir A; Eichler J<br/>
**2017**<br/>
Appl. Environ. Microbiol. 83(6)<br/>
10.1128/AEM.03152-16
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28039139)<br/>
[Citation]()<br/>
N-Glycosylation, the covalent linkage of glycans to select Asn residues of target proteins, is an almost universal posttranslational modification in archaea. However, whereas roles for N-glycosylation have been defined in eukarya and bacteria, the function of archaeal N-glycosylation remains unclear. Here, the impact of perturbed N-glycosylation on the structure and physiology of the haloarchaeon <br/>
</details><br/>
---

*Cell Surface Glycosylation Is Required for Efficient Mating of Haloferax volcanii*<br/>
Shalev, Yarden; Turgeman-Grott, Israela; Tamir, Adi; Eichler, Jerry; Gophna, Uri<br/>
**2017**<br/>
Frontiers in microbiology 8()<br/>
10.3389/fmicb.2017.01253
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://www.ncbi.nlm.nih.gov/pubmed/28725221)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Shalev2017FrontMicrobiol.ris)<br/>
Halophilic archaea use a fusion-based mating system for lateral gene transfer across cells, yet the molecular mechanisms involved remain unknown. Previous work implied that cell fusion involves cell-cell recognition since fusion occurs more efficiently between cells from the same species. Long believed to be restricted only to Eukarya, it is now known that cells of all three domains of life perform N-glycosylation, the covalent attachment of glycans to select target asparagine residues in proteins, and that this post-translational modification is common for archaeal cell surface proteins. Here, we show that differences in glycosylation of the Haloferax volcanii surface-layer glycoprotein, brought about either by changing medium salinity or by knocking out key glycosylation genes, reduced mating success. Thus, different glycosylation patterns are likely to underlie mating preference in halophilic archaea, contributing to speciation processes.<br/>
</details><br/>
---

*Ribosome surface properties may impose limits on the nature of the cytoplasmic proteome.*<br/>
Schavemaker PE; Śmigiel WM; Poolman B<br/>
**2017**<br/>
Elife 6(None)<br/>
10.7554/eLife.30084
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29154755)<br/>
[Citation]()<br/>
Much of the molecular motion in the cytoplasm is diffusive, which possibly limits the tempo of processes. We studied the dependence of protein mobility on protein surface properties and ionic strength. We used surface-modified fluorescent proteins (FPs) and determined their translational diffusion coefficients (<br/>
</details><br/>
---

*Characterization of a pentonolactonase involved in D-xylose and L-arabinose catabolism in the haloarchaeon Haloferax volcanii.*<br/>
Sutter JM; Johnsen U; Schönheit P<br/>
**2017**<br/>
FEMS Microbiol. Lett. 364(13)<br/>
femsle/fnx140
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28854683)<br/>
[Citation]()<br/>
Haloferax volcanii degrades the pentoses D-xylose and L-arabinose via an oxidative pathway to α-ketoglutarate as an intermediate. The initial dehydrogenases of the pathway, D-xylose dehydrogenase (XDH) and L-arabinose dehydrogenase (L-AraDH) catalyze the NADP+ dependent D-xylose and L-arabinose oxidation. It is shown here that the pentoses are oxidized to the corresponding lactones, D-xylono-γ-lactone and L-arabino-γ-lactone, rather than to the respective sugar acids. A putative lactonase gene, xacC, located in genomic vicinity of XDH and L-AraDH, was found to be transcriptionally upregulated by both D-xylose and L-arabinose mediated by the pentose-specific regulator XacR. The recombinant lactonase catalyzed the hydrolysis of D-xylono-γ-lactone and L-arabino-γ-lactone. This is the first report of a functional lactonase involved in sugar catabolism in the domain of archaea.<br/>
</details><br/>
---

*High tolerance to self-targeting of the genome by the endogenous CRISPR-Cas system in an archaeon.*<br/>
Stachler AE; Turgeman-Grott I; Shtifman-Segal E; Allers T; Marchfelder A; Gophna U<br/>
**2017**<br/>
Nucleic Acids Res. 45(9)<br/>
nar/gkx150
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28334774)<br/>
[Citation]()<br/>
CRISPR-Cas systems allow bacteria and archaea to acquire sequence-specific immunity against selfish genetic elements such as viruses and plasmids, by specific degradation of invader DNA or RNA. However, this involves the risk of autoimmunity if immune memory against host DNA is mistakenly acquired. Such autoimmunity has been shown to be highly toxic in several bacteria and is believed to be one of the major costs of maintaining these defense systems. Here we generated an experimental system in which a non-essential gene, required for pigment production and the reddish colony color, is targeted by the CRISPR-Cas I-B system of the halophilic archaeon Haloferax volcanii. We show that under native conditions, where both the self-targeting and native crRNAs are expressed, self-targeting by CRISPR-Cas causes no reduction in transformation efficiency of the plasmid encoding the self-targeting crRNA. Furthermore, under such conditions, no effect on organismal growth rate or loss of the reddish colony phenotype due to mutations in the targeted region could be observed. In contrast, in cells deleted for the pre-crRNA processing gene cas6, where only the self-targeting crRNA exists as mature crRNA, self-targeting leads to moderate toxicity and the emergence of deletion mutants. Sequencing of the deletions caused by CRISPR-Cas self targeting indicated DNA repair via microhomology-mediated end joining.<br/>
</details><br/>
---

*Co-expression and purification of the RadA recombinase with the RadB paralog from Haloferax volcanii yields heteromeric ring-like structures.*<br/>
Patoli BB; Winter JA; Patoli AA; Delahay RM; Bunting KA<br/>
**2017**<br/>
Microbiology (Reading, Engl.) 163(12)<br/>
10.1099/mic.0.000562
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29072558)<br/>
[Citation]()<br/>
The study of archaeal proteins and the processes to which they contribute poses particular challenges due to the often extreme environments in which they function. DNA recombination, replication and repair proteins of the halophilic euryarchaeon, Haloferax volcanii (Hvo) are of particular interest as they tend to resemble eukaryotic counterparts in both structure and activity, and genetic tools are available to facilitate their analysis. In the present study, we show using bioinformatics approaches that the Hvo RecA-like protein RadA is structurally similar to other recombinases although is distinguished by a unique acidic insertion loop. To facilitate expression of Hvo RadA a co-expression approach was used, providing its lone paralog, RadB, as a binding partner. At present, structural and biochemical characterization of Hvo RadA is lacking. Here, we describe for the first time co-expression of Hvo RadA with RadB and purification of these proteins as a complex under in vitro conditions. Purification procedures were performed under high salt concentration (>1 M sodium chloride) to maintain the solubility of the proteins. Quantitative densitometry analysis of the co-expressed and co-purified RadAB complex estimated the ratio of RadA to RadB to be 4 : 1, which suggests that the proteins interact with a specific stoichiometry. Based on a combination of analyses, including size exclusion chromatography, Western blot and electron microscopy observations, we suggest that RadA multimerizes into a ring-like structure in the absence of DNA and nucleoside co-factor.<br/>
</details><br/>
---

*Opsin-Mediated Inhibition of Bacterioruberin Synthesis in Halophilic Archaea.*<br/>
Peck RF; Pleşa AM; Graham SM; Angelini DR; Shaw EL<br/>
**2017**<br/>
J. Bacteriol. 199(21)<br/>
10.1128/JB.00303-17
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28784816)<br/>
[Citation]()<br/>
Halophilic archaea often inhabit environments with limited oxygen, and many produce ion-pumping rhodopsin complexes that allow them to maintain electrochemical gradients when aerobic respiration is inhibited. Rhodopsins require a protein, an opsin, and an organic cofactor, retinal. We previously demonstrated that in <br/>
</details><br/>
---

*Archaeal Persisters: Persister Cell Formation as a Stress Response in Haloferax volcanii*<br/>
Megaw, Julianne; Gilmore, Brendan F.<br/>
**2017**<br/>
Frontiers in Microbiology 8()<br/>
10.3389/fmicb.2017.01589
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://www.frontiersin.org/article/10.3389/fmicb.2017.01589)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Megaw2017FrontMicrobiol.ris)<br/>
Persister cells are phenotypic variants within a microbial population, which are dormant and transiently tolerant to stress. Persistence has been studied extensively in bacteria, and in eukaryotes to a limited extent, however, it has never been observed in archaea. Using the model haloarchaeon, Haloferax volcanii DS2, we demonstrated persister cell formation in this domain, with time-kill curves exhibiting a characteristic biphasic pattern following starvation or exposure to lethal concentrations of various biocidal compounds. Repeated challenges of surviving cells showed that, as with bacteria, persister formation in H. volcanii was not heritable. Additionally, as previously shown with bacteria, persister formation in H. volcanii was suppressed by exogenous indole. The addition of spent culture media to assays conducted on planktonic cells showed that H. volcanii-conditioned media stimulated persistence, whereas conditioned media of other haloarchaea or halophilic bacteria did not, suggesting the involvement of a species-specific signal. Using a TLC overlay assay, the quorum sensing bioreporter Agrobacterium tumefaciens ATCC BAA-2240 detected the presence of C<sub>4</sub> and C<sub>6</sub> acyl homoserine lactone-like signal molecules in a H. volcanii culture extract. While synthetic bacterial AHLs did not induce persistence, this is potentially due to structural differences between bacterial and archaeal signals, and does not discount a quorum sensing component in haloarchaeal persister formation. The observation of persister cell formation by this haloarchaeon may provide some insights into the survival of these organisms in stressful or dynamic environments.<br/>
</details><br/>
---

*Growth phenotype analysis of heme synthetic enzymes in a halophilic archaeon, Haloferax volcanii.*<br/>
Kosugi N; Araki T; Fujita J; Tanaka S; Fujiwara T<br/>
**2017**<br/>
PLoS ONE 12(12)<br/>
10.1371/journal.pone.0189913
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29284023)<br/>
[Citation]()<br/>
Halophilic euryarchaea lack many of the genes necessary for the protoporphyrin-dependent heme biosynthesis pathway previously identified in animals and plants. Bioinformatic analysis suggested the presence of two heme biosynthetic processes, an Fe-coproporphyrinogen III (coproheme) decarboxylase (ChdC) pathway and an alternative heme biosynthesis (Ahb) pathway, in Haloferax volcanii. PitA is specific to the halophilic archaea and has a unique molecular structure in which the ChdC domain is joined to the antibiotics biosynthesis monooxygenase (ABM)-like domain by a histidine-rich linker sequence. The pitA gene deletion variant of H. volcanii showed a phenotype with a significant reduction of aerobic growth. Addition of a protoheme complemented the phenotype, supporting the assumption that PitA participates in the aerobic heme biosynthesis. Deletion of the ahbD gene caused a significant reduction of only anaerobic growth by denitrification or dimethylsulfoxide (DMSO) respiration, and the growth was also complemented by addition of a protoheme. The experimental results suggest that the two heme biosynthesis pathways are utilized selectively under aerobic and anaerobic conditions in H. volcanii. The molecular structure and physiological function of PitA are also discussed on the basis of the limited proteolysis and sequence analysis.<br/>
</details><br/>
---

*Mutations in the major gas vesicle protein GvpA and impacts on gas vesicle formation in Haloferax volcanii.*<br/>
Knitsch R; Schneefeld M; Weitzel K; Pfeifer F<br/>
**2017**<br/>
Mol. Microbiol. 106(4)<br/>
10.1111/mmi.13833
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28898511)<br/>
[Citation]()<br/>
Gas vesicles are proteinaceous, gas-filled nanostructures produced by some bacteria and archaea. The hydrophobic major structural protein GvpA forms the ribbed gas vesicle wall. An in-silico 3D-model of GvpA of the predicted coil-α1-β1-β2-α2-coil structure is available and implies that the two β-chains constitute the hydrophobic interior surface of the gas vesicle wall. To test the importance of individual amino acids in GvpA we performed 85 single substitutions and analyzed these variants in Haloferax volcanii ΔA + A<br/>
</details><br/>
---

*Toward Time-Resolved Analysis of RNA Metabolism in Archaea Using 4-Thiouracil.*<br/>
Knüppel R; Kuttenberger C; Ferreira-Cerca S<br/>
**2017**<br/>
Front Microbiol 8(None)<br/>
10.3389/fmicb.2017.00286
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28286499)<br/>
[Citation]()<br/>
Archaea are widespread organisms colonizing almost every habitat on Earth. However, the molecular biology of archaea still remains relatively uncharacterized. RNA metabolism is a central cellular process, which has been extensively analyzed in both bacteria and eukarya. In contrast, analysis of RNA metabolism dynamic in archaea has been limited to date. To facilitate analysis of the RNA metabolism dynamic at a system-wide scale in archaea, we have established non-radioactive pulse labeling of RNA, using the nucleotide analog 4-thiouracil (4TU) in two commonly used model archaea: the halophile Euryarchaeota <br/>
</details><br/>
---

*Transposon Insertion Mutagenesis for Archaeal Gene Discovery.*<br/>
Kiljunen S; Pajunen MI; Savilahti H<br/>
**2017**<br/>
Methods Mol. Biol. 1498(None)<br/>
10.1007/978-1-4939-6472-7_20
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27709584)<br/>
[Citation]()<br/>
Archaea constitute the third domain of life, but studies on their physiology and other features have lagged behind bacteria and eukarya, largely due to the challenging biology of archaea and concomitant difficulties in methods development. The use of genome-wide en masse insertion mutagenesis is one of the most efficient means to discover the genes behind various biological functions, and such a methodology is described in this chapter for a model archaeon Haloferax volcanii. The strategy successfully employs efficient in vitro transposition in combination with gene targeting in vivo via homologous recombination. The methodology is general and should be transferable to other archaeal species.<br/>
</details><br/>
---

*ThiN as a Versatile Domain of Transcriptional Repressors and Catalytic Enzymes of Thiamine Biosynthesis.*<br/>
Hwang S; Cordova B; Abdo M; Pfeiffer F; Maupin-Furlow JA<br/>
**2017**<br/>
J. Bacteriol. 199(7)<br/>
10.1128/JB.00810-16
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28115546)<br/>
[Citation]()<br/>
Thiamine biosynthesis is commonly regulated by a riboswitch mechanism; however, the enzymatic steps and regulation of this pathway in archaea are poorly understood. <br/>
</details><br/>
---

*A tRNA-derived fragment competes with mRNA for ribosome binding and regulates translation during stress.*<br/>
Gebetsberger J; Wyss L; Mleczko AM; Reuther J; Polacek N<br/>
**2017**<br/>
RNA Biol 14(10)<br/>
10.1080/15476286.2016.1257470
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27892771)<br/>
[Citation]()<br/>
Posttranscriptional processing of RNA molecules is a common strategy to enlarge the structural and functional repertoire of RNomes observed in all 3 domains of life. Fragmentation of RNA molecules of basically all functional classes has been reported to yield smaller non-protein coding RNAs (ncRNAs) that typically possess different roles compared with their parental transcripts. Here we show that a valine tRNA-derived fragment (Val-tRF) that is produced under certain stress conditions in the halophilic archaeon Haloferax volcanii is capable of binding to the small ribosomal subunit. As a consequence of Val-tRF binding mRNA is displaced from the initiation complex which results in global translation attenuation in vivo and in vitro. The fact that the archaeal Val-tRF also inhibits eukaryal as well as bacterial protein biosynthesis implies a functionally conserved mode of action. While tRFs and tRNA halves have been amply identified in recent RNA-seq project, Val-tRF described herein represents one of the first functionally characterized tRNA processing products to date.<br/>
</details><br/>
---

*Assembling Glycan-Charged Dolichol Phosphates: Chemoenzymatic Synthesis of a Haloferax volcanii N-Glycosylation Pathway Intermediate.*<br/>
Elharar Y; Podilapu AR; Guan Z; Kulkarni SS; Eichler J<br/>
**2017**<br/>
Bioconjug. Chem. 28(9)<br/>
10.1021/acs.bioconjchem.7b00436
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28809486)<br/>
[Citation]()<br/>
N-glycosylation, the covalent attachment of glycans to select protein target Asn residues, is a post-translational modification performed by all three domains of life. In the halophilic archaea Haloferax volcanii, in which understanding of this universal protein-processing event is relatively well-advanced, genes encoding the components of the archaeal glycosylation (Agl) pathway responsible for the assembly and attachment of an N-linked pentasaccharide have been identified. As elsewhere, the N-linked glycan is assembled on phosphodolichol carriers before transfer to target Asn residues. However, as little is presently known of the Hfx. volcanii Agl pathway at the protein level, the seemingly unique ability of Archaea to use dolichol phosphate (DolP) as the glycan lipid carrier, rather than dolichol pyrophosphate used by eukaryotes, remains poorly understood. With this in mind, a chemoenzymatic approach was taken to biochemically study AglG, one of the five glycosyltransferases of the pathway. Accordingly, a novel regio- and stereoselective reduction of naturally isolated polyprenol gave facile access to S-dolichol via asymmetric transfer hydrogenation under very mild conditions. This compound was used to generate glucose-charged DolP, a precursor of the N-linked pentasaccharide, as well as DolP-glucose-glucuronic acid and DolP-glucuronic acid. AglG, purified from Hfx. volcanii membranes in hypersaline conditions, like those encountered in situ, was subsequently combined with uridine diphosphate (UDP)-glucuronic acid and DolP-glucose to yield DolP-glucose-glucuronic acid. The in vitro system for the study of AglG activity developed here represents the first such tool for studying halophilic glycosyltransferases and will allow for a detailed understanding of archaeal N-glycosylation.<br/>
</details><br/>
---

*Engineering substrate promiscuity in halophilic alcohol dehydrogenase (HvADH2) by in silico design.*<br/>
Cassidy J; Bruen L; Rosini E; Molla G; Pollegioni L; Paradisi F<br/>
**2017**<br/>
PLoS ONE 12(11)<br/>
10.1371/journal.pone.0187482
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29190711)<br/>
[Citation]()<br/>
An alcohol dehydrogenase from the halophilic archaeon Haloferax volcanii (HvADH2) has been engineered by rational design to broaden its substrate scope towards the conversion of a range of aromatic substrates, including flurbiprofenol, that is an intermediate of the non-steroidal anti-inflammatory drug, flurbiprofen. Wild-type HvADH2 showed minimal activity with flurbiprofenol (11.1 mU/mg). A homology model of HvADH2 was built and docking experiments with this substrate revealed that the biphenyl rings of flurbiprofenol formed strong interactions with residues F85 and F108, preventing its optimal binding in the active site. Mutations at position 85 however did not increase activity. Site directed mutagenesis at position F108 allowed the identification of three variants showing a significant (up to 2.3-fold) enhancement of activity towards flurbiprofenol, when compared to wild-type HvADH2. Interestingly, F108G variant did not show the classic inhibition in the presence of (R)-enantiomer when tested with rac-1-phenylethanol, underling its potential in racemic resolution of secondary alcohols.<br/>
</details><br/>
---

*Succinyl-CoA:Mesaconate CoA-Transferase and Mesaconyl-CoA Hydratase, Enzymes of the Methylaspartate Cycle in Haloarcula hispanica*<br/>
Borjian, Farshad; Johnsen, Ulrike; SchÃ¶nheit, Peter; Berg, Ivan A.<br/>
**2017**<br/>
Frontiers in Microbiology 8()<br/>
10.3389/fmicb.2017.01683
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://www.frontiersin.org/article/10.3389/fmicb.2017.01683)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Borjian2017FrontMicrobiol.ris)<br/>
Growth on acetate or other acetyl-CoA-generating substrates as a sole source of carbon requires an anaplerotic pathway for the conversion of acetyl-CoA into cellular building blocks. Haloarchaea (class Halobacteria) possess two different anaplerotic pathways, the classical glyoxylate cycle and the novel methylaspartate cycle. The methylaspartate cycle was discovered in Haloarcula spp. and operates in âˆ¼40% of sequenced haloarchaea. In this cycle, condensation of one molecule of acetyl-CoA with oxaloacetate gives rise to citrate, which is further converted to 2-oxoglutarate and then to glutamate. The following glutamate rearrangement and deamination lead to mesaconate (methylfumarate) that needs to be activated to mesaconyl-C1-CoA and hydrated to Î²-methylmalyl-CoA. The cleavage of Î²-methylmalyl-CoA results in the formation of propionyl-CoA and glyoxylate. The carboxylation of propionyl-CoA and the condensation of glyoxylate with another acetyl-CoA molecule give rise to two C<sub>4</sub>-dicarboxylic acids, thus regenerating the initial acetyl-CoA acceptor and forming malate, its final product. Here we studied two enzymes of the methylaspartate cycle from Haloarcula hispanica, succinyl-CoA:mesaconate CoA-transferase (mesaconate CoA-transferase, Hah_1336) and mesaconyl-CoA hydratase (Hah_1340). Their genes were heterologously expressed in Haloferax volcanii, and the corresponding enzymes were purified and characterized. Mesaconate CoA-transferase was specific for its physiological substrates, mesaconate and succinyl-CoA, and produced only mesaconyl-C1-CoA and no mesaconyl-C4-CoA. Mesaconyl-CoA hydratase had a 3.5-fold bias for the physiological substrate, mesaconyl-C1-CoA, compared to mesaconyl-C4-CoA, and virtually no activity with other tested enoyl-CoA/3-hydroxyacyl-CoA compounds. Our results further prove the functioning of the methylaspartate cycle in haloarchaea and suggest that mesaconate CoA-transferase and mesaconyl-CoA hydratase can be regarded as characteristic enzymes of this cycle.<br/>
</details><br/>
---

*Extreme halophilic alcohol dehydrogenase mediated highly efficient syntheses of enantiopure aromatic alcohols.*<br/>
Alsafadi D; Alsalman S; Paradisi F<br/>
**2017**<br/>
Org. Biomol. Chem. 15(43)<br/>
10.1039/c7ob02299a
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/29067382)<br/>
[Citation]()<br/>
Enzymatic synthesis of enantiopure aromatic secondary alcohols (including substituted, hetero-aromatic and bicyclic structures) was carried out using halophilic alcohol dehydrogenase ADH2 from Haloferax volcanii (HvADH2). This enzyme showed an unprecedented substrate scope and absolute enatioselectivity. The cofactor NADPH was used catalytically and regenerated in situ by the biocatalyst, in the presence of 5% ethanol. The efficiency of HvADH2 for the conversion of aromatic ketones was markedly influenced by the steric and electronic factors as well as the solubility of ketones in the reaction medium. Furthermore, carbonyl stretching band frequencies ν (C[double bond, length as m-dash]O) have been measured for different ketones to understand the effect of electron withdrawing or donating properties of the ketone substituents on the reaction rate catalyzed by HvADH2. Good correlation was observed between ν (C[double bond, length as m-dash]O) of methyl aryl-ketones and the reaction rate catalyzed by HvADH2. The enzyme catalyzed the reductions of ketone substrates on the preparative scale, demonstrating that HvADH2 would be a valuable biocatalyst for the preparation of chiral aromatic alcohols of pharmaceutical interest.<br/>
</details><br/>
---

*Diversity of DNA Replication in the Archaea.*<br/>
Ausiannikava D; Allers T<br/>
**2017**<br/>
Genes (Basel) 8(2)<br/>
10.3390/genes8020056
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28146124)<br/>
[Citation]()<br/>
DNA replication is arguably the most fundamental biological process. On account of their shared evolutionary ancestry, the replication machinery found in archaea is similar to that found in eukaryotes. DNA replication is initiated at origins and is highly conserved in eukaryotes, but our limited understanding of archaea has uncovered a wide diversity of replication initiation mechanisms. Archaeal origins are sequence-based, as in bacteria, but are bound by initiator proteins that share homology with the eukaryotic origin recognition complex subunit Orc1 and helicase loader Cdc6). Unlike bacteria, archaea may have multiple origins per chromosome and multiple Orc1/Cdc6 initiator proteins. There is no consensus on how these archaeal origins are recognised- some are bound by a single Orc1/Cdc6 protein while others require a multi- Orc1/Cdc6 complex. Many archaeal genomes consist of multiple parts-the main chromosome plus several megaplasmids-and in polyploid species these parts are present in multiple copies. This poses a challenge to the regulation of DNA replication. However, one archaeal species (Haloferax volcanii) can survive without replication origins; instead, it uses homologous recombination as an alternative mechanism of initiation. This diversity in DNA replication initiation is all the more remarkable for having been discovered in only three groups of archaea where in vivo studies are possible.<br/>
</details><br/>
---

*ArtA-Dependent Processing of a Tat Substrate Containing a Conserved Tripartite Structure That Is Not Localized at the C Terminus.*<br/>
Abdul Halim MF; Stoltzfus JD; Schulze S; Hippler M; Pohlschroder M<br/>
**2017**<br/>
J. Bacteriol. 199(7)<br/>
10.1128/JB.00802-16
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28069824)<br/>
[Citation]()<br/>
Most prokaryote-secreted proteins are transported to the cell surface using either the general secretion (Sec) or twin-arginine translocation (Tat) pathway. A majority of secreted proteins are anchored to the cell surface, while the remainder are released into the extracellular environment. The anchored surface proteins play a variety of important roles in cellular processes, ranging from facilitating interactions between cells to maintaining cell stability. The extensively studied S-layer glycoprotein (SLG) of <br/>
</details><br/>
---

*Key Enzymes of the Semiphosphorylative Entner-Doudoroff Pathway in the Haloarchaeon Haloferax volcanii: Characterization of Glucose Dehydrogenase, Gluconate Dehydratase, and 2-Keto-3-Deoxy-6-Phosphogluconate Aldolase.*<br/>
Sutter JM; Tästensen JB; Johnsen U; Soppa J; Schönheit P<br/>
**2016**<br/>
J. Bacteriol. 198(16)<br/>
10.1128/JB.00286-16
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27297879)<br/>
[Citation]()<br/>
UNLABELLED: The halophilic archaeon Haloferax volcanii has been proposed to degrade glucose via the semiphosphorylative Entner-Doudoroff (spED) pathway. So far, the key enzymes of this pathway, glucose dehydrogenase (GDH), gluconate dehydratase (GAD), and 2-keto-3-deoxy-6-phosphogluconate (KDPG) aldolase (KDPGA), have not been characterized, and their functional involvement in glucose degradation has not been demonstrated. Here we report that the genes HVO_1083 and HVO_0950 encode GDH and KDPGA, respectively. The recombinant enzymes show high specificity for glucose and KDPG and did not convert the corresponding C4 epimers galactose and 2-keto-3-deoxy-6-phosphogalactonate at significant rates. Growth studies of knockout mutants indicate the functional involvement of both GDH and KDPGA in glucose degradation. GAD was purified from H. volcanii, and the encoding gene, gad, was identified as HVO_1488. GAD catalyzed the specific dehydration of gluconate and did not utilize galactonate at significant rates. A knockout mutant of GAD lost the ability to grow on glucose, indicating the essential involvement of GAD in glucose degradation. However, following a prolonged incubation period, growth of the Δgad mutant on glucose was recovered. Evidence is presented that under these conditions, GAD was functionally replaced by xylonate dehydratase (XAD), which uses both xylonate and gluconate as substrates. Together, the characterization of key enzymes and analyses of the respective knockout mutants present conclusive evidence for the in vivo operation of the spED pathway for glucose degradation in H. volcanii
IMPORTANCE: The work presented here describes the identification and characterization of the key enzymes glucose dehydrogenase, gluconate dehydratase, and 2-keto-3-deoxy-6-phosphogluconate aldolase and their encoding genes of the proposed semiphosphorylative Entner-Doudoroff pathway in the haloarchaeon Haloferax volcanii The functional involvement of the three enzymes was proven by analyses of the corresponding knockout mutants. These results provide evidence for the in vivo operation of the semiphosphorylative Entner-Doudoroff pathway in haloarchaea and thus expand our understanding of the unusual sugar degradation pathways in the domain Archaea.<br/>
</details><br/>
---

*Gene Repression in Haloarchaea Using the CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats)-Cas I-B System.*<br/>
Stachler AE; Marchfelder A<br/>
**2016**<br/>
J. Biol. Chem. 291(29)<br/>
10.1074/jbc.M116.724062
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27226589)<br/>
[Citation]()<br/>
The clustered regularly interspaced short palindromic repeats (CRISPR)-Cas system is used by bacteria and archaea to fend off foreign genetic elements. Since its discovery it has been developed into numerous applications like genome editing and regulation of transcription in eukaryotes and bacteria. For archaea currently no tools for transcriptional repression exist. Because molecular biology analyses in archaea become more and more widespread such a tool is vital for investigating the biological function of essential genes in archaea. Here we use the model archaeon Haloferax volcanii to demonstrate that its endogenous CRISPR-Cas system I-B can be harnessed to repress gene expression in archaea. Deletion of cas3 and cas6b genes results in efficient repression of transcription. crRNAs targeting the promoter region reduced transcript levels down to 8%. crRNAs targeting the reading frame have only slight impact on transcription. crRNAs that target the coding strand repress expression only down to 88%, whereas crRNAs targeting the template strand repress expression down to 8%. Repression of an essential gene results in reduction of transcription levels down to 22%. Targeting efficiencies can be enhanced by expressing a catalytically inactive Cas3 mutant. Genes can be targeted on plasmids or on the chromosome, they can be monocistronic or part of a polycistronic operon.<br/>
</details><br/>
---

*Transcription-coupled repair of UV damage in the halophilic archaea.*<br/>
Stantial N; Dumpe J; Pietrosimone K; Baltazar F; Crowley DJ<br/>
**2016**<br/>
DNA Repair (Amst.) 41(None)<br/>
10.1016/j.dnarep.2016.03.007
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27088618)<br/>
[Citation]()<br/>
Transcription-coupled repair (TCR) is a subpathway of nucleotide excision repair (NER) in which excision repair proteins are targeted to RNA polymerase-arresting lesions located in the transcribed strand of active genes. TCR has been documented in a variety of bacterial and eukaryotic organisms but has yet to be observed in the Archaea. We used Halobacterium sp. NRC-1 and Haloferax volcanii to determine if TCR occurs in the halophilic archaea. Following UV irradiation of exponentially growing cultures, we quantified the rate of repair of cyclobutane pyrimidine dimers in the two strands of the rpoB2B1A1A2 and the trpDFEG operons of Halobacterium sp. NRC-1 and the pts operon of H. volcanii through the use of a Southern blot assay and strand-specific probes. TCR was observed in all three operons and was dependent on the NER gene uvrA in Halobacterium sp. NRC-1, but not in H. volcanii. The halophilic archaea likely employ a novel mechanism for TCR in which an as yet unknown coupling factor recognizes the arrested archaeal RNA polymerase complex and recruits certain NER proteins to complete the process.<br/>
</details><br/>
---

*Production of halophilic proteins using Haloferax volcanii H1895 in a stirred-tank bioreactor.*<br/>
Strillinger E; Grötzinger SW; Allers T; Eppinger J; Weuster-Botz D<br/>
**2016**<br/>
Appl. Microbiol. Biotechnol. 100(3)<br/>
10.1007/s00253-015-7007-1
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26428236)<br/>
[Citation]()<br/>
The success of biotechnological processes is based on the availability of efficient and highly specific biocatalysts, which can satisfy industrial demands. Extreme and remote environments like the deep brine pools of the Red Sea represent highly interesting habitats for the discovery of novel halophilic and thermophilic enzymes. Haloferax volcanii constitutes a suitable expression system for halophilic enzymes obtained from such brine pools. We developed a batch process for the cultivation of H. volcanii H1895 in controlled stirred-tank bioreactors utilising knockouts of components of the flagella assembly system. The standard medium Hv-YPC was supplemented to reach a higher cell density. Without protein expression, cell dry weight reaches 10 g L(-1). Two halophilic alcohol dehydrogenases were expressed under the control of the tryptophanase promoter p.tna with 16.8 and 3.2 mg gCDW (-1), respectively, at a maximum cell dry weight of 6.5 g L(-1). Protein expression was induced by the addition of L-tryptophan. Investigation of various expression strategies leads to an optimised two-step induction protocol introducing 6 mM L-tryptophan at an OD650 of 0.4 followed by incubation for 16 h and a second induction step with 3 mM L-tryptophan followed by a final incubation time of 4 h. Compared with the uncontrolled shaker-flask cultivations used until date, dry cell mass concentrations were improved by a factor of more than 5 and cell-specific enzyme activities showed an up to 28-fold increased yield of the heterologous proteins. <br/>
</details><br/>
---

*Transcriptional regulation of dimethyl sulfoxide respiration in a haloarchaeon, Haloferax volcanii.*<br/>
Qi Q; Ito Y; Yoshimatsu K; Fujiwara T<br/>
**2016**<br/>
Extremophiles 20(1)<br/>
10.1007/s00792-015-0794-6
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26507955)<br/>
[Citation]()<br/>
The halophilic euryarchaeon Haloferax volcanii can grow anaerobically by DMSO respiration. DMSO reductase was induced by DMSO respiration not only under anaerobic growth conditions but also in denitrifying cells of H. volcanii. Deletion of the dmsR gene, encoding a putative regulator for the DMSO reductase, resulted in the loss of anaerobic growth by DMSO respiration. Reporter experiments revealed that only the anaerobic condition was essential for transcription of the dmsEABCD genes encoding DMSO reductase and that transcription was enhanced threefold by supplementation of DMSO. In the ∆dmsR mutant, transcription of the dmsEABCD genes induced by the anaerobic condition was not enhanced by DMSO, suggesting that DmsR is a DMSO-responsive regulator. Transcriptions of the dmsR and mgd genes for Mo-bisMGD biosynthesis were regulated in the same manner as the dmsEABCD genes. These results suggest that the genetic regulation of DMSO respiration in H. volcanii is controlled by at least two systems: one is the DMSO-responsive DmsR, and the other is an unknown anaerobic regulator. <br/>
</details><br/>
---

*Deciphering the Translation Initiation Factor 5A Modification Pathway in Halophilic Archaea.*<br/>
Prunetti L; Graf M; Blaby IK; Peil L; Makkay AM; Starosta AL; Papke RT; Oshima T; Wilson DN; de Crécy-Lagard V<br/>
**2016**<br/>
Archaea 2016(None)<br/>
2016/7316725
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/28053595)<br/>
[Citation]()<br/>
Translation initiation factor 5A (IF5A) is essential and highly conserved in Eukarya (eIF5A) and Archaea (aIF5A). The activity of IF5A requires hypusine, a posttranslational modification synthesized in Eukarya from the polyamine precursor spermidine. Intracellular polyamine analyses revealed that agmatine and cadaverine were the main polyamines produced in <br/>
</details><br/>
---

*Impact of a homing intein on recombination frequency and organismal fitness.*<br/>
Naor A; Altman-Price N; Soucy SM; Green AG; Mitiagin Y; Turgeman-Grott I; Davidovich N; Gogarten JP; Gophna U<br/>
**2016**<br/>
Proc. Natl. Acad. Sci. U.S.A. 113(32)<br/>
10.1073/pnas.1606416113
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27462108)<br/>
[Citation]()<br/>
Inteins are parasitic genetic elements that excise themselves at the protein level by self-splicing, allowing the formation of functional, nondisrupted proteins. Many inteins contain a homing endonuclease (HEN) domain and rely on its activity for horizontal propagation. However, successful invasion of an entire population will make this activity redundant, and the HEN domain is expected to degenerate quickly under these conditions. Several theories have been proposed for the continued existence of the both active HEN and noninvaded alleles within a population. However, to date, these models were not directly tested experimentally. Using the natural cell fusion ability of the halophilic archaeon Haloferax volcanii we were able to examine this question in vivo, by mating polB intein-positive [insertion site c in the gene encoding DNA polymerase B (polB-c)] and intein-negative cells and examining the dispersal efficiency of this intein in a natural, polyploid population. Through competition between otherwise isogenic intein-positive and intein-negative strains we determined a surprisingly high fitness cost of over 7% for the polB-c intein. Our laboratory culture experiments and samples taken from Israel's Mediterranean coastline show that the polB-c inteins do not efficiently take over an inteinless population through mating, even under ideal conditions. The presence of the HEN/intein promoted recombination when intein-positive and intein-negative cells were mated. Increased recombination due to HEN activity contributes not only to intein dissemination but also to variation at the population level because recombination tracts during repair extend substantially from the homing site.<br/>
</details><br/>
---

*Structure-function relationships of archaeal Cbf5 during in vivo RNA-guided pseudouridylation.*<br/>
Majumder M; Bosmeny MS; Gupta R<br/>
**2016**<br/>
RNA 22(10)<br/>
10.1261/rna.057547.116
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27539785)<br/>
[Citation]()<br/>
In Eukarya and Archaea, in addition to protein-only pseudouridine (Ψ) synthases, complexes containing one guide RNA and four proteins can also produce Ψ. Cbf5 protein is the Ψ synthase in the complex. Previously, we showed that Ψ's at positions 1940, 1942, and 2605 of Haloferax volcanii 23S rRNA are absent in a cbf5-deleted strain, and a plasmid-borne copy of cbf5 can rescue the synthesis of these Ψ's. Based on published reports of the structure of archaeal Cbf5 complexed with other proteins and RNAs, we identified several potential residues and structures in H. volcanii Cbf5, which were expected to play important roles in pseudouridylation. We mutated these structures and determined their effects on Ψ production at the three rRNA positions under in vivo conditions. Mutations of several residues in the catalytic domain and certain residues in the thumb loop either abolished Ψ's or produced partial modification; the latter indicates a slower rate of Ψ formation. The universal catalytic aspartate of Ψ synthases could be replaced by glutamate in Cbf5. A conserved histidine, which is common to Cbf5 and TruB is not needed, but another conserved histidine of Cbf5 is required for the in vivo RNA-guided Ψ formation. We also identified a previously unreported novelty in the pseudouridylation activity of Cbf5 where a single stem-loop of a guide H/ACA RNA is used to produce two closely placed Ψ's and mutations of certain residues of Cbf5 abolished one of these two Ψ's. In summary, this first in vivo study identifies several structures of an archaeal Cbf5 protein that are important for its RNA-guided pseudouridylation activity.<br/>
</details><br/>
---

*Evolutionary consequences of polyploidy in prokaryotes and the origin of mitosis and meiosis.*<br/>
Markov AV; Kaznacheev IS<br/>
**2016**<br/>
Biol. Direct 11(None)<br/>
10.1186/s13062-016-0131-8
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27277956)<br/>
[Citation]()<br/>
BACKGROUND: The origin of eukaryote-specific traits such as mitosis and sexual reproduction remains disputable. There is growing evidence that both mitosis and eukaryotic sex (i.e., the alternation of syngamy and meiosis) may have already existed in the basal eukaryotes. The mating system of the halophilic archaeon Haloferax volcanii probably represents an intermediate stage between typical prokaryotic and eukaryotic sex. H. volcanii is highly polyploid, as well as many other Archaea. Here, we use computer simulation to explore genetic and evolutionary outcomes of polyploidy in amitotic prokaryotes and its possible role in the origin of mitosis, meiosis and eukaryotic sex.
RESULTS: Modeling suggests that polyploidy can confer strong short-term evolutionary advantage to amitotic prokaryotes. However, it also promotes the accumulation of recessive deleterious mutations and the risk of extinction in the long term, especially in highly mutagenic environment. There are several possible strategies that amitotic polyploids can use in order to reduce the genetic costs of polyploidy while retaining its benefits. Interestingly, most of these strategies resemble different components or aspects of eukaryotic sex. They include asexual ploidy cycles, equalization of genome copies by gene conversion, high-frequency lateral gene transfer between relatives, chromosome exchange coupled with homologous recombination, and the evolution of more accurate chromosome distribution during cell division (mitosis). Acquisition of mitosis by an amitotic polyploid results in chromosome diversification and specialization. Ultimately, it transforms a polyploid cell into a functionally monoploid one with multiple unique, highly redundant chromosomes. Specialization of chromosomes makes the previously evolved modes of promiscuous chromosome shuffling deleterious. This can result in selective pressure to develop accurate mechanisms of homolog pairing, and, ultimately, meiosis.
CONCLUSION: Emergence of mitosis and the first evolutionary steps towards eukaryotic sex could have taken place in the ancestral polyploid, amitotic proto-eukaryotes, as they were struggling to survive in the highly mutagenic environment of the Early Proterozoic shallow water microbial communities, through the succession of the following stages: (1) acquisition of high-frequency between-individual genetic exchange coupled with homologous recombination; (2) acquisition of mitosis, followed by rapid chromosome diversification and specialization; (3) evolution of homolog synapsis and meiosis. Additional evidence compatible with this scenario includes mass acquisition of new families of paralogous genes by the basal eukaryotes, and recently discovered correlation between polyploidy and the presence of histones in Archaea.
REVIEWER: This article was reviewed by Eugene Koonin, Uri Gophna and Armen Mulkidjanian. For the full reviews, please go to the Reviewers' comments section.<br/>
</details><br/>
---

*Archaeal Inorganic Pyrophosphatase Displays Robust Activity under High-Salt Conditions and in Organic Solvents.*<br/>
McMillan LJ; Hepowit NL; Maupin-Furlow JA<br/>
**2016**<br/>
Appl. Environ. Microbiol. 82(2)<br/>
10.1128/AEM.03055-15
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26546423)<br/>
[Citation]()<br/>
Soluble inorganic pyrophosphatases (PPAs) that hydrolyze inorganic pyrophosphate (PPi) to orthophosphate (Pi) are commonly used to accelerate and detect biosynthetic reactions that generate PPi as a by-product. Current PPAs are inactivated by high salt concentrations and organic solvents, which limits the extent of their use. Here we report a class A type PPA of the haloarchaeon Haloferax volcanii (HvPPA) that is thermostable and displays robust PPi-hydrolyzing activity under conditions of 25% (vol/vol) organic solvent and salt concentrations from 25 mM to 3 M. HvPPA was purified to homogeneity as a homohexamer by a rapid two-step method and was found to display non-Michaelis-Menten kinetics with a Vmax of 465 U · mg(-1) for PPi hydrolysis (optimal at 42°C and pH 8.5) and Hill coefficients that indicated cooperative binding to PPi and Mg(2+). Similarly to other class A type PPAs, HvPPA was inhibited by sodium fluoride; however, hierarchical clustering and three-dimensional (3D) homology modeling revealed HvPPA to be distinct in structure from characterized PPAs. In particular, HvPPA was highly negative in surface charge, which explained its extreme resistance to organic solvents. To demonstrate that HvPPA could drive thermodynamically unfavorable reactions to completion under conditions of reduced water activity, a novel coupled assay was developed; HvPPA hydrolyzed the PPi by-product generated in 2 M NaCl by UbaA (a "salt-loving" noncanonical E1 enzyme that adenylates ubiquitin-like proteins in the presence of ATP). Overall, we demonstrate HvPPA to be useful for hydrolyzing PPi under conditions of reduced water activity that are a hurdle to current PPA-based technologies.<br/>
</details><br/>
---

*Screening of a Haloferax volcanii Transposon Library Reveals Novel Motility and Adhesion Mutants.*<br/>
Legerme G; Yang E; Esquivel RN; Kiljunen S; Savilahti H; Pohlschroder M<br/>
**2016**<br/>
Life (Basel) 6(4)<br/>
10.3390/life6040041
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27898036)<br/>
[Citation]()<br/>
Archaea, like bacteria, use type IV pili to facilitate surface adhesion. Moreover, archaeal flagella-structures required for motility-share a common ancestry with type IV pili. While the characterization of archaeal homologs of bacterial type IV pilus biosynthesis components has revealed important aspects of flagellum and pilus biosynthesis and the mechanisms regulating motility and adhesion in archaea, many questions remain. Therefore, we screened a <br/>
</details><br/>
---

*Important roles for membrane lipids in haloarchaeal bioenergetics.*<br/>
Kellermann MY; Yoshinaga MY; Valentine RC; Wörmer L; Valentine DL<br/>
**2016**<br/>
Biochim. Biophys. Acta 1858(11)<br/>
10.1016/j.bbamem.2016.08.010
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27565574)<br/>
[Citation]()<br/>
Recent advances in lipidomic analysis in combination with various physiological experiments set the stage for deciphering the structure-function of haloarchaeal membrane lipids. Here we focused primarily on changes in lipid composition of Haloferax volcanii, but also performed a comparative analysis with four other haloarchaeal species (Halobacterium salinarum, Halorubrum lacusprofundi, Halorubrum sodomense and Haloplanus natans) all representing distinctive cell morphologies and behaviors (i.e., rod shape vs. pleomorphic behavior). Common to all five haloarchaea, our data reveal an extraordinary high level of menaquinone, reaching up to 72% of the total lipids. This ubiquity suggests that menaquinones may function beyond their ordinary role as electron and proton transporter, acting simultaneously as ion permeability barriers and as powerful shield against oxidative stress. In addition, we aimed at understanding the role of cations interacting with the characteristic negatively charged surface of haloarchaeal membranes. We propose for instance that by bridging the negative charges of adjacent anionic phospholipids, Mg<br/>
</details><br/>
---

*AglM and VNG1048G, Two Haloarchaeal UDP-Glucose Dehydrogenases, Show Different Salt-Related Behaviors.*<br/>
Kandiba L; Eichler J<br/>
**2016**<br/>
Life (Basel) 6(3)<br/>
10.3390/life6030031
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27527219)<br/>
[Citation]()<br/>
Haloferax volcanii AglM and Halobacterium salinarum VNG1048G are UDP-glucose dehydrogenases involved in N-glycosylation in each species. Despite sharing >60% sequence identity and the ability of VNG1048G to functionally replace AglM in vivo, these proteins behaved differently as salinity changed. Whereas AglM was active in 2-4 M NaCl, VNG1048G lost much of its activity when salinity dropped below 3 M NaCl. To understand the molecular basis of this phenomenon, each protein was examined by size exclusion chromatrography in 2 M NaCl. Whereas AglM appeared as a dodecamer, VNG1048G was essentially detected as a dodecamer and a dimer. The specific activity of the VNG1048G dodecamer was only a sixth of that of AglM, while the dimer was inactive. As such, not only was the oligomeric status of VNG1048G affected by lowered salinity, so was the behavior of the individual dodecamer subunits. Analyzing surface-exposed residues in homology models of the two UDP-glucose dehydrogenases revealed the more acidic and less basic VNG1048G surface, further explaining the greater salt-dependence of the Hbt. salinarum enzyme. <br/>
</details><br/>
---

*Structural characterization of the N-linked pentasaccharide decorating glycoproteins of the halophilic archaeon Haloferax volcanii.*<br/>
Kandiba L; Lin CW; Aebi M; Eichler J; Guerardel Y<br/>
**2016**<br/>
Glycobiology 26(7)<br/>
glycob/cww014
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26863921)<br/>
[Citation]()<br/>
N-Glycosylation is a post-translational modification performed in all three domains of life. In the halophilic archaea Haloferax volcanii, glycoproteins such as the S-layer glycoprotein are modified by an N-linked pentasaccharide assembled by a series of Agl (archaeal glycosylation) proteins. In the present study, mass spectrometry (MS) and nuclear magnetic resonance spectroscopy were used to define the structure of this glycan attached to at least four of the seven putative S-layer glycoprotein N-glycosylation sites, namely Asn-13, Asn-83, Asn-274 and Asn-279. Such approaches detected a trisaccharide corresponding to glucuronic acid (GlcA)-β1,4-GlcA-β1,4-glucose-β1-Asn, a tetrasaccharide corresponding to methyl-O-4-GlcA-β-1,4-galacturonic acid-α1,4-GlcA-β1,4-glucose-β1-Asn, and a pentasaccharide corresponding to hexose-1,2-[methyl-O-4-]GlcA-β-1,4-galacturonic acid-α1,4-GlcA-β1,4-glucose-β1-Asn, with previous MS and radiolabeling experiments showing the hexose at the non-reducing end of the pentasaccharide to be mannose. The present analysis thus corrects the earlier assignment of the penultimate sugar as a methyl ester of a hexuronic acid, instead revealing this sugar to be a methylated GlcA. The assignments made here are in good agreement with what was already known of the Hfx. volcanii N-glycosylation pathway from previous genetic and biochemical efforts while providing new insight into the process.<br/>
</details><br/>
---

*Mechanistic insight into protein modification and sulfur mobilization activities of noncanonical E1 and associated ubiquitin-like proteins of Archaea.*<br/>
Hepowit NL; de Vera IM; Cao S; Fu X; Wu Y; Uthandi S; Chavarria NE; Englert M; Su D; Sӧll D; Kojetin DJ; Maupin-Furlow JA<br/>
**2016**<br/>
FEBS J. 283(19)<br/>
10.1111/febs.13819
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27459543)<br/>
[Citation]()<br/>
Here we provide the first detailed biochemical study of a noncanonical E1-like enzyme with broad specificity for cognate ubiquitin-like (Ubl) proteins that mediates Ubl protein modification and sulfur mobilization to form molybdopterin and thiolated tRNA. Isothermal titration calorimetry and in vivo analyses proved useful in discovering that environmental conditions, ATP binding, and Ubl type controlled the mechanism of association of the Ubl protein with its cognate E1-like enzyme (SAMP and UbaA of the archaeon Haloferax volcanii, respectively). Further analysis revealed that ATP hydrolysis triggered the formation of thioester and peptide bonds within the Ubl:E1-like complex. Importantly, the thioester was an apparent precursor to Ubl protein modification but not sulfur mobilization. Comparative modeling to MoeB/ThiF guided the discovery of key residues within the adenylation domain of UbaA that were needed to bind ATP as well as residues that were specifically needed to catalyze the downstream reactions of sulfur mobilization and/or Ubl protein modification. UbaA was also found to be Ubl-automodified at lysine residues required for early (ATP binding) and late (sulfur mobilization) stages of enzyme activity revealing multiple layers of autoregulation. Cysteine residues, distinct from the canonical E1 'active site' cysteine, were found important in UbaA function supporting a model that this noncanonical E1 is structurally flexible in its active site to allow Ubladenylate, UblE1-like thioester and cysteine persulfide(s) intermediates to form.<br/>
</details><br/>
---

*Anaerobic Growth of Haloarchaeon Haloferax volcanii by Denitrification Is Controlled by the Transcription Regulator NarO.*<br/>
Hattori T; Shiba H; Ashiki K; Araki T; Nagashima YK; Yoshimatsu K; Fujiwara T<br/>
**2016**<br/>
J. Bacteriol. 198(7)<br/>
10.1128/JB.00833-15
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26787768)<br/>
[Citation]()<br/>
UNLABELLED: The extremely halophilic archaeon Haloferax volcanii grows anaerobically by denitrification. A putative DNA-binding protein, NarO, is encoded upstream of the respiratory nitrate reductase gene of H. volcanii. Disruption of the narO gene resulted in a loss of denitrifying growth of H. volcanii, and the expression of the recombinant NarO recovered the denitrification capacity. A novel CXnCXCX7C motif showing no remarkable similarities with known sequences was conserved in the N terminus of the NarO homologous proteins found in the haloarchaea. Restoration of the denitrifying growth was not achieved by expression of any mutant NarO in which any one of the four conserved cysteines was individually replaced by serine. A promoter assay experiment indicated that the narO gene was usually transcribed, regardless of whether it was cultivated under aerobic or anaerobic conditions. Transcription of the genes encoding the denitrifying enzymes nitrate reductase and nitrite reductase was activated under anaerobic conditions. A putative cis element was identified in the promoter sequence of haloarchaeal denitrifying genes. These results demonstrated a significant effect of NarO, probably due to its oxygen-sensing function, on the transcriptional activation of haloarchaeal denitrifying genes.
IMPORTANCE: H. volcanii is an extremely halophilic archaeon capable of anaerobic growth by denitrification. The regulatory mechanism of denitrification has been well understood in bacteria but remains unknown in archaea. In this work, we show that the helix-turn-helix (HTH)-type regulator NarO activates transcription of the denitrifying genes of H. volcanii under anaerobic conditions. A novel cysteine-rich motif, which is critical for transcriptional regulation, is present in NarO. A putative cis element was also identified in the promoter sequence of the haloarchaeal denitrifying genes.<br/>
</details><br/>
---

*A novel archaeal DNA repair factor that acts with the UvrABC system to repair mitomycin C-induced DNA damage in a PCNA-dependent manner.*<br/>
Giroux X; MacNeill SA<br/>
**2016**<br/>
Mol. Microbiol. 99(1)<br/>
10.1111/mmi.13210
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26337406)<br/>
[Citation]()<br/>
The sliding clamp proliferating cell nuclear antigen (PCNA) plays a vital role in a number of DNA repair pathways in eukaryotes and archaea by acting as a stable platform onto which other essential protein factors assemble. Many of these proteins interact with PCNA via a short peptide sequence known as a PIP (PCNA interacting protein) motif. Here we describe the identification and functional analysis of a novel PCNA interacting protein NreA that is conserved in the archaea and that has a PIP motif at its C-terminus. Using the genetically tractable euryarchaeon Haloferax volcanii as a model system, we show that the NreA protein is not required for cell viability but that loss of NreA (or replacement of the wild-type protein with a truncated version lacking the C-terminal PIP motif) results in an increased sensitivity to the DNA damaging agent mitomycin C (MMC) that correlates with delayed repair of MMC-induced chromosomal DNA damage monitored by pulsed-field gel electrophoresis. Genetic epistasis analysis in Hfx. volcanii suggests that NreA works together with the UvrABC proteins in repairing DNA damage resulting from exposure to MMC. The wide distribution of NreA family members implies an important role for the protein in DNA damage repair in all archaeal lineages. <br/>
</details><br/>
---

*Identification of Haloferax volcanii Pilin N-Glycans with Diverse Roles in Pilus Biosynthesis, Adhesion, and Microcolony Formation*<br/>
Esquivel, Rianne N.; Schulze, Stefan; Xu, Rachel; Hippler, Michael; Pohlschroder, Mechthild<br/>
**2016**<br/>
Journal of Biological Chemistry 291(20)<br/>
10.1074/jbc.M115.693556
<details>
<summary>Abstract and Links</summary>

[Link to Publication](http://www.jbc.org/content/291/20/10602.abstract)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Esquivel2016JBiolChem.ris)<br/>
N-Glycosylation is a post-translational modification common to all three domains of life. In many archaea, the oligosacharyltransferase (AglB)-dependent N-glycosylation of flagellins is required for flagella assembly. However, whether N-glycosylation is required for the assembly and/or function of the structurally related archaeal type IV pili is unknown. Here, we show that of six Haloferax volcanii adhesion pilins, PilA1 and PilA2, the most abundant pilins in pili of wild-type and Î”aglB strains, are modified under planktonic conditions in an AglB-dependent manner by the same pentasaccharide detected on H. volcanii flagellins. However, unlike wild-type cells, which have surfaces decorated with discrete pili and form a dispersed layer of cells on a plastic surface, Î”aglB cells have thick pili bundles and form microcolonies. Moreover, expressing PilA1, PilA2, or PilA6 in Î”pilA[1â€“6]Î”aglB stimulates microcolony formation compared with their expression in Î”pilA[1â€“6]. Conversely, expressing PilA3 or PilA4 in Î”pilA[1â€“6] cells results in strong surface adhesion, but not microcolony formation, and neither pilin stimulates surface adhesion in Î”pilA[1â€“6]Î”aglB cells. Although PilA4 assembles into pili in the Î”pilA[1â€“6]Î”aglB cells, these pili are, unlike wild-type pili, curled, perhaps rendering them non-functional. To our knowledge, this is the first demonstration of a differential effect of glycosylation on pilus assembly and function of paralogous pilins. The growth of wild-type cells in low salt media, a condition that decreases AglB glycosylation, also stimulates microcolony formation and inhibits motility, supporting our hypothesis that N-glycosylation plays an important role in regulating the transition between planktonic to sessile cell states as a response to stress.<br/>
</details><br/>
---

*Complementation of an aglB Mutant of Methanococcus maripaludis with Heterologous Oligosaccharyltransferases.*<br/>
Ding Y; Vrionis HA; Schneider J; Berezuk A; Khursigara CM; Jarrell KF<br/>
**2016**<br/>
PLoS ONE 11(12)<br/>
10.1371/journal.pone.0167611
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27907170)<br/>
[Citation]()<br/>
The oligosaccharyltransferase is the signature enzyme for N-linked glycosylation in all domains of life. In Archaea, this enzyme termed AglB, is responsible for transferring lipid carrier-linked glycans to select asparagine residues in a variety of target proteins including archaellins, S-layer proteins and pilins. This study investigated the ability of a variety of AglBs to compensate for the oligosaccharyltransferase activity in Methanococcus maripaludis deleted for aglB, using archaellin FlaB2 as the reporter protein since all archaellins in Mc. maripaludis are modified at multiple sites by an N-linked tetrasaccharide and this modification is required for archaellation. In the Mc. maripaludis ΔaglB strain FlaB2 runs as at a smaller apparent molecular weight in western blots and is nonarchaellated. We demonstrate that AglBs from Methanococcus voltae and Methanothermococcus thermolithotrophicus functionally replaced the oligosaccharyltransferase activity missing in the Mc. maripaludis ΔaglB strain, both returning the apparent molecular weight of FlaB2 to wildtype size and restoring archaellation. This demonstrates that AglB from Mc. voltae has a relaxed specificity for the linking sugar of the transferred glycan since while the N-linked glycan present in Mc. voltae is similar to that of Mc. maripaludis, the Mc. voltae glycan uses N-acetylglucosamine as the linking sugar. In Mc. maripaludis that role is held by N-acetylgalactosamine. This study also identifies aglB from Mtc. thermolithotrophicus for the first time by its activity. Attempts to use AglB from Methanocaldococcus jannaschii, Haloferax volcanii or Sulfolobus acidocaldarius to functionally replace the oligosaccharyltransferase activity missing in the Mc. maripaludis ΔaglB strain were unsuccessful.<br/>
</details><br/>
---

*Proteome targets of ubiquitin-like samp1ylation are associated with sulfur metabolism and oxidative stress in Haloferax volcanii.*<br/>
Dantuluri S; Wu Y; Hepowit NL; Chen H; Chen S; Maupin-Furlow JA<br/>
**2016**<br/>
Proteomics 16(7)<br/>
10.1002/pmic.201500153
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26841191)<br/>
[Citation]()<br/>
Small archeal modifier proteins (SAMPs) are related to ubiquitin in tertiary structure and in their isopeptide linkage to substrate proteins. SAMPs also function in sulfur mobilization to form biomolecules such as molybdopterin and thiolated tRNA. While SAMP1 is essential for anaerobic growth and covalently attached to lysine residues of its molybdopterin synthase partner MoaE (K240 and K247), the full diversity of proteins modified by samp1ylation is not known. Here, we expand the knowledge of proteins isopeptide linked to SAMP1. LC-MS/MS analysis of -Gly-Gly signatures derived from SAMP1 S85R conjugates cleaved with trypsin was used to detect sites of sampylation (23 lysine residues) that mapped to 11 target proteins. Many of the identified target proteins were associated with sulfur metabolism and oxidative stress including MoaE, SAMP-activating E1 enzyme (UbaA), methionine sulfoxide reductase homologs (MsrA and MsrB), and the Fe-S assembly protein SufB. Several proteins were found to have multiple sites of samp1ylation, and the isopeptide linkage at SAMP3 lysines (K18, K55, and K62) revealed hetero-SAMP chain topologies. Follow-up affinity purification of selected protein targets (UbaA and MoaE) confirmed the LC-MS/MS results. 3D homology modeling suggested sampy1ylation is autoregulatory in inhibiting the activity of its protein partners (UbaA and MoaE), while occurring on the surface of some protein targets, such as SufB and MsrA/B. Overall, we provide evidence that SAMP1 is a ubiquitin-like protein modifier that is relatively specific in tagging its protein partners as well as proteins associated with oxidative stress response. <br/>
</details><br/>
---

*Haloferax volcanii UbaA, catalytic engine for sampylation and sulfur transfer.*<br/>
Bochtler M; Piasecka A<br/>
**2016**<br/>
FEBS J. 283(19)<br/>
10.1111/febs.13826
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27542853)<br/>
[Citation]()<br/>
The Small Archaeal Modifier Proteins (SAMPs) from Haloferax volcanii belong to the group of ubiquitin like proteins (Ubls) that act both as protein modifiers and sulfur carriers. The E1-like enzyme UbaA is essential for SAMP activation and therefore required for both sampylation and sulfur transfer. Here, we provide a commentary on the thorough characterization of UbaA by J. Maupin-Furlow and colleagues.<br/>
</details><br/>
---

*Genome-wide identification of transcriptional start sites in the haloarchaeon Haloferax volcanii based on differential RNA-Seq (dRNA-Seq).*<br/>
Babski J; Haas KA; Näther-Schindler D; Pfeiffer F; Förstner KU; Hammelmann M; Hilker R; Becker A; Sharma CM; Marchfelder A; Soppa J<br/>
**2016**<br/>
BMC Genomics 17(1)<br/>
10.1186/s12864-016-2920-y
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/27519343)<br/>
[Citation]()<br/>
BACKGROUND: Differential RNA-Seq (dRNA-Seq) is a recently developed method of performing primary transcriptome analyses that allows for the genome-wide mapping of transcriptional start sites (TSSs) and the identification of novel transcripts. Although the transcriptomes of diverse bacterial species have been characterized by dRNA-Seq, the transcriptome analysis of archaeal species is still rather limited. Therefore, we used dRNA-Seq to characterize the primary transcriptome of the model archaeon Haloferax volcanii.
RESULTS: Three independent cultures of Hfx. volcanii grown under optimal conditions to the mid-exponential growth phase were used to determine the primary transcriptome and map the 5'-ends of the transcripts. In total, 4749 potential TSSs were detected. A position weight matrix (PWM) was derived for the promoter predictions, and the results showed that 64 % of the TSSs were preceded by stringent or relaxed basal promoters. Of the identified TSSs, 1851 belonged to protein-coding genes. Thus, fewer than half (46 %) of the 4040 protein-coding genes were expressed under optimal growth conditions. Seventy-two percent of all protein-coding transcripts were leaderless, which emphasized that this pathway is the major pathway for translation initiation in haloarchaea. A total of 2898 of the TSSs belonged to potential non-coding RNAs, which accounted for an unexpectedly high fraction (61 %) of all transcripts. Most of the non-coding TSSs had not been previously described (2792) and represented novel sequences (59 % of all TSSs). A large fraction of the potential novel non-coding transcripts were cis-antisense RNAs (1244 aTSSs). A strong negative correlation between the levels of antisense transcripts and cognate sense mRNAs was found, which suggested that the negative regulation of gene expression via antisense RNAs may play an important role in haloarchaea. The other types of novel non-coding transcripts corresponded to internal transcripts overlapping with mRNAs (1153 iTSSs) and intergenic small RNA (sRNA) candidates (395 TSSs).
CONCLUSION: This study provides a comprehensive map of the primary transcriptome of Hfx. volcanii grown under optimal conditions. Fewer than half of all protein-coding genes have been transcribed under these conditions. Unexpectedly, more than half of the detected TSSs belonged to several classes of non-coding RNAs. Thus, RNA-based regulation appears to play a more important role in haloarchaea than previously anticipated.<br/>
</details><br/>
---

*Activation of a dormant replication origin is essential for Haloferax mediterranei lacking the primary origins.*<br/>
Yang H; Wu Z; Liu J; Liu X; Wang L; Cai S; Xiang H<br/>
**2015**<br/>
Nat Commun 6(None)<br/>
10.1038/ncomms9321
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26374389)<br/>
[Citation]()<br/>
The use of multiple origins for chromosome replication has been demonstrated in archaea. Similar to the dormant origins in eukaryotes, some potential origins in archaea appear to be inactive during genome replication. We have comprehensively explored the origin utilization in Haloferax mediterranei. Here we report three active chromosomal origins by genome-wide replication profiling, and demonstrate that when these three origins are deleted, a dormant origin becomes activated. Notably, this dormant origin cannot be further deleted when the other origins are already absent and vice versa. Interestingly, a potential origin that appears to stay dormant in its native host H. volcanii lacking the main active origins becomes activated and competent for replication of the entire chromosome when integrated into the chromosome of origin-deleted H. mediterranei. These results indicate that origin-dependent replication is strictly required for H. mediterranei and that dormant replication origins in archaea can be activated if needed. <br/>
</details><br/>
---

*[An efficient genetic knockout system based on linear DNA fragment homologous recombination for halophilic archaea].*<br/>
Xiaoli W; Chuang J; Jianhua L; Xipeng L<br/>
**2015**<br/>
Yi Chuan 37(4)<br/>
10.16288/j.yczz.14-366
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25881705)<br/>
[Citation]()<br/>
With the development of functional genomics, gene-knockout is becoming an important tool to elucidate gene functions in vivo. As a good model strain for archaeal genetics, Haloferax volcanii has received more attention. Although several genetic manipulation systems have been developed for some halophilic archaea, it is time-consuming because of the low percentage of positive clones during the second-recombination selection. These classical gene knockout methods are based on DNA recombination between the genomic homologous sequence and the circular suicide plasmid, which carries a pyrE selection marker and two DNA fragments homologous to the upstream and downstream fragments of the target gene. Many wild-type clones are obtained through a reverse recombination between the plasmid and genome in the classic gene knockout method. Therefore, it is necessary to develop an efficient gene knockout system to increase the positive clone percentage. Here we report an improved gene knockout method using a linear DNA cassette consisting of upstream and downstream homologous fragments, and the pyrE marker. Gene deletions were subsequently detected by colony PCR analysis. We determined the efficiency of our knockout method by deleting the xpb2 gene from the H. volcanii genome, with the percentage of positive clones higher than 50%. Our method provides an efficient gene knockout strategy for halophilic archaea. <br/>
</details><br/>
---

*Enhancing recombinant protein solubility with ubiquitin-like small archeal modifying protein fusion partners.*<br/>
Varga S; Pathare GR; Baka E; Boicu M; Kriszt B; Székács A; Zinzula L; Kukolya J; Nagy I<br/>
**2015**<br/>
J. Microbiol. Methods 118(None)<br/>
10.1016/j.mimet.2015.08.017
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26341610)<br/>
[Citation]()<br/>
A variety of protein expression tags with different biochemical properties has been used to enhance the yield and solubility of recombinant proteins. Ubiquitin, SUMO (small ubiquitin-like modifier) and prokaryotic ubiquitin like MoaD (molybdopterin synthase, small subunit) fusion tags are getting more popular because of their small size. In this paper we report on the use of ubiquitin-like small archaeal modifier proteins (SAMPs) as fusion tags since they proved to increase expression yield, stability and solubility in our experiments. Equally important, they did not co-purify with proteins of the expression host and there was information that their specific JAB1/MPN/Mov34 metalloenzyme (JAMM) protease can recognize the C-terminal VSGG sequence when SAMPs fused, either branched or linearly to target proteins, and cleave it specifically. SAMPs and JAMM proteases from Haloferax volcanii, Thermoplasma acidophilum, Methanococcoides burtonii and Nitrosopumilus maritimus were selected, cloned, expressed heterologously in Escherichia coli and tested as fusion tags and cleaving proteases, respectively. Investigated SAMPs enhanced protein expression and solubility on a wide scale. T. acidophilum SAMPs Ta0895 and Ta01019 were the best performing tags and their effect was comparable to the widely used maltose binding protein (MBP) and N utilization substance protein A (NusA) tags. Moreover, H. volcanii SAMP Hvo_2619 contribution was mediocre, whereas M. burtonii Mbur_1415 could not be expressed. Out of four investigated JAMM proteases, only Hvo_2505 could cleave fusion tags. Interestingly, it was found active not only on its own partner substrate Hvo_2619, but it also cleaved off Ta0895. <br/>
</details><br/>
---

*Preface to the proceedings of Halophiles 2013*<br/>
Papke, R. Thane<br/>
**2015**<br/>
Frontiers in Microbiology 6()<br/>
10.3389/fmicb.2015.00341
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://www.frontiersin.org/article/10.3389/fmicb.2015.00341)<br/>
[Citation](https://github.com/halophiles/halowiki//tree/master/publications/Others/Papke2015FrontMicrobiol.ris)<br/>
<br/>
</details><br/>
---

*A manual curation strategy to improve genome annotation: application to a set of haloarchael genomes.*<br/>
Pfeiffer F; Oesterhelt D<br/>
**2015**<br/>
Life (Basel) 5(2)<br/>
10.3390/life5021427
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26042526)<br/>
[Citation]()<br/>
Genome annotation errors are a persistent problem that impede research in the biosciences. A manual curation effort is described that attempts to produce high-quality genome annotations for a set of haloarchaeal genomes (Halobacterium salinarum and Hbt. hubeiense, Haloferax volcanii and Hfx. mediterranei, Natronomonas pharaonis and Nmn. moolapensis, Haloquadratum walsbyi strains HBSQ001 and C23, Natrialba magadii, Haloarcula marismortui and Har. hispanica, and Halohasta litchfieldiae). Genomes are checked for missing genes, start codon misassignments, and disrupted genes. Assignments of a specific function are preferably based on experimentally characterized homologs (Gold Standard Proteins). To avoid overannotation, which is a major source of database errors, we restrict annotation to only general function assignments when support for a specific substrate assignment is insufficient. This strategy results in annotations that are resistant to the plethora of errors that compromise public databases. Annotation consistency is rigorously validated for ortholog pairs from the genomes surveyed. The annotation is regularly crosschecked against the UniProt database to further improve annotations and increase the level of standardization. Enhanced genome annotations are submitted to public databases (EMBL/GenBank, UniProt), to the benefit of the scientific community. The enhanced annotations are also publically available via HaloLex. <br/>
</details><br/>
---

*Archaeal type IV pili and their involvement in biofilm formation.*<br/>
Pohlschroder M; Esquivel RN<br/>
**2015**<br/>
Front Microbiol 6(None)<br/>
10.3389/fmicb.2015.00190
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25852657)<br/>
[Citation]()<br/>
Type IV pili are ancient proteinaceous structures present on the cell surface of species in nearly all bacterial and archaeal phyla. These filaments, which are required for a diverse array of important cellular processes, are assembled employing a conserved set of core components. While type IV pilins, the structural subunits of pili, share little sequence homology, their signal peptides are structurally conserved allowing for in silico prediction. Recently, in vivo studies in model archaea representing the euryarchaeal and crenarchaeal kingdoms confirmed that several of these pilins are incorporated into type IV adhesion pili. In addition to facilitating surface adhesion, these in vivo studies also showed that several predicted pilins are required for additional functions that are critical to biofilm formation. Examples include the subunits of Sulfolobus acidocaldarius Ups pili, which are induced by exposure to UV light and promote cell aggregation and conjugation, and a subset of the Haloferax volcanii adhesion pilins, which play a critical role in microcolony formation while other pilins inhibit this process. The recent discovery of novel pilin functions such as the ability of haloarchaeal adhesion pilins to regulate swimming motility may point to novel regulatory pathways conserved across prokaryotic domains. In this review, we will discuss recent advances in our understanding of the functional roles played by archaeal type IV adhesion pili and their subunits, with particular emphasis on their involvement in biofilm formation. <br/>
</details><br/>
---

*The oxidative pentose phosphate pathway in the haloarchaeon Haloferax volcanii involves a novel type of glucose-6-phosphate dehydrogenase--The archaeal Zwischenferment.*<br/>
Pickl A; Schönheit P<br/>
**2015**<br/>
FEBS Lett. 589(10)<br/>
10.1016/j.febslet.2015.03.026
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25836736)<br/>
[Citation]()<br/>
The oxidative pentose phosphate pathway (OPPP), catalyzing the oxidation of glucose-6-phosphate to ribulose-5-phosphate is ubiquitous in eukarya and bacteria but has not yet been reported in archaea. In haloarchaea a putative 6-phosphogluconate dehydrogenase (6PGDH) is annotated, whereas a gene coding for glucose-6-phosphate dehydrogenase (Glc6PDH) could not be identified. Here we report the purification and characterization of a novel type of Glc6PDH in Haloferax volcanii that is not related to bacterial and eukaryal Glc6PDHs and the encoding gene is designated as azf (archaeal zwischenferment). Further, recombinant H. volcanii 6PGDH was characterized. Deletion mutant analyses indicate that both, Glc6PDH and 6PGDH, are functionally involved in pentose phosphate formation in vivo. This is the first report on the operation of the OPPP in the domain of archaea. <br/>
</details><br/>
---

*Genome-wide DNA methylation analysis of Haloferax volcanii H26 and identification of DNA methyltransferase related PD-(D/E)XK nuclease family protein HVO_A0006.*<br/>
Ouellette M; Jackson L; Chimileski S; Papke RT<br/>
**2015**<br/>
Front Microbiol 6(None)<br/>
10.3389/fmicb.2015.00251
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25904898)<br/>
[Citation]()<br/>
Restriction-modification (RM) systems have evolved to protect the cell from invading DNAs and are composed of two enzymes: a DNA methyltransferase and a restriction endonuclease. Although RM systems are present in both archaeal and bacterial genomes, DNA methylation in archaea has not been well defined. In order to characterize the function of RM systems in archaeal species, we have made use of the model haloarchaeon Haloferax volcanii. A genomic DNA methylation analysis of H. volcanii strain H26 was performed using PacBio single molecule real-time (SMRT) sequencing. This analysis was also performed on a strain of H. volcanii in which an annotated DNA methyltransferase gene HVO_A0006 was deleted from the genome. Sequence analysis of H26 revealed two motifs which are modified in the genome: C(m4)TAG and GCA(m6)BN6VTGC. Analysis of the ΔHVO_A0006 strain indicated that it exhibited reduced adenine methylation compared to the parental strain and altered the detected adenine motif. However, protein domain architecture analysis and amino acid alignments revealed that HVO_A0006 is homologous only to the N-terminal endonuclease region of Type IIG RM proteins and contains a PD-(D/E)XK nuclease motif, suggesting that HVO_A0006 is a PD-(D/E)XK nuclease family protein. Further bioinformatic analysis of the HVO_A0006 gene demonstrated that the gene is rare among the Halobacteria. It is surrounded by two transposition genes suggesting that HVO_A0006 is a fragment of a Type IIG RM gene, which has likely been acquired through gene transfer, and affects restriction-modification activity by interacting with another RM system component(s). Here, we present the first genome-wide characterization of DNA methylation in an archaeal species and examine the function of a DNA methyltransferase related gene HVO_A0006. <br/>
</details><br/>
---

*Deletion of the Sm1 encoding motif in the lsm gene results in distinct changes in the transcriptome and enhanced swarming activity of Haloferax cells.*<br/>
Maier LK; Benz J; Fischer S; Alstetter M; Jaschinski K; Hilker R; Becker A; Allers T; Soppa J; Marchfelder A<br/>
**2015**<br/>
Biochimie 117(None)<br/>
10.1016/j.biochi.2015.02.023
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25754521)<br/>
[Citation]()<br/>
Members of the Sm protein family are important for the cellular RNA metabolism in all three domains of life. The family includes archaeal and eukaryotic Lsm proteins, eukaryotic Sm proteins and archaeal and bacterial Hfq proteins. While several studies concerning the bacterial and eukaryotic family members have been published, little is known about the archaeal Lsm proteins. Although structures for several archaeal Lsm proteins have been solved already more than ten years ago, we still do not know much about their biological function, however one can confidently propose that the archaeal Lsm proteins will also be involved in RNA metabolism. Therefore, we investigated this protein in the halophilic archaeon Haloferax volcanii. The Haloferax genome encodes a single Lsm protein, the lsm gene overlaps and is co-transcribed with the gene for the ribosomal L37.eR protein. Here, we show that the reading frame of the lsm gene contains a promoter which regulates expression of the overlapping rpl37R gene. This rpl37R specific promoter ensures high expression of the rpl37R gene in exponential growth phase. To investigate the biological function of the Lsm protein we generated a lsm deletion mutant that had the coding sequence for the Sm1 motif removed but still contained the internal promoter for the downstream rpl37R gene. The transcriptome of this deletion mutant was compared to the wild type transcriptome, revealing that several genes are down-regulated and many genes are up-regulated in the deletion strain. Northern blot analyses confirmed down-regulation of two genes. In addition, the deletion strain showed a gain of function in swarming, in congruence with the up-regulation of transcripts encoding proteins required for motility. <br/>
</details><br/>
---

*The Adaptive Immune System of Haloferax volcanii.*<br/>
Maier LK; Dyall-Smith M; Marchfelder A<br/>
**2015**<br/>
Life (Basel) 5(1)<br/>
10.3390/life5010521
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25692903)<br/>
[Citation]()<br/>
To fight off invading genetic elements, prokaryotes have developed an elaborate defence system that is both adaptable and heritable-the CRISPR-Cas system (CRISPR is short for: clustered regularly interspaced short palindromic repeats and Cas: CRISPR associated). Comprised of proteins and multiple small RNAs, this prokaryotic defence system is present in 90% of archaeal and 40% of bacterial species, and enables foreign intruders to be eliminated in a sequence-specific manner. There are three major types (I-III) and at least 14 subtypes of this system, with only some of the subtypes having been analysed in detail, and many aspects of the defence reaction remaining to be elucidated. Few archaeal examples have so far been analysed. Here we summarize the characteristics of the CRISPR-Cas system of Haloferax volcanii, an extremely halophilic archaeon originally isolated from the Dead Sea. It carries a single CRISPR-Cas system of type I-B, with a Cascade like complex composed of Cas proteins Cas5, Cas6b and Cas7. Cas6b is essential for CRISPR RNA (crRNA) maturation but is otherwise not required for the defence reaction. A systematic search revealed that six protospacer adjacent motif (PAM) sequences are recognised by the Haloferax defence system. For successful invader recognition, a non-contiguous seed sequence of 10 base-pairs between the crRNA and the invader is required. <br/>
</details><br/>
---

*An active immune defense with a minimal CRISPR (clustered regularly interspaced short palindromic repeats) RNA and without the Cas6 protein.*<br/>
Maier LK; Stachler AE; Saunders SJ; Backofen R; Marchfelder A<br/>
**2015**<br/>
J. Biol. Chem. 290(7)<br/>
10.1074/jbc.M114.617506
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25512373)<br/>
[Citation]()<br/>
The prokaryotic immune system CRISPR-Cas (clustered regularly interspaced short palindromic repeats-CRISPR-associated) is a defense system that protects prokaryotes against foreign DNA. The short CRISPR RNAs (crRNAs) are central components of this immune system. In CRISPR-Cas systems type I and III, crRNAs are generated by the endonuclease Cas6. We developed a Cas6b-independent crRNA maturation pathway for the Haloferax type I-B system in vivo that expresses a functional crRNA, which we termed independently generated crRNA (icrRNA). The icrRNA is effective in triggering degradation of an invader plasmid carrying the matching protospacer sequence. The Cas6b-independent maturation of the icrRNA allowed mutation of the repeat sequence without interfering with signals important for Cas6b processing. We generated 23 variants of the icrRNA and analyzed them for activity in the interference reaction. icrRNAs with deletions or mutations of the 3' handle are still active in triggering an interference reaction. The complete 3' handle could be removed without loss of activity. However, manipulations of the 5' handle mostly led to loss of interference activity. Furthermore, we could show that in the presence of an icrRNA a strain without Cas6b (Δcas6b) is still active in interference. <br/>
</details><br/>
---

*DNA replication restart and cellular dynamics of Hef helicase/nuclease protein in Haloferax volcanii.*<br/>
Lestini R; Delpech F; Myllykallio H<br/>
**2015**<br/>
Biochimie 118(None)<br/>
10.1016/j.biochi.2015.07.022
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26215377)<br/>
[Citation]()<br/>
Understanding how frequently spontaneous replication arrests occur and how archaea deal with these arrests are very interesting and challenging research topics. Here we will described how genetic and imaging studies have revealed the central role of the archaeal helicase/nuclease Hef belonging to the XPF/MUS81/FANCM family of endonucleases in repair of arrested replication forks. Special focus will be on description of a recently developed combination of genetic and imaging tools to study the dynamic localization of a functional Hef::GFP (Green Fluorescent Protein) fusion protein in the living cells of halophilic archaea Haloferax volcanii. As Archaea provide an excellent and unique model for understanding how DNA replication is regulated to allow replication of a circular DNA molecule either from single or multiple replication origins, we will also summarize recent studies that have revealed peculiar features regarding DNA replication, particularly in halophilic archaea. We strongly believe that fundamental knowledge of our on-going studies will shed light on the evolutionary history of the DNA replication machinery and will help to establish general rules concerning replication restart and the key role of recombination proteins not only in bacteria, yeast and higher eukaryotes but also in archaea. <br/>
</details><br/>
---

*Model-Free RNA Sequence and Structure Alignment Informed by SHAPE Probing Reveals a Conserved Alternate Secondary Structure for 16S rRNA.*<br/>
Lavender CA; Lorenz R; Zhang G; Tamayo R; Hofacker IL; Weeks KM<br/>
**2015**<br/>
PLoS Comput. Biol. 11(5)<br/>
10.1371/journal.pcbi.1004126
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25992778)<br/>
[Citation]()<br/>
Discovery and characterization of functional RNA structures remains challenging due to deficiencies in de novo secondary structure modeling. Here we describe a dynamic programming approach for model-free sequence comparison that incorporates high-throughput chemical probing data. Based on SHAPE probing data alone, ribosomal RNAs (rRNAs) from three diverse organisms--the eubacteria E. coli and C. difficile and the archeon H. volcanii--could be aligned with accuracies comparable to alignments based on actual sequence identity. When both base sequence identity and chemical probing reactivities were considered together, accuracies improved further. Derived sequence alignments and chemical probing data from protein-free RNAs were then used as pseudo-free energy constraints to model consensus secondary structures for the 16S and 23S rRNAs. There are critical differences between these experimentally-informed models and currently accepted models, including in the functionally important neck and decoding regions of the 16S rRNA. We infer that the 16S rRNA has evolved to undergo large-scale changes in base pairing as part of ribosome function. As high-quality RNA probing data become widely available, structurally-informed sequence alignment will become broadly useful for de novo motif and function discovery.<br/>
</details><br/>
---

*Haloferax volcanii, as a Novel Tool for Producing Mammalian Olfactory Receptors Embedded in Archaeal Lipid Bilayer.*<br/>
Lobasso S; Vitale R; Lopalco P; Corcelli A<br/>
**2015**<br/>
Life (Basel) 5(1)<br/>
10.3390/life5010770
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25761264)<br/>
[Citation]()<br/>
The aim of this study was to explore the possibility of using an archaeal microorganism as a host system for expressing mammalian olfactory receptors (ORs). We have selected the archaeon Haloferax volcanii as a cell host system and one of the most extensively investigated OR, namely I7-OR, whose preferred ligands are short-chain aldehydes, such as octanal, heptanal, nonanal. A novel plasmid has been constructed to express the rat I7-OR, fused with a hexahistidine-tag for protein immunodetection. The presence of the recombinant receptor at a membrane level was demonstrated by immunoblot of the membranes isolated from the transgenic archaeal strain. In addition, the lipid composition of archaeonanosomes containing ORs has been characterized in detail by High-Performance Thin-Layer Chromatography (HPTLC) in combination with Matrix-Assisted Laser Desorption Ionization-Time-Of-Flight/Mass Spectrometry (MALDI-TOF/MS) analysis. <br/>
</details><br/>
---

*Deciphering archaeal glycolipids of an extremely halophilic archaeon of the genus Halobellus by MALDI-TOF/MS.*<br/>
Lobasso S; Pérez-Davó A; Vitale R; Sánchez MM; Corcelli A<br/>
**2015**<br/>
Chem. Phys. Lipids 186(None)<br/>
10.1016/j.chemphyslip.2014.11.002
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25447292)<br/>
[Citation]()<br/>
Polar membrane lipids of an archaeal microorganism recently isolated from the natural salt lake Fuente de Piedra (Málaga, Spain) have been studied by means of TLC in combination with MALDI-TOF mass spectrometry. The major phospholipids are the ether lipids phosphatidylglycerophosphate methyl ester and phosphatidylglycerosulfate, while phosphatidylglycerol is barely detectable; in addition the bisphosphatidylglycerol (archaeal cardiolipin) has been detected for the first time in a representative of the genus Halobellus. The structures of glycolipids, including a glycosyl-cardiolipin, have been elucidated by post source decay (PSD) mass spectrometry analysis. Besides the monosulfated diglycosyl diphytanylglyceroldiether, two variants of a bis-sulfated diglycosyl diphytanylglyceroldiether have been identified; furthermore the glycosyl-cardiolipin is found to have the same structure of the analogue present in Halorubrum trapanicum and Haloferax volcanii. The role of the abundant sulfated glycolipids in facing high extracellular salinity is discussed. <br/>
</details><br/>
---

*Deciphering a pathway of Halobacterium salinarum N-glycosylation.*<br/>
Kandiba L; Eichler J<br/>
**2015**<br/>
Microbiologyopen 4(1)<br/>
10.1002/mbo3.215
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25461760)<br/>
[Citation]()<br/>
Genomic analysis points to N-glycosylation as being a common posttranslational modification in Archaea. To date, however, pathways of archaeal N-glycosylation have only been described for few species. With this in mind, the similarities of N-linked glycans decorating glycoproteins in the haloarchaea Haloferax volcanii and Halobacterium salinarum directed a series of bioinformatics, genetic, and biochemical experiments designed to describe that Hbt. salinarum pathway responsible for biogenesis of one of the two N-linked oligosaccharides described in this species. As in Hfx. volcanii, where agl (archaeal glycosylation) genes that encode proteins responsible for the assembly and attachment of a pentasaccharide to target protein Asn residues are clustered in the genome, Hbt. salinarum also contains a group of clustered homologous genes (VNG1048G-VNG1068G). Introduction of these Hbt. salinarum genes into Hfx. volcanii mutant strains deleted of the homologous sequence restored the lost activity. Moreover, transcription of the Hbt. salinarum genes in the native host, as well as in vitro biochemical confirmation of the predicted functions of several of the products of these genes provided further support for assignments made following bioinformatics and genetic experiments. Based on the results obtained in this study, the first description of an N-glycosylation pathway in Hbt. salinarum is offered.<br/>
</details><br/>
---

*XacR - a novel transcriptional regulator of D-xylose and L-arabinose catabolism in the haloarchaeon Haloferax volcanii.*<br/>
Johnsen U; Sutter JM; Schulz AC; Tästensen JB; Schönheit P<br/>
**2015**<br/>
Environ. Microbiol. 17(5)<br/>
10.1111/1462-2920.12603
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25141768)<br/>
[Citation]()<br/>
The haloarchaeon Haloferax volcanii degrades D-xylose and L-arabinose via oxidative pathways to α-ketoglutarate. The genes involved in these pathways are clustered and were transcriptionally upregulated by both D-xylose and L-arabinose suggesting a common regulator. Adjacent to the gene cluster, a putative IclR-like transcriptional regulator, HVO_B0040, was identified. It is shown that HVO_B0040, designated xacR, encodes an activator of both D-xylose and L-arabinose catabolism: in ΔxacR cells, transcripts of genes involved in pentose catabolism could not be detected; transcript formation could be recovered by complementation, indicating XacR dependent transcriptional activation. Upstream activation promoter regions and nucleotide sequences that were essential for XacR-mediated activation of pentose-specific genes were identified by in vivo deletion and scanning mutagenesis. Besides its activator function XacR acted as repressor of its own synthesis: xacR deletion resulted in an increase of xacR promoter activity. A palindromic sequence was identified at the operator site of xacR promoter, and mutation of this sequence also resulted in an increase and thus derepression of xacR promoter activity. It is concluded that the palindromic sequence represents the binding site of XacR as repressor. This is the first report of a transcriptional regulator of pentose catabolism in the domain of archaea.<br/>
</details><br/>
---

*Inhibiting NAD+-dependent DNA ligase activity with 2-(cyclopentyloxy)-5'-deoxyadenosine (CPOdA) offers a new tool for DNA replication and repair studies in the model archaeon Haloferax volcanii.*<br/>
Giroux X; MacNeill SA<br/>
**2015**<br/>
FEMS Microbiol. Lett. 362(21)<br/>
femsle/fnv181
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26420852)<br/>
[Citation]()<br/>
DNA ligases play an essential role in many aspects of DNA metabolism in all three domains of life. The haloarchaeal organism Haloferax volcanii encodes both ATP- and NAD(+)-dependent DNA ligase enzymes designated LigA and LigN, respectively. Neither LigA nor LigN alone is required for cell viability but they share an essential function, most likely the ligation of Okazaki fragments during chromosome replication. Here we show that 2-(cyclopentyloxy)-5'-deoxyadenosine (referred to as CPOdA), originally developed as a inhibitor of bacterial NAD(+)-dependent DNA ligases, is a potent inhibitor of the growth of Hfx. volcanii cells expressing LigN alone, causing chromosome fragmentation and cell death, while cells expressing LigA are unaffected. Growth inhibition occurs at significantly lower CPOdA concentrations (MIC ≤ 50 ng ml(-1)) than those required for inhibition of bacterial growth (≥2 μg ml(-1)). CPOdA has the potential to become a vital tool in DNA replication and repair studies in this important model organism.<br/>
</details><br/>
---

*Molecular Genetic Methods to Study DNA Replication Protein Function in Haloferax volcanii, A Model Archaeal Organism.*<br/>
Giroux X; MacNeill SA<br/>
**2015**<br/>
Methods Mol. Biol. 1300(None)<br/>
10.1007/978-1-4939-2596-4_13
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25916714)<br/>
[Citation]()<br/>
Successful high-fidelity chromosomal DNA replication is fundamental to all forms of cellular life and requires the complex interplay of a variety of essential and nonessential protein factors in a spatially and temporally coordinated manner. Much of what is known about the enzymes and mechanisms of chromosome replication has come from analysis of simple microbial model systems, such as yeast and archaea. Archaea possess a highly simplified eukaryotic-like replication apparatus, making them an excellent model for gaining novel insights into conserved aspects of protein function at the heart of the replisome. Amongst the thermophilic archaea, a number of species have proved useful for biochemical analysis of protein function, but few of these organisms are suited to genetic analysis. One archaeal organism that is genetically tractable is the mesophilic euryarchaeon Haloferax volcanii, a halophile that grows aerobically in high salt medium at an optimum temperature of 40-45 °C and with a doubling time of 2-3 h. The Hfx. volcanii genome has been sequenced and a range of methods have been developed to allow reverse genetic analysis of protein function in vivo, including techniques for gene replacement and gene deletion, transcriptional regulation, point mutation and gene tagging. Here we briefly summarize current knowledge of the chromosomal DNA replication machinery in the haloarchaea before describing in detail the molecular methods available to probe protein structure and function within the Hfx. volcanii replication apparatus. <br/>
</details><br/>
---

*Archaeal membrane-associated proteases: insights on Haloferax volcanii and other haloarchaea.*<br/>
Giménez MI; Cerletti M; De Castro RE<br/>
**2015**<br/>
Front Microbiol 6(None)<br/>
10.3389/fmicb.2015.00039
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25774151)<br/>
[Citation]()<br/>
The function of membrane proteases range from general house-keeping to regulation of cellular processes. Although the biological role of these enzymes in archaea is poorly understood, some of them are implicated in the biogenesis of the archaeal cell envelope and surface structures. The membrane-bound ATP-dependent Lon protease is essential for cell viability and affects membrane carotenoid content in Haloferax volcanii. At least two different proteases are needed in this archaeon to accomplish the posttranslational modifications of the S-layer glycoprotein. The rhomboid protease RhoII is involved in the N-glycosylation of the S-layer protein with a sulfoquinovose-containing oligosaccharide while archaeosortase ArtA mediates the proteolytic processing coupled-lipid modification of this glycoprotein facilitating its attachment to the archaeal cell surface. Interestingly, two different signal peptidase I homologs exist in H. volcanii, Sec11a and Sec11b, which likely play distinct physiological roles. Type IV prepilin peptidase PibD processes flagellin/pilin precursors, being essential for the biogenesis and function of the archaellum and other cell surface structures in H. volcanii. <br/>
</details><br/>
---

*Secretion of Tat-dependent halolysin SptA capable of autocatalytic activation and its relation to haloarchaeal growth.*<br/>
Du X; Li M; Tang W; Zhang Y; Zhang L; Wang J; Li T; Tang B; Tang XF<br/>
**2015**<br/>
Mol. Microbiol. 96(3)<br/>
10.1111/mmi.12955
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25641392)<br/>
[Citation]()<br/>
Halolysins are Tat-dependent extracellular subtilases of haloarchaea. Whether halolysins can be activated before transport across the cytoplasmic membrane in a folded state and how haloarchaea minimize the risk of intracellular activation of halolysins and proteolysis of cellular proteins are unknown. Here, we report that both the precursor and proform of halolysin SptA from Natrinema sp. J7-2 mature autocatalytically, and the SptA maturation proceeds less efficiently in the presence of KCl than NaCl. When produced in Haloferax volcanii, most SptA molecules are secreted into the culture medium, but a small number of molecules can be activated intracellularly, affecting the cell's growth. Furthermore, retardation of SptA secretion in Hfx. volcanii via mutation of the Tat signal peptide leads to intracellular accumulation of the active enzyme and subsequent cell death. Although the Sec signal peptide can mediate SptA secretion in Hfx. volcanii, the secreted protein undergoes proteolysis. In Natrinema sp. J7-2, SptA is secreted primarily during stationary phase, and the intracellular accumulation of mature enzyme occurs during the stationary and death phases. The growth phase-dependent synthesis of SptA, highly efficient secretion system, and high intracellular KCl concentration, contribute to the suppression of premature activation of this enzyme in Natrinema sp. J7-2. <br/>
</details><br/>
---

*CetZ tubulin-like proteins control archaeal cell shape.*<br/>
Duggin IG; Aylett CH; Walsh JC; Michie KA; Wang Q; Turnbull L; Dawson EM; Harry EJ; Whitchurch CB; Amos LA; Löwe J<br/>
**2015**<br/>
Nature 519(7543)<br/>
10.1038/nature13983
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25533961)<br/>
[Citation]()<br/>
Tubulin is a major component of the eukaryotic cytoskeleton, controlling cell shape, structure and dynamics, whereas its bacterial homologue FtsZ establishes the cytokinetic ring that constricts during cell division. How such different roles of tubulin and FtsZ evolved is unknown. Studying Archaea may provide clues as these organisms share characteristics with Eukarya and Bacteria. Here we report the structure and function of proteins from a distinct family related to tubulin and FtsZ, named CetZ, which co-exists with FtsZ in many archaea. CetZ X-ray crystal structures showed the FtsZ/tubulin superfamily fold, and one crystal form contained sheets of protofilaments, suggesting a structural role. However, inactivation of CetZ proteins in Haloferax volcanii did not affect cell division. Instead, CetZ1 was required for differentiation of the irregular plate-shaped cells into a rod-shaped cell type that was essential for normal swimming motility. CetZ1 formed dynamic cytoskeletal structures in vivo, relating to its capacity to remodel the cell envelope and direct rod formation. CetZ2 was also implicated in H. volcanii cell shape control. Our findings expand the known roles of the FtsZ/tubulin superfamily to include archaeal cell shape dynamics, suggesting that a cytoskeletal role might predate eukaryotic cell evolution, and they support the premise that a major function of the microbial rod shape is to facilitate swimming. <br/>
</details><br/>
---

*Nucleotides Flanking the Start Codon in hsp70 mRNAs with Very Short 5'-UTRs Greatly Affect Gene Expression in Haloarchaea.*<br/>
Chen W; Yang G; He Y; Zhang S; Chen H; Shen P; Chen X; Huang YP<br/>
**2015**<br/>
PLoS ONE 10(9)<br/>
10.1371/journal.pone.0138473
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26379277)<br/>
[Citation]()<br/>
Leaderless translation is prevalent in haloarchaea, with many of these leaderless transcripts possessing short 5'-untranslated regions (UTRs) less than 10 nucleotides. Whereas, little is known about the function of this very short 5'-UTR. Our previous studies determined that just four nucleotides preceded the start codon of hsp70 mRNA in Natrinema sp. J7, with residues -3A and +4G, relative to the A of the ATG start codon, acting as the preferred bases around the start codon of all known haloarchaeal hsp70 genes. Here, we examined the effects of nucleotides flanking the start codon on gene expression. The results revealed that shortening and deletion of the short 5'-UTR enhanced transcript levels; however, it led to significant reductions in overall translational efficiency. AUG was efficiently used as start codons, in both the presence and absence of short 5'-UTRs. GUG also could initiate translation, even though it was so inefficient that it would not be detected without considerably elevated transcript. Nucleotide substitutions at position -4 to +6 were shown to affect gene expression by transcript and/or translational levels. Notably, -3A and A/U nucleotides at position +4+6 were more optimal for gene expression. Nucleotide transversions of -3A to -3C and +4G to +4T with hsp70 promoter from either Haloferax volcanii DS70 or Halobacterium salinarum NRC-1 showed the same effects on gene expression as that of Natrinema sp. J7. Taken together, our results suggest that the nucleotides flanking the start codon in hsp70 mRNAs with very short 5'-UTRs play an important role in haloarchaeal gene expression.<br/>
</details><br/>
---

*Data in support of global role of the membrane protease LonB in Archaea: Potential protease targets revealed by quantitative proteome analysis of a lonB mutant in Haloferax volcanii.*<br/>
Cerletti M; Paggi RA; Ramallo Guevara C; Poetsch A; De Castro RE<br/>
**2015**<br/>
Data Brief 4(None)<br/>
10.1016/j.dib.2015.04.013
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26217762)<br/>
[Citation]()<br/>
This data article provides information in support of the research article "Global role of the membrane protease LonB in Archaea: Potential protease targets revealed by quantitative proteome analysis of a lonB mutant in Haloferax volcanii" [1]. The proteome composition of a wt and a LonB protease mutant strain (suboptimal expression) in the archaeon Haloferax volcanii was assessed by a quantitative shotgun proteomic approach. Membrane and cytosol fractions of H. volcanii strains were examined at two different growth stages (exponential and stationary phase). Data is supplied in the present article. This study represents the first proteome examination of a Lon-deficient cell of the Archaea Domain. <br/>
</details><br/>
---

*The role of Cas8 in type I CRISPR interference.*<br/>
Cass SD; Haas KA; Stoll B; Alkhnbashi OS; Sharma K; Urlaub H; Backofen R; Marchfelder A; Bolt EL<br/>
**2015**<br/>
Biosci. Rep. 35(3)<br/>
10.1042/BSR20150043
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26182359)<br/>
[Citation]()<br/>
CRISPR (clustered regularly interspaced short palindromic repeat) systems provide bacteria and archaea with adaptive immunity to repel invasive genetic elements. Type I systems use 'cascade' [CRISPR-associated (Cas) complex for antiviral defence] ribonucleoprotein complexes to target invader DNA, by base pairing CRISPR RNA (crRNA) to protospacers. Cascade identifies PAMs (protospacer adjacent motifs) on invader DNA, triggering R-loop formation and subsequent DNA degradation by Cas3. Cas8 is a candidate PAM recognition factor in some cascades. We analysed Cas8 homologues from type IB CRISPR systems in archaea Haloferax volcanii (Hvo) and Methanothermobacter thermautotrophicus (Mth). Cas8 was essential for CRISPR interference in Hvo and purified Mth Cas8 protein responded to PAM sequence when binding to nucleic acids. Cas8 interacted physically with Cas5-Cas7-crRNA complex, stimulating binding to PAM containing substrates. Mutation of conserved Cas8 amino acid residues abolished interference in vivo and altered catalytic activity of Cas8 protein in vitro. This is experimental evidence that Cas8 is important for targeting Cascade to invader DNA.<br/>
</details><br/>
---

*Ubiquitin-Like Protein SAMP1 and JAMM/MPN+ Metalloprotease HvJAMM1 Constitute a System for Reversible Regulation of Metabolic Enzyme Activity in Archaea.*<br/>
Cao S; Hepowit N; Maupin-Furlow JA<br/>
**2015**<br/>
PLoS ONE 10(5)<br/>
10.1371/journal.pone.0128399
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26010867)<br/>
[Citation]()<br/>
Ubiquitin/ubiquitin-like (Ub/Ubl) proteins are involved in diverse cellular processes by their covalent linkage to protein substrates. Here, we provide evidence for a post-translational modification system that regulates enzyme activity which is composed of an archaeal Ubl protein (SAMP1) and a JAMM/MPN+ metalloprotease (HvJAMM1). Molybdopterin (MPT) synthase activity was found to be inhibited by covalent linkage of SAMP1 to the large subunit (MoaE) of MPT synthase. HvJAMM1 was shown to cleave the covalently linked inactive form of SAMP1-MoaE to the free functional individual SAMP1 and MoaE subunits of MPT synthase, suggesting reactivation of MPT synthase by this metalloprotease. Overall, this study provides new insight into the broad idea that Ub/Ubl modification is a post-translational process that can directly and reversibly regulate the activity of metabolic enzymes. In particular, we show that Ub/Ubl linkages on the active site residues of an enzyme (MPT synthase) can inhibit its catalytic activity and that the enzyme can be reactivated through cleavage by a JAMM/MPN+ metalloprotease. <br/>
</details><br/>
---

*Global role of the membrane protease LonB in Archaea: Potential protease targets revealed by quantitative proteome analysis of a lonB mutant in Haloferax volcanii.*<br/>
Cerletti M; Paggi RA; Guevara CR; Poetsch A; De Castro RE<br/>
**2015**<br/>
J Proteomics 121(None)<br/>
10.1016/j.jprot.2015.03.016
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25829260)<br/>
[Citation]()<br/>
UNLABELLED: The membrane-associated LonB protease is essential for viability in Haloferax volcanii, however, the cellular processes affected by this protease in archaea are unknown. In this study, the impact of a lon conditional mutation (down-regulation) on H. volcanii physiology was examined by comparing proteomes of parental and mutant cells using shotgun proteomics. A total of 1778 proteins were identified (44% of H. volcanii predicted proteome) and 142 changed significantly in amount (≥2 fold). Of these, 66 were augmented in response to Lon deficiency suggesting they could be Lon substrates. The "Lon subproteome" included soluble and predicted membrane proteins expected to participate in diverse cellular processes. The dramatic stabilization of phytoene synthase (57 fold) in concert with overpigmentation of lon mutant cells suggests that Lon controls carotenogenesis in H. volcanii. Several hypothetical proteins, which may reveal novel functions and/or be involved in adaptation to extreme environments, were notably increased (300 fold). This study, which represents the first proteome examination of a Lon deficient archaeal cell, shows that Lon has a strong impact on H. volcanii physiology evidencing the cellular processes controlled by this protease in Archaea. Additionally, this work provides a platform for the discovery of novel targets of Lon proteases.
BIOLOGICAL SIGNIFICANCE: The proteome of a Lon-deficient archaeal cell was examined for the first time showing that Lon has a strong impact on H. volcanii physiology and evidencing the proteins and cellular processes controlled by this protease in Archaea. This work will facilitate future investigations aiming to address Lon function in archaea and provides a platform for the discovery of endogenous targets of the archaeal-type Lon as well as novel targets/processes regulated by Lon proteases. This knowledge will advance the understanding on archaeal physiology and the biological function of membrane proteases in microorganisms.<br/>
</details><br/>
---

*Nonsense suppression in archaea.*<br/>
Bhattacharya A; Köhrer C; Mandal D; RajBhandary UL<br/>
**2015**<br/>
Proc. Natl. Acad. Sci. U.S.A. 112(19)<br/>
10.1073/pnas.1501558112
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25918386)<br/>
[Citation]()<br/>
Bacterial strains carrying nonsense suppressor tRNA genes played a crucial role in early work on bacterial and bacterial viral genetics. In eukaryotes as well, suppressor tRNAs have played important roles in the genetic analysis of yeast and worms. Surprisingly, little is known about genetic suppression in archaea, and there has been no characterization of suppressor tRNAs or identification of nonsense mutations in any of the archaeal genes. Here, we show, using the β-gal gene as a reporter, that amber, ochre, and opal suppressors derived from the serine and tyrosine tRNAs of the archaeon Haloferax volcanii are active in suppression of their corresponding stop codons. Using a promoter for tRNA expression regulated by tryptophan, we also show inducible and regulatable suppression of all three stop codons in H. volcanii. Additionally, transformation of a ΔpyrE2 H. volcanii strain with plasmids carrying the genes for a pyrE2 amber mutant and the serine amber suppressor tRNA yielded transformants that grow on agar plates lacking uracil. Thus, an auxotrophic amber mutation in the pyrE2 gene can be complemented by expression of the amber suppressor tRNA. These results pave the way for generating archaeal strains carrying inducible suppressor tRNA genes on the chromosome and their use in archaeal and archaeviral genetics. We also provide possible explanations for why suppressor tRNAs have not been identified in archaea. <br/>
</details><br/>
---

*Permuting the PGF Signature Motif Blocks both Archaeosortase-Dependent C-Terminal Cleavage and Prenyl Lipid Attachment for the Haloferax volcanii S-Layer Glycoprotein.*<br/>
Abdul Halim MF; Karch KR; Zhou Y; Haft DH; Garcia BA; Pohlschroder M<br/>
**2015**<br/>
J. Bacteriol. 198(5)<br/>
10.1128/JB.00849-15
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/26712937)<br/>
[Citation]()<br/>
UNLABELLED: For years, the S-layer glycoprotein (SLG), the sole component of many archaeal cell walls, was thought to be anchored to the cell surface by a C-terminal transmembrane segment. Recently, however, we demonstrated that the Haloferax volcanii SLG C terminus is removed by an archaeosortase (ArtA), a novel peptidase. SLG, which was previously shown to be lipid modified, contains a C-terminal tripartite structure, including a highly conserved proline-glycine-phenylalanine (PGF) motif. Here, we demonstrate that ArtA does not process an SLG variant where the PGF motif is replaced with a PFG motif (slg(G796F,F797G)). Furthermore, using radiolabeling, we show that SLG lipid modification requires the PGF motif and is ArtA dependent, lending confirmation to the use of a novel C-terminal lipid-mediated protein-anchoring mechanism by prokaryotes. Similar to the case for the ΔartA strain, the growth, cellular morphology, and cell wall of the slg(G796F,F797G) strain, in which modifications of additional H. volcanii ArtA substrates should not be altered, are adversely affected, demonstrating the importance of these posttranslational SLG modifications. Our data suggest that ArtA is either directly or indirectly involved in a novel proteolysis-coupled, covalent lipid-mediated anchoring mechanism. Given that archaeosortase homologs are encoded by a broad range of prokaryotes, it is likely that this anchoring mechanism is widely conserved.
IMPORTANCE: Prokaryotic proteins bound to cell surfaces through intercalation, covalent attachment, or protein-protein interactions play critical roles in essential cellular processes. Unfortunately, the molecular mechanisms that anchor proteins to archaeal cell surfaces remain poorly characterized. Here, using the archaeon H. volcanii as a model system, we report the first in vivo studies of a novel protein-anchoring pathway involving lipid modification of a peptidase-processed C terminus. Our findings not only yield important insights into poorly understood aspects of archaeal biology but also have important implications for key bacterial species, including those of the human microbiome. Additionally, insights may facilitate industrial applications, given that photosynthetic cyanobacteria encode uncharacterized homologs of this evolutionarily conserved enzyme, or may spur development of unique drug delivery systems.<br/>
</details><br/>
---

*Polyploidy in haloarchaea: advantages for growth and survival.*<br/>
Zerulla K; Soppa J<br/>
**2014**<br/>
Front Microbiol 5(None)<br/>
10.3389/fmicb.2014.00274
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24982654)<br/>
[Citation]()<br/>
The investigated haloarchaeal species, Halobacterium salinarum, Haloferax mediterranei, and H. volcanii, have all been shown to be polyploid. They contain several replicons that have independent copy number regulation, and most have a higher copy number during exponential growth phase than in stationary phase. The possible evolutionary advantages of polyploidy for haloarchaea, most of which have experimental support for at least one species, are discussed. These advantages include a low mutation rate and high resistance toward X-ray irradiation and desiccation, which depend on homologous recombination. For H. volcanii, it has been shown that gene conversion operates in the absence of selection, which leads to the equalization of genome copies. On the other hand, selective forces might lead to heterozygous cells, which have been verified in the laboratory. Additional advantages of polyploidy are survival over geological times in halite deposits as well as at extreme conditions on earth and at simulated Mars conditions. Recently, it was found that H. volcanii uses genomic DNA as genetic material and as a storage polymer for phosphate. In the absence of phosphate, H. volcanii dramatically decreases its genome copy number, thereby enabling cell multiplication, but diminishing the genetic advantages of polyploidy. Stable storage of phosphate is proposed as an alternative driving force for the emergence of DNA in early evolution. Several additional potential advantages of polyploidy are discussed that have not been addressed experimentally for haloarchaea. An outlook summarizes selected current trends and possible future developments. <br/>
</details><br/>
---

*DNA as a phosphate storage polymer and the alternative advantages of polyploidy for growth or survival.*<br/>
Zerulla K; Chimileski S; Näther D; Gophna U; Papke RT; Soppa J<br/>
**2014**<br/>
PLoS ONE 9(4)<br/>
10.1371/journal.pone.0094819
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24733558)<br/>
[Citation]()<br/>
Haloferax volcanii uses extracellular DNA as a source for carbon, nitrogen, and phosphorous. However, it can also grow to a limited extend in the absence of added phosphorous, indicating that it contains an intracellular phosphate storage molecule. As Hfx. volcanii is polyploid, it was investigated whether DNA might be used as storage polymer, in addition to its role as genetic material. It could be verified that during phosphate starvation cells multiply by distributing as well as by degrading their chromosomes. In contrast, the number of ribosomes stayed constant, revealing that ribosomes are distributed to descendant cells, but not degraded. These results suggest that the phosphate of phosphate-containing biomolecules (other than DNA and RNA) originates from that stored in DNA, not in rRNA. Adding phosphate to chromosome depleted cells rapidly restores polyploidy. Quantification of desiccation survival of cells with different ploidy levels showed that under phosphate starvation Hfx. volcanii diminishes genetic advantages of polyploidy in favor of cell multiplication. The consequences of the usage of genomic DNA as phosphate storage polymer are discussed as well as the hypothesis that DNA might have initially evolved in evolution as a storage polymer, and the various genetic benefits evolved later. <br/>
</details><br/>
---

*Identification in Haloferax volcanii of phosphomevalonate decarboxylase and isopentenyl phosphate kinase as catalysts of the terminal enzyme reactions in an archaeal alternate mevalonate pathway.*<br/>
Vannice JC; Skaff DA; Keightley A; Addo JK; Wyckoff GJ; Miziorko HM<br/>
**2014**<br/>
J. Bacteriol. 196(5)<br/>
10.1128/JB.01230-13
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24375100)<br/>
[Citation]()<br/>
Mevalonate (MVA) metabolism provides the isoprenoids used in archaeal lipid biosynthesis. In synthesis of isopentenyl diphosphate, the classical MVA pathway involves decarboxylation of mevalonate diphosphate, while an alternate pathway has been proposed to involve decarboxylation of mevalonate monophosphate. To identify the enzymes responsible for metabolism of mevalonate 5-phosphate to isopentenyl diphosphate in Haloferax volcanii, two open reading frames (HVO_2762 and HVO_1412) were selected for expression and characterization. Characterization of these proteins indicated that one enzyme is an isopentenyl phosphate kinase that forms isopentenyl diphosphate (in a reaction analogous to that of Methanococcus jannaschii MJ0044). The second enzyme exhibits a decarboxylase activity that has never been directly attributed to this protein or any homologous protein. It catalyzes the synthesis of isopentenyl phosphate from mevalonate monophosphate, a reaction that has been proposed but never demonstrated by direct experimental proof, which is provided in this account. This enzyme, phosphomevalonate decarboxylase (PMD), exhibits strong inhibition by 6-fluoromevalonate monophosphate but negligible inhibition by 6-fluoromevalonate diphosphate (a potent inhibitor of the classical mevalonate pathway), reinforcing its selectivity for monophosphorylated ligands. Inhibition by the fluorinated analog also suggests that the PMD utilizes a reaction mechanism similar to that demonstrated for the classical MVA pathway decarboxylase. These observations represent the first experimental demonstration in H. volcanii of both the phosphomevalonate decarboxylase and isopentenyl phosphate kinase reactions that are required for an alternate mevalonate pathway in an archaeon. These results also represent, to our knowledge, the first identification and characterization of any phosphomevalonate decarboxylase. <br/>
</details><br/>
---

*Diversity of the biosynthesis pathway for threonylcarbamoyladenosine (t(6)A), a universal modification of tRNA.*<br/>
Thiaville PC; Iwata-Reuyl D; de Crécy-Lagard V<br/>
**2014**<br/>
RNA Biol 11(12)<br/>
10.4161/15476286.2014.992277
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25629598)<br/>
[Citation]()<br/>
The tRNA modification field has a rich literature covering biochemical analysis going back more than 40 years, but many of the corresponding genes were only identified in the last decade. In recent years, comparative genomic-driven analysis has allowed for the identification of the genes and subsequent characterization of the enzymes responsible for N6-threonylcarbamoyladenosine (t(6)A). This universal modification, located in the anticodon stem-loop at position 37 adjacent to the anticodon of tRNAs, is found in nearly all tRNAs that decode ANN codons. The t(6)A biosynthesis enzymes and synthesis pathways have now been identified, revealing both a core set of enzymes and kingdom-specific variations. This review focuses on the elucidation of the pathway, diversity of the synthesis genes, and proposes a new nomenclature for t(6)A synthesis enzymes. <br/>
</details><br/>
---

*The accessory gas vesicle protein GvpM of haloarchaea and its interaction partners during gas vesicle formation.*<br/>
Tavlaridou S; Winter K; Pfeifer F<br/>
**2014**<br/>
Extremophiles 18(4)<br/>
10.1007/s00792-014-0650-0
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24846741)<br/>
[Citation]()<br/>
Gas vesicles consist predominantly of the hydrophobic GvpA and GvpC, and the accessory proteins GvpF through GvpM are required in minor amounts during formation. GvpM and its putative interaction partners were investigated. GvpM interacted with GvpH, GvpJ and GvpL, but not with GvpG. Interactions were also observed in vivo in Haloferax volcanii transformants using Gvp fusions to the green fluorescent protein smGFP. Cells producing the hydrophobic M(GF)P contained a single fluorescent aggregate per cell, whereas cells containing L(GFP) or H(GFP) were fully fluorescent. The soluble L(GFP) formed stable co-aggregates with GvpM in L(GFP)M transformants, but the presence of GvpH resulted in the absence of M(GF)P foci in HM(GFP) transformants. Substitution- and deletion mutants of GvpM determined functionally important amino acids (aa). Substitution of a polar by a non-polar aa in the N-terminal region of GvpM had no effect, whereas a substitution of a non-polar by a polar aa in this region inhibited gas vesicle formation in transformants. Substitutions in region 44-48 of GvpM strongly reduced the number of gas vesicles, and deletions at the N-terminus resulted in Vac(-) transformants. Gas vesicle morphology was not affected by any mutation, implying that GvpM is required during initial stages of gas vesicle assembly.<br/>
</details><br/>
---

*Pseudouridine: still mysterious, but never a fake (uridine)!*<br/>
Spenkuch F; Motorin Y; Helm M<br/>
**2014**<br/>
RNA Biol 11(12)<br/>
10.4161/15476286.2014.992278
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25616362)<br/>
[Citation]()<br/>
Pseudouridine (Ψ) is the most abundant of >150 nucleoside modifications in RNA. Although Ψ was discovered as the first modified nucleoside more than half a century ago, neither the enzymatic mechanism of its formation, nor the function of this modification are fully elucidated. We present the consistent picture of Ψ synthases, their substrates and their substrate positions in model organisms of all domains of life as it has emerged to date and point out the challenges that remain concerning higher eukaryotes and the elucidation of the enzymatic mechanism. <br/>
</details><br/>
---

*Polyploidy in archaea and bacteria: about desiccation resistance, giant cell size, long-term survival, enforcement by a eukaryotic host and additional aspects.*<br/>
Soppa J<br/>
**2014**<br/>
J. Mol. Microbiol. Biotechnol. 24(5-6)<br/>
10.1159/000368855
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25732342)<br/>
[Citation]()<br/>
During recent years, it has become clear that many species of archaea and bacteria are polyploid and contain more than 10 copies of their chromosome. In this contribution, eight examples are discussed to highlight different aspects of polyploidy in prokaryotes. The species discussed are the bacteria Azotobacter vinelandii, Deinococcus radiodurans, Sinorhizobium meliloti, and Epulopiscium as well as the archaea Methanocaldococcus jannaschii, Methanococcus maripaludis, Haloferax volcanii, and haloarchaeal isolates from salt deposits. The topics include possible laboratory artifacts, resistance against double-strand breaks, long-term survival, relaxation of DNA segregation and septum formation, enforced polyploidy by a eukaryotic host, genome equalization by gene conversion, and the nongenetic usage of genomic DNA as a phosphate storage polymer. Together, the selected topics give an overview of the biodiversity of polyploidy in archaea and bacteria.<br/>
</details><br/>
---

*Identification and characterization of 2-keto-3-deoxygluconate kinase and 2-keto-3-deoxygalactonate kinase in the haloarchaeon Haloferax volcanii.*<br/>
Pickl A; Johnsen U; Archer RM; Schönheit P<br/>
**2014**<br/>
FEMS Microbiol. Lett. 361(1)<br/>
10.1111/1574-6968.12617
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25287957)<br/>
[Citation]()<br/>
The halophilic archaeon Haloferax volcanii has been proposed to degrade glucose via the semi-phosphorylative Entner-Doudoroff pathway, involving 2-keto-3-deoxygluconate kinase (KDGK) as key enzyme. So far, neither the enzyme has been characterized nor the encoding gene has been identified. In the genome of H. volcanii, two genes, HVO_0549 (kdgK1) and HVO_A0328 (kdgK2), are annotated encoding putative KDGK-1 and KDGK-2. To identify the physiological role of both kinases, transcriptional regulation analyses of both genes and growth experiments of the respective deletion mutants were performed on different sugars. Further, recombinant KDGK-1 and KDGK-2 were characterized. Together, the data indicate that KDGK-1 represents the functional constitutively expressed KDG kinase in glucose degradation, whereas KDGK-2 is an inducible 2-keto-3-deoxygalactonate kinase likely involved in d-galactose catabolism.<br/>
</details><br/>
---

*A simple technique to improve the resolution of membrane acidic proteins of the haloarchaeon Haloferax volcanii by 2D electrophoresis.*<br/>
Paggi RA; Giménez MI; De Castro RE; Cesari A<br/>
**2014**<br/>
Electrophoresis 35(24)<br/>
10.1002/elps.201400407
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25224925)<br/>
[Citation]()<br/>
Proteins present in the archaeal cell envelope play key roles in a variety of processes necessary for survival in extreme environments. The haloarchaeon Haloferax volcanii is a good model for membrane proteomic studies because its genome sequence is known, it can be genetically manipulated, and a number of studies at the "omics" level have been performed in this organism. This work reports an easy strategy to improve the resolution of acidic membrane proteins from H. volcanii by 2DE. The method is based on the solubilization, delipidation, and salt removal from membrane proteins. Due to the abundance of the S-layer glycoprotein (SLG) in membrane protein extracts, other proteins from the envelope are consequently underrepresented. Thus, a protocol to reduce the amount of the SLG by EDTA treatment was applied and 11 cm narrow range pH (3.9-5.1) IPG strips were used to fractionate the remaining proteins. Using this method, horizontal streaking was substantially decreased and at least 75 defined spots (20% of the predicted membrane proteome within this pI/Mw range) were reproducibly detected. Two of these spots were identified as thermosome subunit 1 and NADH dehydrogenase from H. volcanii, confirming that proteins from the membrane fraction were enriched. Removal of the SLG from membrane protein extracts can be applied to increase protein load for 2DE as well as for other proteomic methods. <br/>
</details><br/>
---

*A rhomboid protease gene deletion affects a novel oligosaccharide N-linked to the S-layer glycoprotein of Haloferax volcanii.*<br/>
Parente J; Casabuono A; Ferrari MC; Paggi RA; De Castro RE; Couto AS; Giménez MI<br/>
**2014**<br/>
J. Biol. Chem. 289(16)<br/>
10.1074/jbc.M113.546531
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24596091)<br/>
[Citation]()<br/>
Rhomboid proteases occur in all domains of life; however, their physiological role is not completely understood, and nothing is known of the biology of these enzymes in Archaea. One of the two rhomboid homologs of Haloferax volcanii (RhoII) is fused to a zinc finger domain. Chromosomal deletion of rhoII was successful, indicating that this gene is not essential for this organism; however, the mutant strain (MIG1) showed reduced motility and increased sensitivity to novobiocin. Membrane preparations of MIG1 were enriched in two glycoproteins, identified as the S-layer glycoprotein and an ABC transporter component. The H. volcanii S-layer glycoprotein has been extensively used as a model to study haloarchaeal protein N-glycosylation. HPLC analysis of oligosaccharides released from the S-layer glycoprotein after PNGase treatment revealed that MIG1 was enriched in species with lower retention times than those derived from the parent strain. Mass spectrometry analysis showed that the wild type glycoprotein released a novel oligosaccharide species corresponding to GlcNAc-GlcNAc(Hex)2-(SQ-Hex)6 in contrast to the mutant protein, which contained the shorter form GlcNAc2(Hex)2-SQ-Hex-SQ. A glycoproteomics approach of the wild type glycopeptide fraction revealed Asn-732 peptide fragments linked to the sulfoquinovose-containing oligosaccharide. This work describes a novel N-linked oligosaccharide containing a repeating SQ-Hex unit bound to Asn-732 of the H. volcanii S-layer glycoprotein, a position that had not been reported as glycosylated. Furthermore, this study provides the first insight on the biological role of rhomboid proteases in Archaea, suggesting a link between protein glycosylation and this protease family. <br/>
</details><br/>
---

*Structural and biochemical properties of an extreme 'salt-loving' proteasome activating nucleotidase from the archaeon Haloferax volcanii.*<br/>
Prunetti L; Reuter CJ; Hepowit NL; Wu Y; Barrueto L; Miranda HV; Kelly K; Maupin-Furlow JA<br/>
**2014**<br/>
Extremophiles 18(2)<br/>
10.1007/s00792-013-0615-8
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24343376)<br/>
[Citation]()<br/>
In eukaryotes, the 26S proteasome degrades ubiquitinylated proteins in an ATP-dependent manner. Archaea mediate a form of post-translational modification of proteins termed sampylation that resembles ubiquitinylation. Sampylation was identified in Haloferax volcanii, a moderate halophilic archaeon that synthesizes homologs of 26S proteasome subunits including 20S core particles and regulatory particle triple-A ATPases (Rpt)-like proteasome-associated nucleotidases (PAN-A/1 and PAN-B/2). To determine whether sampylated proteins associate with the Rpt subunit homologs, PAN-A/1 was purified to homogeneity from Hfx. volcanii and analyzed for its subunit stoichiometry, nucleotide-hydrolyzing activity and binding to sampylated protein targets. PAN-A/1 was found to be associated as a dodecamer (630 kDa) with a configuration in TEM suggesting a complex of two stacked hexameric rings. PAN-A/1 had high affinity for ATP (K m of 0.44 mM) and hydrolyzed this nucleotide with a specific activity of 0.33 ± 0.1 μmol Pi/h per mg protein and maximum at 42 °C. PAN-A1 was stabilized by 2 M salt with a decrease in activity at lower concentrations of salt that correlated with dissociation of the dodecamer into trimers to monomers. Binding of PAN-A/1 to a sampylated protein was demonstrated by modification of a far Western blotting technique (derived from the standard Western blot method to detect protein-protein interaction in vitro) for halophilic proteins. Overall, our results support a model in which sampylated proteins associate with the PAN-A/1 AAA+ ATPase in proteasome-mediated proteolysis and/or protein remodeling and provide a method for assay of halophilic protein-protein interactions. <br/>
</details><br/>
---

*The effects of extremes of pH on the growth and transcriptomic profiles of three haloarchaea.*<br/>
Moran-Reyna A; Coker JA<br/>
**2014**<br/>
F1000Res 3(None)<br/>
10.12688/f1000research.4789.2
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25285207)<br/>
[Citation]()<br/>
The halophilic archaea (haloarchaea) live in saline environments, which are found across the globe.  In addition to salinity, these niches can be quite dynamic and experience extreme conditions such as low oxygen content, radiation (gamma and UV), pH and temperature.  However, of all the naturally occurring stresses faced by the haloarchaea, only one, pH, has not been previously investigated in regard to the changes induced in the transcriptome. Therefore, we endeavored to determine the responses in three haloarchaea: Halorubrum lacusprofundi (Hla), Haloferax volcanii (Hvo), and Halobacterium sp. NRC-1 (NRC-1) to growth under acidic and alkaline pH. Our observations showed that the transcriptomes of Hvo and NRC-1 regulated stress, motility, and ABC transporters in a similar manner, which is in line with previous reports from other prokaryotes when grown in an acidic environment.  However, the pattern for Hla was more species specific. For alkaline stress, all three haloarchaea responded in a manner similar to well-studied archaea and bacteria showing the haloarchaeal response was general to prokaryotes. Additionally, we performed an analysis on the changes in the transcriptomes of the three haloarchaea when shifting from one pH extreme to the other. The results showed that the transcriptomes of all three haloarchaea respond more similarly when moving from alkaline to acidic conditions compared to a shift in the opposite direction. Interestingly, our studies also showed that individual genes of multiple paralogous gene families ( tbp, tfb, orc/ cdc6, etc.) found in the haloarchaea were regulated under specific stresses thereby providing evidence that they modulate the response to various environmental stresses. The studies described here are the first to catalog the changes in the haloarchaeal transcriptomes under growth in extreme pH and help us understand how life is able to thrive under all conditions present on Earth and, if present, on extraterrestrial bodies as well. <br/>
</details><br/>
---

*Chromosome replication origins: do we really need them?*<br/>
Michel B; Bernander R<br/>
**2014**<br/>
Bioessays 36(6)<br/>
10.1002/bies.201400003
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24644021)<br/>
[Citation]()<br/>
Replication of the main chromosome in the halophilic archaeon Haloferax volcanii was recently reported to continue despite deletion of all active replication origins. Equally surprising, the deletion strain grew faster than the parent strain. It was proposed that origin-less H. volcanii duplicate their chromosomes via recombination-dependent replication. Here, we recall our present knowledge of this mode of chromosome replication in different organisms. We consider the likelihood that it accounts for the viability of H. volcanii deleted for its main specific replication origins, as well as possible alternative interpretations of the results. The selective advantages of having defined chromosome replication origins are discussed from a functional and evolutionary perspective. <br/>
</details><br/>
---

*Light-dependent expression of four cryptic archaeal circadian gene homologs.*<br/>
Maniscalco M; Nannen J; Sodi V; Silver G; Lowrey PL; Bidle KA<br/>
**2014**<br/>
Front Microbiol 5(None)<br/>
10.3389/fmicb.2014.00079
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24624125)<br/>
[Citation]()<br/>
Circadian rhythms are important biological signals that have been found in almost all major groups of life from bacteria to man, yet it remains unclear if any members of the second major prokaryotic domain of life, the Archaea, also possess a biological clock. As an initial investigation of this question, we examined the regulation of four cyanobacterial-like circadian gene homologs present in the genome of the haloarchaeon Haloferax volcanii. These genes, designated cirA, cirB, cirC, and cirD, display similarity to the KaiC-family of cyanobacterial clock proteins, which act to regulate rhythmic gene expression and to control the timing of cell division. Quantitative RT-PCR analysis was used to examine the expression of each of the four cir genes in response to 12 h light/12 h dark cycles (LD 12:12) in H. volcanii during balanced growth. Our data reveal that there is an approximately two to sixteen-fold increase in cir gene expression when cells are shifted from light to constant darkness, and this pattern of gene expression oscillates with the light conditions in a rhythmic manner. Targeted single- and double-gene knockouts in the H. volcanii cir genes result in disruption of light-dependent, rhythmic gene expression, although it does not lead to any significant effect on growth under these conditions. Restoration of light-dependent, rhythmic gene expression was demonstrated by introducing, in trans, a wild-type copy of individual cir genes into knockout strains. These results are noteworthy as this is the first attempt to characterize the transcriptional expression and regulation of the ubiquitous kaiC homologs found among archaeal genomes. <br/>
</details><br/>
---

*Identification and codon reading properties of 5-cyanomethyl uridine, a new modified nucleoside found in the anticodon wobble position of mutant haloarchaeal isoleucine tRNAs.*<br/>
Mandal D; Köhrer C; Su D; Babu IR; Chan CT; Liu Y; Söll D; Blum P; Kuwahara M; Dedon PC; Rajbhandary UL<br/>
**2014**<br/>
RNA 20(2)<br/>
10.1261/rna.042358.113
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24344322)<br/>
[Citation]()<br/>
Most archaea and bacteria use a modified C in the anticodon wobble position of isoleucine tRNA to base pair with A but not with G of the mRNA. This allows the tRNA to read the isoleucine codon AUA without also reading the methionine codon AUG. To understand why a modified C, and not U or modified U, is used to base pair with A, we mutated the C34 in the anticodon of Haloarcula marismortui isoleucine tRNA (tRNA2(Ile)) to U, expressed the mutant tRNA in Haloferax volcanii, and purified and analyzed the tRNA. Ribosome binding experiments show that although the wild-type tRNA2(Ile) binds exclusively to the isoleucine codon AUA, the mutant tRNA binds not only to AUA but also to AUU, another isoleucine codon, and to AUG, a methionine codon. The G34 to U mutant in the anticodon of another H. marismortui isoleucine tRNA species showed similar codon binding properties. Binding of the mutant tRNA to AUG could lead to misreading of the AUG codon and insertion of isoleucine in place of methionine. This result would explain why most archaea and bacteria do not normally use U or a modified U in the anticodon wobble position of isoleucine tRNA for reading the codon AUA. Biochemical and mass spectrometric analyses of the mutant tRNAs have led to the discovery of a new modified nucleoside, 5-cyanomethyl U in the anticodon wobble position of the mutant tRNAs. 5-Cyanomethyl U is present in total tRNAs from euryarchaea but not in crenarchaea, eubacteria, or eukaryotes. <br/>
</details><br/>
---

*Archaeal ubiquitin-like SAMP3 is isopeptide-linked to proteins via a UbaA-dependent mechanism.*<br/>
Miranda HV; Antelmann H; Hepowit N; Chavarria NE; Krause DJ; Pritz JR; Bäsell K; Becher D; Humbard MA; Brocchieri L; Maupin-Furlow JA<br/>
**2014**<br/>
Mol. Cell Proteomics 13(1)<br/>
10.1074/mcp.M113.029652
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24097257)<br/>
[Citation]()<br/>
SAMP1 and SAMP2 are ubiquitin-like proteins that function as protein modifiers and are required for the production of sulfur-containing biomolecules in the archaeon Haloferax volcanii. Here we report a novel small archaeal modifier protein (named SAMP3) with a β-grasp fold and C-terminal diglycine motif characteristic of ubiquitin that is functional in protein conjugation in Hfx. volcanii. SAMP3 conjugates were dependent on the ubiquitin-activating E1 enzyme homolog of archaea (UbaA) for synthesis and were cleaved by the JAMM/MPN+ domain metalloprotease HvJAMM1. Twenty-three proteins (28 lysine residues) were found to be isopeptide-linked to the C-terminal carboxylate of SAMP3, and 331 proteins were reproducibly found associated with SAMP3 in a UbaA-dependent manner based on tandem mass spectrometry (MS/MS) analysis. The molybdopterin (MPT) synthase large subunit homolog MoaE, found samp3ylated at conserved active site lysine residues in MS/MS analysis, was also shown to be covalently bound to SAMP3 by immunoprecipitation and tandem affinity purifications. HvJAMM1 was demonstrated to catalyze the cleavage of SAMP3 from MoaE, suggesting a mechanism of controlling MPT synthase activity. The levels of samp3ylated proteins and samp3 transcripts were found to be increased by the addition of dimethyl sulfoxide to aerobically growing cells. Thus, we propose a model in which samp3ylation is covalent and reversible and controls the activity of enzymes such as MPT synthase. Sampylation of MPT synthase may govern the levels of molybdenum cofactor available and thus facilitate the scavenging of oxygen prior to the transition to respiration with molybdenum-cofactor-containing terminal reductases that use alternative electron acceptors such as dimethyl sulfoxide. Overall, our study of SAMP3 provides new insight into the diversity of functional ubiquitin-like protein modifiers and the network of ubiquitin-like protein targets in Archaea. <br/>
</details><br/>
---

*Generation of comprehensive transposon insertion mutant library for the model archaeon, Haloferax volcanii, and its use for gene discovery.*<br/>
Kiljunen S; Pajunen MI; Dilks K; Storf S; Pohlschroder M; Savilahti H<br/>
**2014**<br/>
BMC Biol. 12(None)<br/>
10.1186/s12915-014-0103-3
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25488358)<br/>
[Citation]()<br/>
BACKGROUND: Archaea share fundamental properties with bacteria and eukaryotes. Yet, they also possess unique attributes, which largely remain poorly characterized. Haloferax volcanii is an aerobic, moderately halophilic archaeon that can be grown in defined media. It serves as an excellent archaeal model organism to study the molecular mechanisms of biological processes and cellular responses to changes in the environment. Studies on haloarchaea have been impeded by the lack of efficient genetic screens that would facilitate the identification of protein functions and respective metabolic pathways.
RESULTS: Here, we devised an insertion mutagenesis strategy that combined Mu in vitro DNA transposition and homologous-recombination-based gene targeting in H. volcanii. We generated an insertion mutant library, in which the clones contained a single genomic insertion. From the library, we isolated pigmentation-defective and auxotrophic mutants, and the respective insertions pinpointed a number of genes previously known to be involved in carotenoid and amino acid biosynthesis pathways, thus validating the performance of the methodologies used. We also identified mutants that had a transposon insertion in a gene encoding a protein of unknown or putative function, demonstrating that novel roles for non-annotated genes could be assigned.
CONCLUSIONS: We have generated, for the first time, a random genomic insertion mutant library for a halophilic archaeon and used it for efficient gene discovery. The library will facilitate the identification of non-essential genes behind any specific biochemical pathway. It represents a significant step towards achieving a more complete understanding of the unique characteristics of halophilic archaea.<br/>
</details><br/>
---

*Haloferax volcanii N-glycosylation: delineating the pathway of dTDP-rhamnose biosynthesis.*<br/>
Kaminski L; Eichler J<br/>
**2014**<br/>
PLoS ONE 9(5)<br/>
10.1371/journal.pone.0097441
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24831810)<br/>
[Citation]()<br/>
In the halophilic archaea Haloferax volcanii, the surface (S)-layer glycoprotein can be modified by two distinct N-linked glycans. The tetrasaccharide attached to S-layer glycoprotein Asn-498 comprises a sulfated hexose, two hexoses and a rhamnose. While Agl11-14 have been implicated in the appearance of the terminal rhamnose subunit, the precise roles of these proteins have yet to be defined. Accordingly, a series of in vitro assays conducted with purified Agl11-Agl14 showed these proteins to catalyze the stepwise conversion of glucose-1-phosphate to dTDP-rhamnose, the final sugar of the tetrasaccharide glycan. Specifically, Agl11 is a glucose-1-phosphate thymidylyltransferase, Agl12 is a dTDP-glucose-4,6-dehydratase and Agl13 is a dTDP-4-dehydro-6-deoxy-glucose-3,5-epimerase, while Agl14 is a dTDP-4-dehydrorhamnose reductase. Archaea thus synthesize nucleotide-activated rhamnose by a pathway similar to that employed by Bacteria and distinct from that used by Eukarya and viruses. Moreover, a bioinformatics screen identified homologues of agl11-14 clustered in other archaeal genomes, often as part of an extended gene cluster also containing aglB, encoding the archaeal oligosaccharyltransferase. This points to rhamnose as being a component of N-linked glycans in Archaea other than Hfx. volcanii. <br/>
</details><br/>
---

*Haloferax volcanii, a prokaryotic species that does not use the Shine Dalgarno mechanism for translation initiation at 5'-UTRs.*<br/>
Kramer P; Gäbel K; Pfeiffer F; Soppa J<br/>
**2014**<br/>
PLoS ONE 9(4)<br/>
10.1371/journal.pone.0094979
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24733188)<br/>
[Citation]()<br/>
It was long assumed that translation initiation in prokaryotes generally occurs via the so-called Shine Dalgarno (SD) mechanism. Recently, it became clear that translation initiation in prokaryotes is more heterogeneous. In the haloarchaeon Haloferax volcanii, the majority of transcripts is leaderless and most transcripts with a 5'-UTR lack a SD motif. Nevertheless, a bioinformatic analysis predicted that 20-30% of all genes are preceded by a SD motif in haloarchaea. To analyze the importance of the SD mechanism for translation initiation in haloarchaea experimentally the monocistronic sod gene was chosen, which contains a 5'-UTR with an extensive SD motif of seven nucleotides and a length of 19 nt, the average length of 5'UTRs in this organism. A translational fusion of part of the sod gene with the dhfr reporter gene was constructed. A mutant series was generated that matched the SD motif from zero to eight positions, respectively. Surprisingly, there was no correlation between the base pairing ability between transcripts and 16S rRNA and translational efficiency in vivo under several different growth conditions. Furthermore, complete replacement of the SD motif by three unrelated sequences did not reduce translational efficiency. The results indicate that H. volcanii does not make use of the SD mechanism for translation initiation in 5'-UTRs. A genome analysis revealed that while the number of SD motifs in 5'-UTRs is rare, their fraction within open reading frames is high. Possible biological functions for intragenic SD motifs are discussed, including re-initiation of translation at distal genes in operons. <br/>
</details><br/>
---

*The haloarchaeal MCM proteins: bioinformatic analysis and targeted mutagenesis of the β7-β8 and β9-β10 hairpin loops and conserved zinc binding domain cysteines.*<br/>
Kristensen TP; Maria Cherian R; Gray FC; MacNeill SA<br/>
**2014**<br/>
Front Microbiol 5(None)<br/>
10.3389/fmicb.2014.00123
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24723920)<br/>
[Citation]()<br/>
The hexameric MCM complex is the catalytic core of the replicative helicase in eukaryotic and archaeal cells. Here we describe the first in vivo analysis of archaeal MCM protein structure and function relationships using the genetically tractable haloarchaeon Haloferax volcanii as a model system. Hfx. volcanii encodes a single MCM protein that is part of the previously identified core group of haloarchaeal MCM proteins. Three structural features of the N-terminal domain of the Hfx. volcanii MCM protein were targeted for mutagenesis: the β7-β8 and β9-β10 β-hairpin loops and putative zinc binding domain. Five strains carrying single point mutations in the β7-β8 β-hairpin loop were constructed, none of which displayed impaired cell growth under normal conditions or when treated with the DNA damaging agent mitomycin C. However, short sequence deletions within the β7-β8 β-hairpin were not tolerated and neither was replacement of the highly conserved residue glutamate 187 with alanine. Six strains carrying paired alanine substitutions within the β9-β10 β-hairpin loop were constructed, leading to the conclusion that no individual amino acid within that hairpin loop is absolutely required for MCM function, although one of the mutant strains displays greatly enhanced sensitivity to mitomycin C. Deletions of two or four amino acids from the β9-β10 β-hairpin were tolerated but mutants carrying larger deletions were inviable. Similarly, it was not possible to construct mutants in which any of the conserved zinc binding cysteines was replaced with alanine, underlining the likely importance of zinc binding for MCM function. The results of these studies demonstrate the feasibility of using Hfx. volcanii as a model system for reverse genetic analysis of archaeal MCM protein function and provide important confirmation of the in vivo importance of conserved structural features identified by previous bioinformatic, biochemical and structural studies. <br/>
</details><br/>
---

*Generation and phenotyping of a collection of sRNA gene deletion mutants of the haloarchaeon Haloferax volcanii.*<br/>
Jaschinski K; Babski J; Lehr M; Burmester A; Benz J; Heyer R; Dörr M; Marchfelder A; Soppa J<br/>
**2014**<br/>
PLoS ONE 9(3)<br/>
10.1371/journal.pone.0090763
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24637842)<br/>
[Citation]()<br/>
The haloarchaeon Haloferax volcanii was shown to contain 145 intergenic and 45 antisense sRNAs. In a comprehensive approach to unravel various biological roles of haloarchaeal sRNAs in vivo, 27 sRNA genes were selected and deletion mutants were generated. The phenotypes of these mutants were compared to that of the parent strain under ten different conditions, i.e. growth on four different carbon sources, growth at three different salt concentrations, and application of four different stress conditions. In addition, cell morphologies in exponential and stationary phase were observed. Furthermore, swarming of 17 mutants was analyzed. 24 of the 27 mutants exhibited a difference from the parent strain under at least one condition, revealing that haloarchaeal sRNAs are involved in metabolic regulation, growth under extreme conditions, regulation of morphology and behavior, and stress adaptation. Notably, 7 deletion mutants showed a gain of function phenotype, which has not yet been described for any other prokaryotic sRNA gene deletion mutant. Comparison of the transcriptomes of one sRNA gene deletion mutant and the parent strain led to the identification of differentially expressed genes. Genes for flagellins and chemotaxis were up-regulated in the mutant, in accordance with its gain of function swarming phenotype. While the deletion mutant analysis underscored that haloarchaeal sRNAs are involved in many biological functions, the degree of conservation is extremely low. Only 3 of the 27 genes are conserved in more than 10 haloarchaeal species. 22 of the 27 genes are confined to H. volcanii, indicating a fast evolution of haloarchaeal sRNA genes. <br/>
</details><br/>
---

*Conserved active site cysteine residue of archaeal THI4 homolog is essential for thiamine biosynthesis in Haloferax volcanii.*<br/>
Hwang S; Cordova B; Chavarria N; Elbanna D; McHugh S; Rojas J; Pfeiffer F; Maupin-Furlow JA<br/>
**2014**<br/>
BMC Microbiol. 14(None)<br/>
10.1186/s12866-014-0260-0
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25348237)<br/>
[Citation]()<br/>
BACKGROUND: Thiamine (vitamin B1) is synthesized de novo by certain yeast, fungi, plants, protozoans, bacteria and archaea. The pathway of thiamine biosynthesis by archaea is poorly understood, particularly the route of sulfur relay to form the thiazole ring. Archaea harbor structural homologs of both the bacterial (ThiS-ThiF) and eukaryotic (THI4) proteins that mobilize sulfur to thiazole ring precursors by distinct mechanisms.
RESULTS: Based on comparative genome analysis, halophilic archaea are predicted to synthesize the pyrimidine moiety of thiamine by the bacterial pathway, initially suggesting that also a bacterial ThiS-ThiF type mechanism for synthesis of the thiazole ring is used in which the sulfur carrier ThiS is first activated by ThiF-catalyzed adenylation. The only ThiF homolog of Haloferax volcanii (UbaA) was deleted but this had no effect on growth in the absence of thiamine. Usage of the eukaryotic THI4-type sulfur relay was initially considered less likely for thiamine biosynthesis in archaea, since the active-site cysteine residue of yeast THI4p that donates the sulfur to the thiazole ring by a suicide mechanism is replaced by a histidine residue in many archaeal THI4 homologs and these are described as D-ribose-1,5-bisphosphate isomerases. The THI4 homolog of the halophilic archaea, including Hfx. volcanii (HVO_0665, HvThi4) was found to differ from that of methanogens and thermococci by having a cysteine residue (Cys165) corresponding to the conserved active site cysteine of yeast THI4p (Cys205). Deletion of HVO_0665 generated a thiamine auxotroph that was trans-complemented by a wild-type copy of HVO_0665, but not the modified gene encoding an HvThi4 C165A variant.
CONCLUSIONS: Based on our results, we conclude that the archaeon Hfx. volcanii uses a yeast THI4-type mechanism for sulfur relay to form the thiazole ring of thiamine. We extend this finding to a relatively large group of archaea, including haloarchaea, ammonium oxidizing archaea, and some methanogen and Pyrococcus species, by observing that these organisms code for THI4 homologs that have a conserved active site cysteine residue which is likely used in thiamine biosynthesis. Thus, archaeal members of IPR002922 THI4 family that have a conserved cysteine active site should be reexamined for a function in thiamine biosynthesis.<br/>
</details><br/>
---

*A conserved type IV pilin signal peptide H-domain is critical for the post-translational regulation of flagella-dependent motility.*<br/>
Esquivel RN; Pohlschroder M<br/>
**2014**<br/>
Mol. Microbiol. 93(3)<br/>
10.1111/mmi.12673
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24945931)<br/>
[Citation]()<br/>
In many bacteria and archaea, type IV pili facilitate surface adhesion, the initial step in biofilm formation. Haloferax volcanii has a specific set of adhesion pilins (PilA1-A6) that, although diverse, contain an absolutely conserved signal peptide hydrophobic (H) domain. Data presented here demonstrate that these pilins (PilA1-A6) also play an important role in regulating flagella-dependent motility, which allows cells to rapidly transition between planktonic and sessile states. Cells lacking adhesion pilins exhibit a severe motility defect, however, expression of any one of the adhesion pilins in trans can rescue the motility and adhesion. Conversely, while deleting pilB3-C3, genes required for PilA pilus biosynthesis, results in cells lacking pili and having an adhesion defect, it does not affect motility, indicating that motility regulation requires the presence of pilins, but not assembled pili. Mutagenesis studies revealed that the pilin-dependent motility regulatory mechanism does not require the diverse C-terminal region of the PilA pilins but specifically involves the conserved H-domain. This novel post-translational regulatory mechanism, which employs components that promote biofilm formation to inhibit motility, can provide a rapid response to changing environmental conditions. A model for this regulatory mechanism, which may also be present in other prokaryotes, is discussed. <br/>
</details><br/>
---

*Biofilms formed by the archaeon Haloferax volcanii exhibit cellular differentiation and social motility, and facilitate horizontal gene transfer.*<br/>
Chimileski S; Franklin MJ; Papke RT<br/>
**2014**<br/>
BMC Biol. 12(None)<br/>
10.1186/s12915-014-0065-5
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/25124934)<br/>
[Citation]()<br/>
BACKGROUND: Archaea share a similar microbial lifestyle with bacteria, and not surprisingly then, also exist within matrix-enclosed communities known as biofilms. Advances in biofilm biology have been made over decades for model bacterial species, and include characterizations of social behaviors and cellular differentiation during biofilm development. Like bacteria, archaea impact ecological and biogeochemical systems. However, the biology of archaeal biofilms is only now being explored. Here, we investigated the development, composition and dynamics of biofilms formed by the haloarchaeon Haloferax volcanii DS2.
RESULTS: Biofilms were cultured in static liquid and visualized with fluorescent cell membrane dyes and by engineering cells to express green fluorescent protein (GFP). Analysis by confocal scanning laser microscopy showed that H. volcanii cells formed microcolonies within 24 h, which developed into larger clusters by 48 h and matured into flake-like towers often greater than 100 μm in height after 7 days. To visualize the extracellular matrix, biofilms formed by GFP-expressing cells were stained with concanavalin A, DAPI, Congo red and thioflavin T. Stains colocalized with larger cellular structures and indicated that the extracellular matrix may contain a combination of polysaccharides, extracellular DNA and amyloid protein. Following a switch to biofilm growth conditions, a sub-population of cells differentiated into chains of long rods sometimes exceeding 25 μm in length, compared to their planktonic disk-shaped morphology. Time-lapse photography of static liquid biofilms also revealed wave-like social motility. Finally, we quantified gene exchange between biofilm cells, and found that it was equivalent to the mating frequency of a classic filter-based experimental method.
CONCLUSIONS: The developmental processes, functional properties and dynamics of H. volcanii biofilms provide insight on how haloarchaeal species might persist, interact and exchange DNA in natural communities. H. volcanii demonstrates some biofilm phenotypes similar to bacterial biofilms, but also has interesting phenotypes that may be unique to this organism or to this class of organisms, including changes in cellular morphology and an unusual form of social motility. Because H. volcanii has one of the most advanced genetic systems for any archaeon, the phenotypes reported here may promote the study of genetic and developmental processes in archaeal biofilms.<br/>
</details><br/>
---

*Archaeal Tuc1/Ncs6 homolog required for wobble uridine tRNA thiolation is associated with ubiquitin-proteasome, translation, and RNA processing system homologs.*<br/>
Chavarria NE; Hwang S; Cao S; Fu X; Holman M; Elbanna D; Rodriguez S; Arrington D; Englert M; Uthandi S; Söll D; Maupin-Furlow JA<br/>
**2014**<br/>
PLoS ONE 9(6)<br/>
10.1371/journal.pone.0099104
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24906001)<br/>
[Citation]()<br/>
While cytoplasmic tRNA 2-thiolation protein 1 (Tuc1/Ncs6) and ubiquitin-related modifier-1 (Urm1) are important in the 2-thiolation of 5-methoxycarbonylmethyl-2-thiouridine (mcm5s2U) at wobble uridines of tRNAs in eukaryotes, the biocatalytic roles and properties of Ncs6/Tuc1 and its homologs are poorly understood. Here we present the first report of an Ncs6 homolog of archaea (NcsA of Haloferax volcanii) that is essential for maintaining cellular pools of thiolated tRNA(Lys)UUU and for growth at high temperature. When purified from Hfx. volcanii, NcsA was found to be modified at Lys204 by isopeptide linkage to polymeric chains of the ubiquitin-fold protein SAMP2. The ubiquitin-activating E1 enzyme homolog of archaea (UbaA) was required for this covalent modification. Non-covalent protein partners that specifically associated with NcsA were also identified including UbaA, SAMP2, proteasome activating nucleotidase (PAN)-A/1, translation elongation factor aEF-1α and a β-CASP ribonuclease homolog of the archaeal cleavage and polyadenylation specificity factor 1 family (aCPSF1). Together, our study reveals that NcsA is essential for growth at high temperature, required for formation of thiolated tRNA(Lys)UUU and intimately linked to homologs of ubiquitin-proteasome, translation and RNA processing systems.<br/>
</details><br/>
---

*Extracellular DNA metabolism in Haloferax volcanii.*<br/>
Chimileski S; Dolas K; Naor A; Gophna U; Papke RT<br/>
**2014**<br/>
Front Microbiol 5(None)<br/>
10.3389/fmicb.2014.00057
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24600440)<br/>
[Citation]()<br/>
Extracellular DNA is found in all environments and is a dynamic component of the microbial ecosystem. Microbial cells produce and interact with extracellular DNA through many endogenous mechanisms. Extracellular DNA is processed and internalized for use as genetic information and as a major source of macronutrients, and plays several key roles within prokaryotic biofilms. Hypersaline sites contain some of the highest extracellular DNA concentrations measured in nature-a potential rich source of carbon, nitrogen, and phosphorus for halophilic microorganisms. We conducted DNA growth studies for the halophilic archaeon Haloferax volcanii DS2 and show that this model Halobacteriales strain is capable of using exogenous double-stranded DNA as a nutrient. Further experiments with varying medium composition, DNA concentration, and DNA types revealed that DNA is utilized primarily as a phosphorus source, that growth on DNA is concentration-dependent, and that DNA isolated from different sources is metabolized selectively, with a bias against highly divergent methylated DNA. Additionally, fluorescence microscopy showed that labeled DNA co-localized with H. volcanii cells. The gene Hvo_1477 was also identified using a comparative genomic approach as a factor likely to be involved in DNA processing at the cell surface, and deletion of Hvo_1477 created a strain deficient in the ability to grow on extracellular DNA. Widespread distribution of Hvo_1477 homologs in archaea suggests metabolism of extracellular DNA may be of broad ecological and physiological relevance in this domain of life. <br/>
</details><br/>
---

*The LonB protease controls membrane lipids composition and is essential for viability in the extremophilic haloarchaeon Haloferax volcanii.*<br/>
Cerletti M; Martínez MJ; Giménez MI; Sastre DE; Paggi RA; De Castro RE<br/>
**2014**<br/>
Environ. Microbiol. 16(6)<br/>
10.1111/1462-2920.12385
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24428705)<br/>
[Citation]()<br/>
Although homologs of the ATP-dependent Lon protease exist in all domains of life, the relevance of this protease in archaeal physiology remains a mystery. In this study, we have constructed and phenotypically characterized deletion and conditional lon mutants in the model haloarchaeon Haloferax volcanii to elucidate the role of the unusual membrane-bound LonB protease in archaea. Hvlon could be deleted from the chromosome only when a copy of the wild type gene was provided in trans suggesting that Lon is essential for survival in this archaeon. Successful complementation of the lethal phenotype of ΔHvlon was attained by expression of the heterologous protease gene Nmlon from the haloalkaliphilic archaeon Natrialba magadii, meaning that the biological function of Lon is conserved in these organisms. Suboptimal cellular levels of Lon protein affected growth rate, cell shape, cell pigmentation, lipid composition and sensitivity to various antibiotics. The contents of bacterioruberins and some polar lipids were increased in the lon mutants suggesting that Lon is linked to maintenance of membrane lipid balance which likely affects cell viability in this archaeon. The phenotypes associated to a membrane-bound LonB protease mutant were examined for the first time providing insight on the relevance of this protease in archaeal physiology. <br/>
</details><br/>
---

*Substrate promiscuity: AglB, the archaeal oligosaccharyltransferase, can process a variety of lipid-linked glycans.*<br/>
Cohen-Rosenzweig C; Guan Z; Shaanan B; Eichler J<br/>
**2014**<br/>
Appl. Environ. Microbiol. 80(2)<br/>
10.1128/AEM.03191-13
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24212570)<br/>
[Citation]()<br/>
Across evolution, N-glycosylation involves oligosaccharyltransferases that transfer lipid-linked glycans to selected Asn residues of target proteins. While these enzymes catalyze similar reactions in each domain, differences exist in terms of the chemical composition, length and degree of phosphorylation of the lipid glycan carrier, the sugar linking the glycan to the lipid carrier, and the composition and structure of the transferred glycan. To gain insight into how oligosaccharyltransferases cope with such substrate diversity, the present study analyzed the archaeal oligosaccharyltransferase AglB from four haloarchaeal species. Accordingly, it was shown that despite processing distinct lipid-linked glycans in their native hosts, AglB from Haloarcula marismortui, Halobacterium salinarum, and Haloferax mediterranei could readily replace their counterpart from Haloferax volcanii when introduced into Hfx. volcanii cells deleted of aglB. As the four enzymes show significant sequence and apparently structural homology, it appears that the functional similarity of the four AglB proteins reflects the relaxed substrate specificity of these enzymes. Such demonstration of AglB substrate promiscuity is important not only for better understanding of N-glycosylation in Archaea and elsewhere but also for efforts aimed at transforming Hfx. volcanii into a glycoengineering platform.<br/>
</details><br/>
---

*Small regulatory RNAs in Archaea.*<br/>
Babski J; Maier LK; Heyer R; Jaschinski K; Prasse D; Jäger D; Randau L; Schmitz RA; Marchfelder A; Soppa J<br/>
**2014**<br/>
RNA Biol 11(5)<br/>
10.4161/rna.28452
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24755959)<br/>
[Citation]()<br/>
Small regulatory RNAs (sRNAs) are universally distributed in all three domains of life, Archaea, Bacteria, and Eukaryotes. In bacteria, sRNAs typically function by binding near the translation start site of their target mRNAs and thereby inhibit or activate translation. In eukaryotes, miRNAs and siRNAs typically bind to the 3'-untranslated region (3'-UTR) of their target mRNAs and influence translation efficiency and/or mRNA stability. In archaea, sRNAs have been identified in all species investigated using bioinformatic approaches, RNomics, and RNA-Seq. Their size can vary significantly between less than 50 to more than 500 nucleotides. Differential expression of sRNA genes has been studied using northern blot analysis, microarrays, and RNA-Seq. In addition, biological functions have been unraveled by genetic approaches, i.e., by characterization of designed mutants. As in bacteria, it was revealed that archaeal sRNAs are involved in many biological processes, including metabolic regulation, adaptation to extreme conditions, stress responses, and even in regulation of morphology and cellular behavior. Recently, the first target mRNAs were identified in archaea, including one sRNA that binds to the 5'-region of two mRNAs in Methanosarcina mazei Gö1 and a few sRNAs that bind to 3'-UTRs in Sulfolobus solfataricus, three Pyrobaculum species, and Haloferax volcanii, indicating that archaeal sRNAs appear to be able to target both the 5'-UTR or the 3'-UTRs of their respective target mRNAs. In addition, archaea contain tRNA-derived fragments (tRFs), and one tRF has been identified as a major ribosome-binding sRNA in H. volcanii, which downregulates translation in response to stress. Besides regulatory sRNAs, archaea contain further classes of sRNAs, e.g., CRISPR RNAs (crRNAs) and snoRNAs. <br/>
</details><br/>
---

*A complex of Cas proteins 5, 6, and 7 is required for the biogenesis and stability of clustered regularly interspaced short palindromic repeats (crispr)-derived rnas (crrnas) in Haloferax volcanii.*<br/>
Brendel J; Stoll B; Lange SJ; Sharma K; Lenz C; Stachler AE; Maier LK; Richter H; Nickel L; Schmitz RA; Randau L; Allers T; Urlaub H; Backofen R; Marchfelder A<br/>
**2014**<br/>
J. Biol. Chem. 289(10)<br/>
10.1074/jbc.M113.508184
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24459147)<br/>
[Citation]()<br/>
The clustered regularly interspaced short palindromic repeats/CRISPR-associated (CRISPR-Cas) system is a prokaryotic defense mechanism against foreign genetic elements. A plethora of CRISPR-Cas versions exist, with more than 40 different Cas protein families and several different molecular approaches to fight the invading DNA. One of the key players in the system is the CRISPR-derived RNA (crRNA), which directs the invader-degrading Cas protein complex to the invader. The CRISPR-Cas types I and III use the Cas6 protein to generate mature crRNAs. Here, we show that the Cas6 protein is necessary for crRNA production but that additional Cas proteins that form a CRISPR-associated complex for antiviral defense (Cascade)-like complex are needed for crRNA stability in the CRISPR-Cas type I-B system in Haloferax volcanii in vivo. Deletion of the cas6 gene results in the loss of mature crRNAs and interference. However, cells that have the complete cas gene cluster (cas1-8b) removed and are transformed with the cas6 gene are not able to produce and stably maintain mature crRNAs. crRNA production and stability is rescued only if cas5, -6, and -7 are present. Mutational analysis of the cas6 gene reveals three amino acids (His-41, Gly-256, and Gly-258) that are essential for pre-crRNA cleavage, whereas the mutation of two amino acids (Ser-115 and Ser-224) leads to an increase of crRNA amounts. This is the first systematic in vivo analysis of Cas6 protein variants. In addition, we show that the H. volcanii I-B system contains a Cascade-like complex with a Cas7, Cas5, and Cas6 core that protects the crRNA. <br/>
</details><br/>
---

*(R)-mevalonate 3-phosphate is an intermediate of the mevalonate pathway in Thermoplasma acidophilum.*<br/>
Azami Y; Hattori A; Nishimura H; Kawaide H; Yoshimura T; Hemmi H<br/>
**2014**<br/>
J. Biol. Chem. 289(23)<br/>
10.1074/jbc.M114.562686
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24755225)<br/>
[Citation]()<br/>
The lack of a few conserved enzymes in the classical mevalonate pathway and the widespread existence of isopentenyl phosphate kinase suggest the presence of a partly modified mevalonate pathway in most archaea and in some bacteria. In the pathway, (R)-mevalonate 5-phosphate is thought to be metabolized to isopentenyl diphosphate via isopentenyl phosphate. The long anticipated enzyme that catalyzes the reaction from (R)-mevalonate 5-phosphate to isopentenyl phosphate was recently identified in a Cloroflexi bacterium, Roseiflexus castenholzii, and in a halophilic archaeon, Haloferax volcanii. However, our trial to convert the intermediates of the classical and modified mevalonate pathways into isopentenyl diphosphate using cell-free extract from a thermophilic archaeon Thermoplasma acidophilum implied that the branch point intermediate of these known pathways, i.e. (R)-mevalonate 5-phosphate, is unlikely to be the precursor of isoprenoid. Through the process of characterizing the recombinant homologs of mevalonate pathway-related enzymes from the archaeon, a distant homolog of diphosphomevalonate decarboxylase was found to catalyze the phosphorylation of (R)-mevalonate to yield (R)-mevalonate 3-phosphate. The product could be converted into isopentenyl phosphate, probably through (R)-mevalonate 3,5-bisphosphate, by the action of unidentified T. acidophilum enzymes fractionated by anion-exchange chromatography. These findings demonstrate the presence of a third alternative "Thermoplasma-type" mevalonate pathway, which involves (R)-mevalonate 3-phosphotransferase and probably both (R)-mevalonate 3-phosphate 5-phosphotransferase and (R)-mevalonate 3,5-bisphosphate decarboxylase, in addition to isopentenyl phosphate kinase. <br/>
</details><br/>
---

*Covalent immobilization of alcohol dehydrogenase (ADH2) from Haloferax volcanii: how to maximize activity and optimize performance of halophilic enzymes.*<br/>
Alsafadi D; Paradisi F<br/>
**2014**<br/>
Mol. Biotechnol. 56(3)<br/>
10.1007/s12033-013-9701-5
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24062264)<br/>
[Citation]()<br/>
Alcohol dehydrogenase from halophilic archaeon Haloferax volcanii (HvADH2) was successfully covalently immobilized on metal-derivatized epoxy Sepabeads. The immobilization conditions were optimized by investigating several parameters that affect the halophilic enzyme-support interaction. The highest immobilization efficiency (100 %) and retention activity (60 %) were achieved after 48 h of incubation of the enzyme with Ni-epoxy Sepabeads support in 100 mM Tris-HCl buffer, pH 8, containing 3 M KCl at 5 °C. No significant stabilization was observed after blocking the unreacted epoxy groups with commonly used hydrophilic agents. A significant increase in the stability of the immobilized enzyme was achieved by blocking the unreacted epoxy groups with ethylamine. The immobilization process increased the enzyme stability, thermal activity, and organic solvents tolerance when compared to its soluble counterpart, indicating that the immobilization enhances the structural and conformational stability. One step purification-immobilization of this enzyme has been carried out on metal chelate-epoxy Sepabeads, as an efficient method to obtain immobilized biocatalyst directly from bacterial extracts. <br/>
</details><br/>
---

*Ionic strength-dependent conformations of a ubiquitin-like small archaeal modifier protein (SAMP1) from Haloferax volcanii.*<br/>
Ye K; Liao S; Zhang W; Fan K; Zhang X; Zhang J; Xu C; Tu X<br/>
**2013**<br/>
Protein Sci. 22(9)<br/>
10.1002/pro.2302
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23818097)<br/>
[Citation]()<br/>
Eukaryotic ubiquitin and ubiquitin-like systems play crucial roles in various cellular biological processes. In this work, we determined the solution structure of SAMP1 from Haloferax volcanii by NMR spectroscopy. Under low ionic conditions, SAMP1 presented two distinct conformations, one folded β-grasp and the other disordered. Interestingly, SAMP1 underwent a conformational conversion from disorder to order with ion concentration increasing, indicating that the ordered conformation is the functional form of SAMP1 under the physiological condition of H. volcanii. Furthermore, SAMP1 could interact with proteasome-activating nucleotidase B, supposing a potential role of SAMP1 in the protein degradation pathway mediated by proteasome. <br/>
</details><br/>
---

*Conserved substitution patterns around nucleosome footprints in eukaryotes and Archaea derive from frequent nucleosome repositioning through evolution.*<br/>
Warnecke T; Becker EA; Facciotti MT; Nislow C; Lehner B<br/>
**2013**<br/>
PLoS Comput. Biol. 9(11)<br/>
10.1371/journal.pcbi.1003373
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24278010)<br/>
[Citation]()<br/>
Nucleosomes, the basic repeat units of eukaryotic chromatin, have been suggested to influence the evolution of eukaryotic genomes, both by altering the propensity of DNA to mutate and by selection acting to maintain or exclude nucleosomes in particular locations. Contrary to the popular idea that nucleosomes are unique to eukaryotes, histone proteins have also been discovered in some archaeal genomes. Archaeal nucleosomes, however, are quite unlike their eukaryotic counterparts in many respects, including their assembly into tetramers (rather than octamers) from histone proteins that lack N- and C-terminal tails. Here, we show that despite these fundamental differences the association between nucleosome footprints and sequence evolution is strikingly conserved between humans and the model archaeon Haloferax volcanii. In light of this finding we examine whether selection or mutation can explain concordant substitution patterns in the two kingdoms. Unexpectedly, we find that neither the mutation nor the selection model are sufficient to explain the observed association between nucleosomes and sequence divergence. Instead, we demonstrate that nucleosome-associated substitution patterns are more consistent with a third model where sequence divergence results in frequent repositioning of nucleosomes during evolution. Indeed, we show that nucleosome repositioning is both necessary and largely sufficient to explain the association between current nucleosome positions and biased substitution patterns. This finding highlights the importance of considering the direction of causality between genetic and epigenetic change. <br/>
</details><br/>
---

*Expression in Haloferax volcanii of 3-hydroxy-3-methylglutaryl coenzyme A synthase facilitates isolation and characterization of the active form of a key enzyme required for polyisoprenoid cell membrane biosynthesis in halophilic archaea.*<br/>
VanNice JC; Skaff DA; Wyckoff GJ; Miziorko HM<br/>
**2013**<br/>
J. Bacteriol. 195(17)<br/>
10.1128/JB.00485-13
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23794621)<br/>
[Citation]()<br/>
Enzymes of the isoprenoid biosynthetic pathway in halophilic archaea remain poorly characterized, and parts of the pathway remain cryptic. This situation may be explained, in part, by the difficulty of expressing active, functional recombinant forms of these enzymes. The use of newly available expression plasmids and hosts has allowed the expression and isolation of catalytically active Haloferax volcanii 3-hydroxy-3-methylglutaryl coenzyme A (CoA) synthase (EC 2.3.310). This accomplishment has permitted studies that represent, to the best of our knowledge, the first characterization of an archaeal hydroxymethylglutaryl CoA synthase. Kinetic characterization indicates that, under optimal assay conditions, which include 4 M KCl, the enzyme exhibits catalytic efficiency and substrate saturation at metabolite levels comparable to those reported for the enzyme from nonhalophilic organisms. This enzyme is unique in that it is the first hydroxymethylglutaryl CoA synthase that is insensitive to feedback substrate inhibition by acetoacetyl-CoA. The enzyme supports reaction catalysis in the presence of various organic solvents. Haloferax 3-hydroxy-3-methylglutaryl CoA synthase is sensitive to inactivation by hymeglusin, a specific inhibitor known to affect prokaryotic and eukaryotic forms of the enzyme, with experimentally determined Ki and kinact values of 570 ± 120 nM and 17 ± 3 min(-1), respectively. In in vivo experiments, hymeglusin blocks the propagation of H. volcanii cells, indicating the critical role that the mevalonate pathway plays in isoprenoid biosynthesis by these archaea. <br/>
</details><br/>
---

*Haloferax volcanii cells lacking the flagellin FlgA2 are hypermotile.*<br/>
Tripepi M; Esquivel RN; Wirth R; Pohlschröder M<br/>
**2013**<br/>
Microbiology (Reading, Engl.) 159(Pt 11)<br/>
10.1099/mic.0.069617-0
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23989184)<br/>
[Citation]()<br/>
Motility driven by rotational movement of flagella allows bacteria and archaea to seek favourable conditions and escape toxic ones. However, archaeal flagella share structural similarities with bacterial type IV pili rather than bacterial flagella. The Haloferax volcanii genome contains two flagellin genes, flgA1 and flgA2. While FlgA1 has been shown to be a major flagellin, the function of FlgA2 is elusive. In this study, it was determined that although FlgA2 by itself does not confer motility to non-motile ΔflgA1 Hfx. volcanii, a subset of these mutant cells contains a flagellum. Consistent with FlgA2 being assembled into functional flagella, FlgA1 expressed from a plasmid can only complement a ΔflgA1 strain when co-expressed with chromosomal or plasmid-encoded FlgA2. Surprisingly, a mutant strain lacking FlgA2, but expressing chromosomally encoded FlgA1, is hypermotile, a phenotype that is accompanied by an increased number of flagella per cell, as well as an increased flagellum length. Site-directed mutagenesis resulting in early translational termination of flgA2 suggests that the hypermotility of the ΔflgA2 strain is not due to transcriptional regulation. This, and the fact that plasmid-encoded FlgA2 expression in a ΔflgA2 strain does not reduce its hypermotility, suggests a possible regulatory role for FlgA2 that depends on the relative abundance of FlgA1. Taken together, our results indicate that FlgA2 plays both structural and regulatory roles in Hfx. volcanii flagella-dependent motility. Future studies will build upon the data presented here to elucidate the significance of the hypermotility of this ΔflgA2 mutant, and will illuminate the regulation and function of archaeal flagella. <br/>
</details><br/>
---

*Effect of an overproduction of accessory Gvp proteins on gas vesicle formation in Haloferax volcanii.*<br/>
Tavlaridou S; Faist K; Weitzel K; Pfeifer F<br/>
**2013**<br/>
Extremophiles 17(2)<br/>
10.1007/s00792-013-0515-y
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23338749)<br/>
[Citation]()<br/>
Gas vesicle formation in haloarchaea requires the expression of the p-vac region consisting of 14 genes, gvpACNO and gvpDEFGHIJKLM. Expression of gvpFGHIJKLM leads to essential accessory proteins formed in minor amounts. An overexpression of gvpG, gvpH or gvpM in addition to p-vac inhibited gas vesicle formation, whereas large amounts of all other Gvp proteins did not disturb the synthesis. The unbalanced expression and in particular an aggregation of the overproduced Gvp with other accessory Gvp derived from p-vac could be a reason for the inhibition. Western analyses demonstrated that the hydrophobic GvpM (and GvpJ) indeed form multimers. Fluorescent dots of GvpM-GFP were seen in cells in vivo underlining an aggregation of GvpM. In search for proteins neutralizing the inhibitory effect in case of GvpM, p-vac +pGM(ex), +pHM(ex), +pJM(ex), and +pLM(ex) transformants were constructed. The inhibitory effect of GvpM on gas vesicle formation was suppressed by GvpH, GvpJ or GvpL, but not by GvpG. Western analyses demonstrated that pHM(ex) and pJM(ex) transformants contained additional larger protein bands when probed with an antiserum raised against GvpH or GvpJ, implying interactions. The balanced amount of GvpM-GvpH and GvpM-GvpJ appears to be important during gas vesicle genesis.<br/>
</details><br/>
---

*A comparison of two novel alcohol dehydrogenase enzymes (ADH1 and ADH2) from the extreme halophile Haloferax volcanii.*<br/>
Timpson LM; Liliensiek AK; Alsafadi D; Cassidy J; Sharkey MA; Liddell S; Allers T; Paradisi F<br/>
**2013**<br/>
Appl. Microbiol. Biotechnol. 97(1)<br/>
10.1007/s00253-012-4074-4
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22526808)<br/>
[Citation]()<br/>
Haloarchaeal alcohol dehydrogenases are exciting biocatalysts with potential industrial applications. In this study, two alcohol dehydrogenase enzymes from the extremely halophilic archaeon Haloferax volcanii (HvADH1 and HvADH2) were homologously expressed and subsequently purified by immobilized metal-affinity chromatography. The proteins appeared to copurify with endogenous alcohol dehydrogenases, and a double Δadh2 Δadh1 gene deletion strain was constructed to prevent this occurrence. Purified HvADH1 and HvADH2 were compared in terms of stability and enzymatic activity over a range of pH values, salt concentrations, and temperatures. Both enzymes were haloalkaliphilic and thermoactive for the oxidative reaction and catalyzed the reductive reaction at a slightly acidic pH. While the NAD(+)-dependent HvADH1 showed a preference for short-chain alcohols and was inherently unstable, HvADH2 exhibited dual cofactor specificity, accepted a broad range of substrates, and, with respect to HvADH1, was remarkably stable. Furthermore, HvADH2 exhibited tolerance to organic solvents. HvADH2 therefore displays much greater potential as an industrially useful biocatalyst than HvADH1.<br/>
</details><br/>
---

*Requirements for a successful defence reaction by the CRISPR-Cas subtype I-B system.*<br/>
Stoll B; Maier LK; Lange SJ; Brendel J; Fischer S; Backofen R; Marchfelder A<br/>
**2013**<br/>
Biochem. Soc. Trans. 41(6)<br/>
10.1042/BST20130098
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24256235)<br/>
[Citation]()<br/>
Uptake of foreign mobile genetic elements is often detrimental and can result in cell death. For protection against invasion, prokaryotes have developed several defence mechanisms, which take effect at all stages of infection; an example is the recently discovered CRISPR (clustered regularly interspaced short palindromic repeats)-Cas (CRISPR-associated) immune system. This defence system directly degrades invading genetic material and is present in almost all archaea and many bacteria. Current data indicate a large variety of mechanistic molecular approaches. Although almost all archaea carry this defence weapon, only a few archaeal systems have been fully characterized. In the present paper, we summarize the prerequisites for the detection and degradation of invaders in the halophilic archaeon Haloferax volcanii. H. volcanii encodes a subtype I-B CRISPR-Cas system and the defence can be triggered by a plasmid-based invader. Six different target-interference motifs are recognized by the Haloferax defence and a 9-nt non-contiguous seed sequence is essential. The repeat sequence has the potential to fold into a minimal stem-loop structure, which is conserved in haloarchaea and might be recognized by the Cas6 endoribonuclease during the processing of CRISPR loci into mature crRNA (CRISPR RNA). Individual crRNA species were present in very different concentrations according to an RNA-Seq analysis and many were unable to trigger a successful defence reaction. Recognition of the plasmid invader does not depend on its copy number, but instead results indicate a dependency on the type of origin present on the plasmid.<br/>
</details><br/>
---

*Use of GFP-GvpE fusions to quantify the GvpD-mediated reduction of the transcriptional activator GvpE in haloarchaea.*<br/>
Schmidt I; Pfeifer F<br/>
**2013**<br/>
Arch. Microbiol. 195(6)<br/>
10.1007/s00203-013-0885-7
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23589224)<br/>
[Citation]()<br/>
Gas vesicle formation of Halobacterium salinarum is regulated by the transcriptional activator GvpE, and in the presence of the repressing protein GvpD, the amount of GvpE is strongly reduced. The green fluorescence protein was used to report this GvpD-mediated reduction of GvpE in vivo in Haloferax volcanii transformants. Both N- or C-terminal fusions of GFP to GvpE were tested, but only the N-terminal fusion reported the reduction. The fluorescence of GFP-GvpE was 62 % reduced with GvpD wild type (DWT), 78 % with the super-repressor D3-AAA, and only 10 % with the repression defect DMut6. Further analysis of D3-AAA indicated that the super-repression was due to the alteration R496A. GFP-GvpE variants defect in promoter activation was tested in the presence of DWT, D3-AAA and DMut6, and two of them were more stable. Overall, the GFP-GvpE fusion was suitable to study and quantify the amount of GvpE in vivo.<br/>
</details><br/>
---

*Specificity of archaeal caspase activity in the extreme halophile Haloferax volcanii.*<br/>
Seth-Pasricha M; Bidle KA; Bidle KD<br/>
**2013**<br/>
Environ Microbiol Rep 5(2)<br/>
10.1111/1758-2229.12010
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23565123)<br/>
[Citation]()<br/>
Caspase-like proteases are key initiators and executioners of programmed cell death (PCD), which is initiated by environmental stimuli and manifests in organisms ranging from unicellular microbes to higher eukaryotes. Archaea had been absent from the caspase inheritance discussion due to a lack of gene homologues. We recently demonstrated extremely high, basal caspase-like catalytic activity in the model haloarcheon, Haloferax volcanii, which was linked to the cellular stress response and was widespread among diverse Archaea. Here, we rigorously tested the catalytic specificity of the observed archaeal caspase-like activities using hydrolytic assays with a diverse suite of protease substrates and inhibitors compared with known model serine and cysteine proteases (trypsin, cathepsin, papain, and human caspase-8). Our experiments demonstrate that exponentially growing H. volcanii possesses a highly specific caspase-like activity that most closely resembles caspase-4, is preferentially inhibited by the pancaspase inhibitor, zVAD-FMK, and has no crossreactivity with other known protease families. Our findings firmly root the extremely high levels of caspase-like activity as the dominant proteolytic activity in this extreme haloarcheaon, thereby providing further support for housekeeping functions in Haloarchaea. Given the deep archaeal roots of eukaryotes, we suggest that this activity served as a foundation for stress pathways in higher organisms.<br/>
</details><br/>
---

*Haloferax chudinovii sp. nov., a halophilic archaeon from Permian potassium salt deposits.*<br/>
Saralov AI; Baslerov RV; Kuznetsov BB<br/>
**2013**<br/>
Extremophiles 17(3)<br/>
10.1007/s00792-013-0534-8
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23525908)<br/>
[Citation]()<br/>
Three pigmented strains of halophilic archaea (RS75, RS77, RS79) were isolated from the monoliths of mottled sylvinite from the Verkhnekamsk salt deposit (Solikamsk, Russia). The cells were nonmotile, gram-negative, pleomorphic, disk-shaped or ovoid, 0.8-1.0 × 1.5-2.5 μm. The organism was a chemoorganotrophic obligate aerobe producing catalase and oxidase. A number of carbohydrates and carboxylic acids were used as growth substrates. Growth occurred in the presence of 7-27% NaCl (with the optimum at 15-18%), 0.02-20% KCl (0.2-1%), 0.2-16% MgCl2 (2-3%), in the temperature range from 23 to 51 °C (40-45 °C), and pH 5.5-8.0 (6.8-7.0). The membranes contained carotenoids of the bacterioruberin series. Phosphatidylglyceromethylphosphate (PGP-Me), phosphatidylglycerol (PG), sulfated diglycosyl diether (S-DGD-1) predominated among the polar lipids. The DNA G + C content was 64.0-65.0 mol %. Phylogenetic analysis of the 16S rRNA gene sequences showed high similarity of the new strains to Haloferax species: H. denitrificans (99.2%) and H. volcanii (99.1%), H. larsenii (96.9%) and H. elongans (96.6%). DNA-DNA hybridization revealed 93-95% similarity between strain RS75 and strains RS77 and RS79; the similarity levels between strain RS75 and the type strains of Haloferax denitrificans VKM B-1754(T) and Halobacterium salinarum VKM B-1769(T) were 50 and 10%, respectively. According to its phenotypic and genotypic characteristics, the organism was classified as a member of the genus Haloferax, forming a new species with the proposed name Haloferax chudinovii sp. nov. type strain is RS75(T) (=VKPM B-11279(T)).<br/>
</details><br/>
---

*Mapping of active replication origins in vivo in thaum- and euryarchaeal replicons.*<br/>
Pelve EA; Martens-Habbena W; Stahl DA; Bernander R<br/>
**2013**<br/>
Mol. Microbiol. 90(3)<br/>
10.1111/mmi.12382
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23991938)<br/>
[Citation]()<br/>
We report mapping of active replication origins in thaum- and euryarchaeal replicons using high-throughput sequencing-based marker frequency analysis. The chromosome of the thaumarchaeon Nitrosopumilus maritimus is shown to contain a single origin of replication, whereas the main chromosome in the halophilic euryarchaea Haloferax mediterranei and Haloferax volcanii each contains two origins. All replication origins specified bidirectional replication, and the two origins in the halophiles were initiated in synchrony. The pHM500 plasmid of H. mediterranei is shown to contain a single origin, and the copy numbers of five plasmid replicons in the two halophiles were inferred to be close to that of the main chromosome. Origin recognition boxes (ORBs) that provide binding sites for Orc1/Cdc6 replication initiator proteins are identified at all chromosomal origins, as well as in a range of additional thaumarchaeal species. An annotation update is provided for all three species. <br/>
</details><br/>
---

*Dihydroxyacetone metabolism in Haloferax volcanii.*<br/>
Ouellette M; Makkay AM; Papke RT<br/>
**2013**<br/>
Front Microbiol 4(None)<br/>
10.3389/fmicb.2013.00376
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24379808)<br/>
[Citation]()<br/>
Dihydroxyacetone (DHA) is a ketose sugar that can be produced by oxidizing glycerol. DHA in the environment is taken up and phosphorylated to DHA-phosphate by glycerol kinase or DHA kinase. In hypersaline environments, it is hypothesized that DHA is produced as an overflow product from glycerol utilization by organisms such as Salinibacter ruber. Previous research has demonstrated that the halobacterial species Haloquadratum walsbyi can use DHA as a carbon source, and putative DHA kinase genes were hypothesized to be involved in this process. However, DHA metabolism has not been demonstrated in other halobacterial species, and the role of the DHA kinase genes was not confirmed. In this study, we examined the metabolism of DHA in Haloferax volcanii because putative DHA kinase genes were annotated in its genome, and it has an established genetic system to assay growth of mutant knockouts. Experiments in which Hfx. volcanii was grown on DHA as the sole carbon source demonstrated growth, and that it is concentration dependent. Three annotated DHA kinase genes (HVO_1544, HVO_1545, and HVO_1546), which are homologous to the putative DHA kinase genes present in Hqm. walsbyi, as well as the glycerol kinase gene (HVO_1541), were deleted to examine the effect of these genes on the growth of Hfx. volcanii on DHA. Experiments demonstrated that the DHA kinase deletion mutant exhibited diminished, but not absence of growth on DHA compared to the parent strain. Deletion of the glycerol kinase gene also reduced growth on DHA, and did so more than deletion of the DHA kinase. The results indicate that Hfx. volcanii can metabolize DHA and that DHA kinase plays a role in this metabolism. However, the glycerol kinase appears to be the primary enzyme involved in this process. BLASTp analyses demonstrate that the DHA kinase genes are patchily distributed among the Halobacteria, whereas the glycerol kinase gene is widely distributed, suggesting a widespread capability for DHA metabolism. <br/>
</details><br/>
---

*A halocin-H4 mutant Haloferax mediterranei strain retains the ability to inhibit growth of other halophilic archaea.*<br/>
Naor A; Yair Y; Gophna U<br/>
**2013**<br/>
Extremophiles 17(6)<br/>
10.1007/s00792-013-0579-8
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24037372)<br/>
[Citation]()<br/>
Many members of the Halobacteriaceae were found to produce halocins, molecules that inhibit the growth of other halophilic archaea. Halocin H4 that is produced by Haloferax mediterranei and inhibits the growth of Halobacterium salinarum is one of the best studied halocins to date. The gene encoding this halocin had been previously identified as halH4, located on one of Hfx. mediterranei megaplasmids. We generated a mutant of the halH4 gene and examined the killing ability of the Haloferax mediterranei halH4 mutant with respect to both Halobacterium salinarum and Haloferax volcanii. We showed that both wild-type Hfx. mediterranei and the halH4 mutant strain efficiently inhibited the growth of both species, indicating halocin redundancy. Surprisingly, the halH4 deletion mutant exhibited faster growth in standard medium than the wild type, and is likely to have a better response to several nucleotides, which could explain this phenotype. <br/>
</details><br/>
---

*Cell fusion and hybrids in Archaea: prospects for genome shuffling and accelerated strain development for biotechnology.*<br/>
Naor A; Gophna U<br/>
**2013**<br/>
Bioengineered 4(3)<br/>
10.4161/bioe.22649
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23111319)<br/>
[Citation]()<br/>
The ability to exchange DNA between cells is a molecular process that exists in different species in the domain Archaea. Such horizontal gene transfer events were shown to take place between distant species of archaea and to result in the transfer of large genomic regions. Here we describe recent progress in this field, discuss the potential use of natural gene exchange processes to perform genome shuffling and argue its possible biotechnological applications.<br/>
</details><br/>
---

*Essential requirements for the detection and degradation of invaders by the Haloferax volcanii CRISPR/Cas system I-B.*<br/>
Maier LK; Lange SJ; Stoll B; Haas KA; Fischer S; Fischer E; Duchardt-Ferner E; Wöhnert J; Backofen R; Marchfelder A<br/>
**2013**<br/>
RNA Biol 10(5)<br/>
10.4161/rna.24282
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23594992)<br/>
[Citation]()<br/>
To fend off foreign genetic elements, prokaryotes have developed several defense systems. The most recently discovered defense system, CRISPR/Cas, is sequence-specific, adaptive and heritable. The two central components of this system are the Cas proteins and the CRISPR RNA. The latter consists of repeat sequences that are interspersed with spacer sequences. The CRISPR locus is transcribed into a precursor RNA that is subsequently processed into short crRNAs. CRISPR/Cas systems have been identified in bacteria and archaea, and data show that many variations of this system exist. We analyzed the requirements for a successful defense reaction in the halophilic archaeon Haloferax volcanii. Haloferax encodes a CRISPR/Cas system of the I-B subtype, about which very little is known. Analysis of the mature crRNAs revealed that they contain a spacer as their central element, which is preceded by an eight-nucleotide-long 5' handle that originates from the upstream repeat. The repeat sequences have the potential to fold into a minimal stem loop. Sequencing of the crRNA population indicated that not all of the spacers that are encoded by the three CRISPR loci are present in the same abundance. By challenging Haloferax with an invader plasmid, we demonstrated that the interaction of the crRNA with the invader DNA requires a 10-nucleotide-long seed sequence. In addition, we found that not all of the crRNAs from the three CRISPR loci are effective at triggering the degradation of invader plasmids. The interference does not seem to be influenced by the copy number of the invader plasmid. <br/>
</details><br/>
---

*Hot and sweet: protein glycosylation in Crenarchaeota.*<br/>
Meyer BH; Albers SV<br/>
**2013**<br/>
Biochem. Soc. Trans. 41(1)<br/>
10.1042/BST20120296
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23356316)<br/>
[Citation]()<br/>
Every living cell is covered with a dense and complex array of covalently attached sugars or sugar chains. The majority of these glycans are linked to proteins via the so-called glycosylation process. Protein glycosylation is found in all three domains of life: Eukarya, Bacteria and Archaea. However, on the basis of the limit in analytic tools for glycobiology and genetics in Archaea, only in the last few years has research on archaeal glycosylation pathways started mainly in the Euryarchaeota Haloferax volcanii, Methanocaldococcus maripaludis and Methanococcus voltae. Recently, major steps of the crenarchaeal glycosylation process of the thermoacidophilic archaeon Sulfolobus acidocaldarius have been described. The present review summarizes the proposed N-glycosylation pathway of S. acidocaldarius, describing the phenotypes of the mutants disrupted in N-glycan biosynthesis as well as giving insights into the archaeal O-linked and glycosylphosphatidylinositol anchor glycosylation process.<br/>
</details><br/>
---

*The ring of confidence: a haloarchaeal CRISPR/Cas system.*<br/>
Maier LK; Stoll B; Brendel J; Fischer S; Pfeiffer F; Dyall-Smith M; Marchfelder A<br/>
**2013**<br/>
Biochem. Soc. Trans. 41(1)<br/>
10.1042/BST20120263
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23356314)<br/>
[Citation]()<br/>
To survive the constant invasions by foreign genetic elements, prokaryotes have evolved various defensive systems. Almost all sequenced archaea, and half of the analysed bacteria use the CRISPR (clustered regularly interspaced short palindromic repeats)/Cas (CRISPR-associated) system, a recently identified prokaryotic immune system that can fend off invading elements in a sequence-specific manner. Few archaeal CRISPR/Cas systems have been analysed so far, and the molecular details of many of the steps involved in adaptation and defence are yet to be understood. In the present paper, we summarize our current knowledge about the CRISPR/Cas system in Haloferax volcanii, an extremely halophilic archaeon that was isolated from the Dead Sea. H. volcanii encodes a type I-B CRISPR/Cas system, and carries three CRISPR loci and eight Cas proteins. Although in laboratory culture for more than three decades, this defence system was shown to be still active. All three CRISPR loci are transcribed and processed into mature crRNAs (CRISPR RNAs). Cells challenged with engineered plasmids can recognize and eliminate these invading elements if they contain the correct PAM (protospacer adjacent motif) and a sequence that can be recognized by one of the CRISPR spacers.<br/>
</details><br/>
---

*Prospects for robust biocatalysis: engineering of novel specificity in a halophilic amino acid dehydrogenase.*<br/>
Munawar N; Engel PC<br/>
**2013**<br/>
Extremophiles 17(1)<br/>
10.1007/s00792-012-0491-7
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23104166)<br/>
[Citation]()<br/>
Heat- and solvent-tolerant enzymes from halophiles, potentially important industrially, offer a robust framework for protein engineering, but few solved halophilic structures exist to guide this. Homology modelling has guided mutations in glutamate dehydrogenase (GDH) from Halobacterium salinarum to emulate conversion of a mesophilic GDH to a methionine dehydrogenase. Replacement of K89, A163 and S367 by leucine, glycine and alanine converted halophilic GDH into a dehydrogenase accepting L-methionine, L-norleucine and L-norvaline as substrates. Over-expression in the halophilic expression host Haloferax volcanii and three-step purification gave 98 % pure protein exhibiting maximum activity at pH 10. This enzyme also showed enhanced thermostability and organic solvent tolerance even at 70 °C, offering a biocatalyst resistant to harsh industrial environments. To our knowledge, this is the first reported amino acid specificity change engineered in a halophilic enzyme, encouraging use of mesophilic models to guide engineering of novel halophilic biocatalysts for industrial application. Calibrated gel filtration experiments show that both the mutant and the wild-type enzyme are stable hexamers.<br/>
</details><br/>
---

*Construction of a shuttle expression vector with a promoter functioning in both halophilic Archaea and Bacteria.*<br/>
Lv J; Wang S; Zeng C; Huang Y; Chen X<br/>
**2013**<br/>
FEMS Microbiol. Lett. 349(1)<br/>
10.1111/1574-6968.12278
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24106795)<br/>
[Citation]()<br/>
A shuttle expression vector, designated as pAJ, was constructed based on the Haloferax volcanii-Escherichia coli shuttle vector pSY1. This new construct contains the amyH promoter from Haloarcula hispanica and was able to confer the promoter activity in both Hfx. volcanii and E. coli. pAJ successfully expressed proteins in Hfx. volcanii or E. coli, rendering it feasible to express target proteins in corresponding domains. In addition, pAJ contains a multiple cloning site with 11 restriction sites and a 6×His tag sequence, and the vector size was decreased to 8903 bp. To the best of our knowledge, pAJ is the first reported shuttle expression vector that can express proteins in both Bacteria and Archaea. Importantly, pAJ can even express the haloarchaeal heat shock protein DnaK in both domains. In conclusion, this novel vector only provides researchers with a new means to manipulate genes or express proteins in Haloarchaea but also serves as a convenient tool for the comparative study of the function of some highly conserved genes in Haloarchaea and in Bacteria. <br/>
</details><br/>
---

*Intracellular dynamics of archaeal FANCM homologue Hef in response to halted DNA replication.*<br/>
Lestini R; Laptenok SP; Kühn J; Hink MA; Schanne-Klein MC; Liebl U; Myllykallio H<br/>
**2013**<br/>
Nucleic Acids Res. 41(22)<br/>
nar/gkt816
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24049073)<br/>
[Citation]()<br/>
Hef is an archaeal member of the DNA repair endonuclease XPF (XPF)/Crossover junction endonuclease MUS81 (MUS81)/Fanconi anemia, complementation group M (FANCM) protein family that in eukaryotes participates in the restart of stalled DNA replication forks. To investigate the physiological roles of Hef in maintaining genome stability in living archaeal cells, we studied the localization of Hef-green fluorescent protein fusions by fluorescence microscopy. Our studies revealed that Haloferax volcanii Hef proteins formed specific localization foci under regular growth conditions, the number of which specifically increased in response to replication arrest. Purification of the full-length Hef protein from its native host revealed that it forms a stable homodimer in solution, with a peculiar elongated configuration. Altogether our data indicate that the shape of Hef, significant physicochemical constraints and/or interactions with DNA limit the apparent cytosolic diffusion of halophilic DNA replication/repair complexes, and demonstrate that Hef proteins are dynamically recruited to archaeal eukaryotic-like chromatin to counteract DNA replication stress. We suggest that the evolutionary conserved function of Hef/FANCM proteins is to enhance replication fork stability by directly interacting with collapsed replication forks. <br/>
</details><br/>
---

*Ionic strength-dependent conformations of a ubiquitin-like small archaeal modifier protein (SAMP2) from Haloferax volcanii.*<br/>
Liao S; Zhang W; Fan K; Ye K; Zhang X; Zhang J; Xu C; Tu X<br/>
**2013**<br/>
Sci Rep 3(None)<br/>
10.1038/srep02136
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23823798)<br/>
[Citation]()<br/>
Ubiquitin-like proteins play important roles in diverse biological processes. In this study, we present an unexpected finding that a ubiquitin-like small archaeal modifier protein (SAMP2) from Haloferax volcanii adopts two distinct states under low ionic condition. One of these is similar to the β-grasp structure conserved in ubiquitin-like proteins from eukaryotes; the other is disordered, like prokaryotic ubiquitin-like protein, Pup. Furthermore, our study reveals that the conformation of SAMP2 is dependent on ionic strength. With the increase of ion concentration, SAMP2 undergoes a conformational conversion from disorder to order, indicating that the ordered conformation is the functional form of SAMP2 under the physiological condition of H. volcanii.<br/>
</details><br/>
---

*Crystal structure of the ubiquitin-like small archaeal modifier protein 2 from Haloferax volcanii.*<br/>
Li Y; Maciejewski MW; Martin J; Jin K; Zhang Y; Maupin-Furlow JA; Hao B<br/>
**2013**<br/>
Protein Sci. 22(9)<br/>
10.1002/pro.2305
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23821306)<br/>
[Citation]()<br/>
The discovery of ubiquitin-like small archaeal modifier protein 2 (SAMP2) that forms covalent polymeric chains in Haloferax volcanii has generated tremendous interest in the function and regulation of this protein. At present, it remains unclear whether the Hfx. volcanii modifier protein SAMP1 has such polyubiquitinating-like activity. Although SAMP1 and SAMP2 use the same conjugation machinery to modify their target proteins, each can impart distinct functional consequences. To better understand the mechanism of SAMP2 conjugation, we have sought to characterize the biophysical and structural properties of the protein from Hfx. volcanii. SAMP2 is only partially structured under mesohalic solution conditions and adopts a well-folded compact conformation in the presence of 2.5M of NaCl. Its 2.3-Å-resolution crystal structure reveals a characteristic α/β central core domain and a unique β-hinge motif. This motif anchors an unusual C-terminal extension comprising the diglycine tail as well as two lysine residues that can potentially serve to interlink SAMP2 moieties. Mutational alternation of the structural malleability of this β-hinge motif essentially abolishes the conjugation activity of SAMP2 in vivo. In addition, NMR structural studies of the putative ubiquitin-like protein HVO_2177 from Hfx. volcanii show that like SAMP1, HVO_2177 forms a classic β-grasp fold in a salt-independent manner. These results provide insights into the structure-function relationship of sampylating proteins of fundamental importance in post-translational protein modification and environmental cues in Archaea. <br/>
</details><br/>
---

*Heterologous overexpression, purification and characterisation of an alcohol dehydrogenase (ADH2) from Halobacterium sp. NRC-1.*<br/>
Liliensiek AK; Cassidy J; Gucciardo G; Whitely C; Paradisi F<br/>
**2013**<br/>
Mol. Biotechnol. 55(2)<br/>
10.1007/s12033-013-9666-4
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23645073)<br/>
[Citation]()<br/>
Replacement of chemical steps with biocatalytic ones is becoming increasingly more interesting due to the remarkable catalytic properties of enzymes, such as their wide range of substrate specificities and variety of chemo-, stereo- and regioselective reactions. This study presents characterisation of an alcohol dehydrogenase (ADH) from the halophilic archaeum Halobacterium sp. NRC-1 (HsADH2). A hexahistidine-tagged recombinant version of HsADH2 (His-HsADH2) was heterologously overexpressed in Haloferax volcanii. The enzyme was purified in one step by immobilised Ni-affinity chromatography. His-HsADH2 was halophilic and mildly thermophilic with optimal activity for ethanol oxidation at 4 M KCl around 60 °C and pH 10.0. The enzyme was extremely stable, retaining 80 % activity after 30 days. His-HsADH2 showed preference for NADP(H) but interestingly retained 60 % activity towards NADH. The enzyme displayed broad substrate specificity, with maximum activity obtained for 1-propanol. The enzyme also accepted secondary alcohols such as 2-butanol and even 1-phenylethanol. In the reductive reaction, working conditions for His-HsADH2 were optimised for acetaldehyde and found to be 4 M KCl and pH 6.0. His-HsADH2 displayed intrinsic organic solvent tolerance, which is highly relevant for biotechnological applications.<br/>
</details><br/>
---

*CISA: contig integrator for sequence assembly of bacterial genomes.*<br/>
Lin SH; Liao YC<br/>
**2013**<br/>
PLoS ONE 8(3)<br/>
10.1371/journal.pone.0060843
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23556006)<br/>
[Citation]()<br/>
A plethora of algorithmic assemblers have been proposed for the de novo assembly of genomes, however, no individual assembler guarantees the optimal assembly for diverse species. Optimizing various parameters in an assembler is often performed in order to generate the most optimal assembly. However, few efforts have been pursued to take advantage of multiple assemblies to yield an assembly of high accuracy. In this study, we employ various state-of-the-art assemblers to generate different sets of contigs for bacterial genomes. A tool, named CISA, has been developed to integrate the assemblies into a hybrid set of contigs, resulting in assemblies of superior contiguity and accuracy, compared with the assemblies generated by the state-of-the-art assemblers and the hybrid assemblies merged by existing tools. This tool is implemented in Python and requires MUMmer and BLAST+ to be installed on the local machine. The source code of CISA and examples of its use are available at http://sb.nhri.org.tw/CISA/.<br/>
</details><br/>
---

*Two distinct N-glycosylation pathways process the Haloferax volcanii S-layer glycoprotein upon changes in environmental salinity.*<br/>
Kaminski L; Guan Z; Yurist-Doutsch S; Eichler J<br/>
**2013**<br/>
MBio 4(6)<br/>
10.1128/mBio.00716-13
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24194539)<br/>
[Citation]()<br/>
UNLABELLED: N-glycosylation in Archaea presents aspects of this posttranslational modification not seen in either Eukarya or Bacteria. In the haloarchaeon Haloferax volcanii, the surface (S)-layer glycoprotein can be simultaneously modified by two different N-glycans. Asn-13 and Asn-83 are modified by a pentasaccharide, whereas Asn-498 is modified by a tetrasaccharide of distinct composition, with N-glycosylation at this position being related to environmental conditions. Specifically, N-glycosylation of Asn-498 is detected when cells are grown in the presence of 1.75 but not 3.4 M NaCl. While deletion of genes encoding components of the pentasaccharide assembly pathway had no effect on the biosynthesis of the tetrasaccharide bound to Asn-498, deletion of genes within the cluster spanning HVO_2046 to HVO_2061 interfered with the assembly and attachment of the Asn-498-linked tetrasaccharide. Transfer of the "low-salt" tetrasaccharide from the dolichol phosphate carrier upon which it is assembled to S-layer glycoprotein Asn-498 did not require AglB, the oligosaccharyltransferase responsible for pentasaccharide attachment to Asn-13 and Asn-83. Finally, although biogenesis of the low-salt tetrasaccharide is barely discernible upon growth at the elevated salinity, this glycan was readily detected under such conditions in strains deleted of pentasaccharide biosynthesis pathway genes, indicative of cross talk between the two N-glycosylation pathways.
IMPORTANCE: In the haloarchaeon Haloferax volcanii, originally from the Dead Sea, the pathway responsible for the assembly and attachment of a pentasaccharide to the S-layer glycoprotein, a well-studied glycoprotein in this species, has been described. More recently, it was shown that in response to growth in low salinity, the same glycoprotein is modified by a novel tetrasaccharide. In the present study, numerous components of the pathway used to synthesize this "low-salt" tetrasaccharide are described. As such, this represents the first report of two N-glycosylation pathways able to simultaneously modify a single protein as a function of environmental salinity. Moreover, and to the best of our knowledge, the ability to N-glycosylate the same protein with different and unrelated glycans has not been observed in either Eukarya or Bacteria or indeed beyond the halophilic archaea, for which similar dual modification of the Halobacterium salinarum S-layer glycoprotein was reported.<br/>
</details><br/>
---

*Add salt, add sugar: N-glycosylation in Haloferax volcanii.*<br/>
Kaminski L; Naparstek S; Kandiba L; Cohen-Rosenzweig C; Arbiv A; Konrad Z; Eichler J<br/>
**2013**<br/>
Biochem. Soc. Trans. 41(1)<br/>
10.1042/BST20120142
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23356324)<br/>
[Citation]()<br/>
Although performed by members of all three domains of life, the archaeal version of N-glycosylation remains the least understood. Studies on Haloferax volcanii have, however, begun to correct this situation. A combination of bioinformatics, molecular biology, biochemical and mass spectrometry approaches have served to delineate the Agl pathway responsible for N-glycosylation of the S-layer glycoprotein, a reporter of this post-translational modification in Hfx. volcanii. More recently, differential N-glycosylation of the S-layer glycoprotein as a function of environmental salinity was demonstrated, showing that this post-translational modification serves an adaptive role in Hfx. volcanii. Furthermore, manipulation of the Agl pathway, together with the capability of Hfx. volcanii to N-glycosylate non-native proteins, forms the basis for establishing this species as a glyco-engineering platform. In the present review, these and other recent findings are addressed.<br/>
</details><br/>
---

*Lipid modification gives rise to two distinct Haloferax volcanii S-layer glycoprotein populations.*<br/>
Kandiba L; Guan Z; Eichler J<br/>
**2013**<br/>
Biochim. Biophys. Acta 1828(3)<br/>
10.1016/j.bbamem.2012.11.023
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23201543)<br/>
[Citation]()<br/>
The S-layer glycoprotein is the sole component of the protein shell surrounding Haloferax volcanii cells. The deduced amino acid sequence of the S-layer glycoprotein predicts the presence of a C-terminal membrane-spanning domain. However, several earlier observations, including the ability of EDTA to selectively solubilize the protein, are inconsistent with the presence of a trans-membrane sequence. In the present report, sequential solubilization of the S-layer glycoprotein by EDTA and then with detergent revealed the existence of two distinct populations of the S-layer glycoprotein. Whereas both S-layer glycoprotein populations underwent signal peptide cleavage and N-glycosylation, base hydrolysis followed by mass spectrometry revealed that a lipid, likely archaetidic acid, modified only the EDTA-solubilized version of the protein. These observations are consistent with the S-layer glycoprotein being initially synthesized as an integral membrane protein and subsequently undergoing a processing event in which the extracellular portion of the protein is separated from the membrane-spanning domain and transferred to a waiting lipid moiety.<br/>
</details><br/>
---

*L-Arabinose degradation pathway in the haloarchaeon Haloferax volcanii involves a novel type of L-arabinose dehydrogenase.*<br/>
Johnsen U; Sutter JM; Zaiß H; Schönheit P<br/>
**2013**<br/>
Extremophiles 17(6)<br/>
10.1007/s00792-013-0572-2
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23949136)<br/>
[Citation]()<br/>
The pathway of L-arabinose degradation was studied in the haloarchaeon Haloferax volcanii. It is shown that L-arabinose is oxidatively degraded to α-ketoglutarate. During growth on L-arabinose, L-arabinose dehydrogenase (L-AraDH) was induced. The enzyme was purified as a 130 kDa homotetrameric protein catalyzing the oxidation of L-arabinose with both NADP(+) and NAD(+). The gene encoding L-AraDH was identified as HVO_B0032 and recombinant L-AraDH showed similar properties as the native enzyme. The L-AraDH deletion mutant did not grow on L-arabinose, but grew unaffected on glucose and D-xylose, indicating a specific involvement in L-arabinose degradation. Phylogenetic analyses attribute the first archaeal L-AraDH to the extended short-chain dehydrogenase/reductase (SDRe) family, where it is part of a novel cluster and thus differs from known archaeal and bacterial pentose dehydrogenases. Further, cell extracts of H. volcanii catalyzed the NADP(+)-dependent conversion of L-arabinoate to α-ketoglutarate. The genes involved in that conversion were identified by analyses of transcripts and deletion mutants as HVO_B0038A, HVO_B0027 and HVO_B0039 recently reported to be involved in D-xylonate conversion to α-ketoglutarate in H. volcanii (Johnsen et al. 2009). <br/>
</details><br/>
---

*Accelerated growth in the absence of DNA replication origins.*<br/>
Hawkins M; Malla S; Blythe MJ; Nieduszynski CA; Allers T<br/>
**2013**<br/>
Nature 503(7477)<br/>
10.1038/nature12650
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24185008)<br/>
[Citation]()<br/>
DNA replication initiates at defined sites called origins, which serve as binding sites for initiator proteins that recruit the replicative machinery. Origins differ in number and structure across the three domains of life and their properties determine the dynamics of chromosome replication. Bacteria and some archaea replicate from single origins, whereas most archaea and all eukaryotes replicate using multiple origins. Initiation mechanisms that rely on homologous recombination operate in some viruses. Here we show that such mechanisms also operate in archaea. We use deep sequencing to study replication in Haloferax volcanii and identify four chromosomal origins of differing activity. Deletion of individual origins results in perturbed replication dynamics and reduced growth. However, a strain lacking all origins has no apparent defects and grows significantly faster than wild type. Origin-less cells initiate replication at dispersed sites rather than at discrete origins and have an absolute requirement for the recombinase RadA, unlike strains lacking individual origins. Our results demonstrate that homologous recombination alone can efficiently initiate the replication of an entire cellular genome. This raises the question of what purpose replication origins serve and why they have evolved. <br/>
</details><br/>
---

*A comprehensive analysis of the importance of translation initiation factors for Haloferax volcanii applying deletion and conditional depletion mutants.*<br/>
Gäbel K; Schmitt J; Schulz S; Näther DJ; Soppa J<br/>
**2013**<br/>
PLoS ONE 8(11)<br/>
10.1371/journal.pone.0077188
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24244275)<br/>
[Citation]()<br/>
Translation is an important step in gene expression. The initiation of translation is phylogenetically diverse, since currently five different initiation mechanisms are known. For bacteria the three initiation factors IF1 - IF3 are described in contrast to archaea and eukaryotes, which contain a considerably higher number of initiation factor genes. As eukaryotes and archaea use a non-overlapping set of initiation mechanisms, orthologous proteins of both domains do not necessarily fulfill the same function. The genome of Haloferax volcanii contains 14 annotated genes that encode (subunits of) initiation factors. To gain a comprehensive overview of the importance of these genes, it was attempted to construct single gene deletion mutants of all genes. In 9 cases single deletion mutants were successfully constructed, showing that the respective genes are not essential. In contrast, the genes encoding initiation factors aIF1, aIF2γ, aIF5A, aIF5B, and aIF6 were found to be essential. Factors aIF1A and aIF2β are encoded by two orthologous genes in H. volcanii. Attempts to generate double mutants failed in both cases, indicating that also these factors are essential. A translatome analysis of one of the single aIF2β deletion mutants revealed that the translational efficiency of the second ortholog was enhanced tenfold and thus the two proteins can replace one another. The phenotypes of the single deletion mutants also revealed that the two aIF1As and aIF2βs have redundant but not identical functions. Remarkably, the gene encoding aIF2α, a subunit of aIF2 involved in initiator tRNA binding, could be deleted. However, the mutant had a severe growth defect under all tested conditions. Conditional depletion mutants were generated for the five essential genes. The phenotypes of deletion mutants and conditional depletion mutants were compared to that of the wild-type under various conditions, and growth characteristics are discussed. <br/>
</details><br/>
---

*N-glycosylation in Haloferax volcanii: adjusting the sweetness.*<br/>
Eichler J; Arbiv A; Cohen-Rosenzweig C; Kaminski L; Kandiba L; Konrad Z<br/>
**2013**<br/>
Front Microbiol 4(None)<br/>
10.3389/fmicb.2013.00403
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24399998)<br/>
[Citation]()<br/>
Long believed to be restricted to Eukarya, it is now known that cells of all three domains of life perform N-glycosylation, the covalent attachment of glycans to select target protein asparagine residues. Still, it is only in the last decade that pathways of N-glycosylation in Archaea have been delineated. In the haloarchaeon Haloferax volcanii, a series of Agl (archaeal glycosylation) proteins is responsible for the addition of an N-linked pentasaccharide to modified proteins, including the surface (S)-layer glycoprotein, the sole component of the surface layer surrounding the cell. The S-layer glycoprotein N-linked glycosylation profile changes, however, as a function of surrounding salinity. Upon growth at different salt concentrations, the S-layer glycoprotein is either decorated by the N-linked pentasaccharide introduced above or by both this pentasaccharide as well as a tetrasaccharide of distinct composition. Recent efforts have identified Agl5-Agl15 as components of a second Hfx. volcanii N-glycosylation pathway responsible for generating the tetrasaccharide attached to S-layer glycoprotein when growth occurs in 1.75 M but not 3.4 M NaCl-containing medium. <br/>
</details><br/>
---

*Novel archaeal adhesion pilins with a conserved N terminus.*<br/>
Esquivel RN; Xu R; Pohlschroder M<br/>
**2013**<br/>
J. Bacteriol. 195(17)<br/>
10.1128/JB.00572-13
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23794623)<br/>
[Citation]()<br/>
Type IV pili play important roles in a wide array of processes, including surface adhesion and twitching motility. Although archaeal genomes encode a diverse set of type IV pilus subunits, the functions for most remain unknown. We have now characterized six Haloferax volcanii pilins, PilA[1-6], each containing an identical 30-amino-acid N-terminal hydrophobic motif that is part of a larger highly conserved domain of unknown function (Duf1628). Deletion mutants lacking up to five of the six pilin genes display no significant adhesion defects; however, H. volcanii lacking all six pilins (ΔpilA[1-6]) does not adhere to glass or plastic. Consistent with these results, the expression of any one of these pilins in trans is sufficient to produce functional pili in the ΔpilA[1-6] strain. PilA1His and PilA2His only partially rescue this phenotype, whereas ΔpilA[1-6] strains expressing PilA3His or PilA4His adhere even more strongly than the parental strain. Most surprisingly, expressing either PilA5His or PilA6His in the ΔpilA[1-6] strain results in microcolony formation. A hybrid protein in which the conserved N terminus of the mature PilA1His is replaced with the corresponding N domain of FlgA1 is processed by the prepilin peptidase, but it does not assemble functional pili, leading us to conclude that Duf1628 can be annotated as the N terminus of archaeal PilA adhesion pilins. Finally, the pilin prediction program, FlaFind, which was trained primarily on archaeal flagellin sequences, was successfully refined to more accurately predict pilins based on the in vivo verification of PilA[1-6]. <br/>
</details><br/>
---

*Cu-NirK from Haloferax mediterranei as an example of metalloprotein maturation and exportation via Tat system.*<br/>
Esclapez J; Zafrilla B; Martínez-Espinosa RM; Bonete MJ<br/>
**2013**<br/>
Biochim. Biophys. Acta 1834(6)<br/>
10.1016/j.bbapap.2013.03.002
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23499847)<br/>
[Citation]()<br/>
The green Cu-NirK from Haloferax mediterranei (Cu-NirK) has been expressed, refolded and retrieved as a trimeric enzyme using an expression method developed for halophilic Archaea. This method utilizes Haloferax volcanii as a halophilic host and an expression vector with a constitutive and strong promoter. The enzymatic activity of recombinant Cu-NirK was detected in both cellular fractions (cytoplasmic fraction and membranes) and in the culture media. The characterization of the enzyme isolated from the cytoplasmic fraction as well as the culture media revealed important differences in the primary structure of both forms indicating that Hfx. mediterranei could carry out a maturation and exportation process within the cell before the protein is exported to the S-layer. Several conserved signals found in Cu-NirK from Hfx. mediterranei sequence indicate that these processes are closely related to the Tat system. Furthermore, the N-terminal sequence of the two Cu-NirK subunits constituting different isoforms revealed that translation of this protein could begin at two different points, identifying two possible start codons. The hypothesis proposed in this work for halophilic Cu-NirK processing and exportation via the Tat system represents the first approximation of this mechanism in the Halobacteriaceae family and in Prokarya in general.<br/>
</details><br/>
---

*Post-translation modification in Archaea: lessons from Haloferax volcanii and other haloarchaea.*<br/>
Eichler J; Maupin-Furlow J<br/>
**2013**<br/>
FEMS Microbiol. Rev. 37(4)<br/>
10.1111/1574-6976.12012
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23167813)<br/>
[Citation]()<br/>
As an ever-growing number of genome sequences appear, it is becoming increasingly clear that factors other than genome sequence impart complexity to the proteome. Of the various sources of proteomic variability, post-translational modifications (PTMs) most greatly serve to expand the variety of proteins found in the cell. Likewise, modulating the rates at which different proteins are degraded also results in a constantly changing cellular protein profile. While both strategies for generating proteomic diversity are adopted by organisms across evolution, the responsible pathways and enzymes in Archaea are often less well described than are their eukaryotic and bacterial counterparts. Studies on halophilic archaea, in particular Haloferax volcanii, originally isolated from the Dead Sea, are helping to fill the void. In this review, recent developments concerning PTMs and protein degradation in the haloarchaea are discussed.<br/>
</details><br/>
---

*DNA damage induces nucleoid compaction via the Mre11-Rad50 complex in the archaeon Haloferax volcanii.*<br/>
Delmas S; Duggin IG; Allers T<br/>
**2013**<br/>
Mol. Microbiol. 87(1)<br/>
10.1111/mmi.12091
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23145964)<br/>
[Citation]()<br/>
In prokaryotes the genome is organized in a dynamic structure called the nucleoid, which is embedded in the cytoplasm. We show here that in the archaeon Haloferax volcanii, compaction and reorganization of the nucleoid is induced by stresses that damage the genome or interfere with its replication. The fraction of cells exhibiting nucleoid compaction was proportional to the dose of the DNA damaging agent, and results obtained in cells defective for nucleotide excision repair suggest that breakage of DNA strands triggers reorganization of the nucleoid. We observed that compaction depends on the Mre11-Rad50 complex, suggesting a link to DNA double-strand break repair. However, compaction was observed in a radA mutant, indicating that the role of Mre11-Rad50 in nucleoid reorganisation is independent of homologous recombination. We therefore propose that nucleoid compaction is part of a DNA damage response that accelerates cell recovery by helping DNA repair proteins to locate their targets, and facilitating the search for intact DNA sequences during homologous recombination.<br/>
</details><br/>
---

*AglQ is a novel component of the Haloferax volcanii N-glycosylation pathway.*<br/>
Arbiv A; Yurist-Doutsch S; Guan Z; Eichler J<br/>
**2013**<br/>
PLoS ONE 8(11)<br/>
10.1371/journal.pone.0081782
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/24236216)<br/>
[Citation]()<br/>
N-glycosylation is a post-translational modification performed by members of all three domains of life. Studies on the halophile Haloferax volcanii have offered insight into the archaeal version of this universal protein-processing event. In the present study, AglQ was identified as a novel component of the pathway responsible for the assembly and addition of a pentasaccharide to select Asn residues of Hfx. volcanii glycoproteins, such as the S-layer glycoprotein. In cells deleted of aglQ, both dolichol phosphate, the lipid carrier used in Hfx. volcanii N-glycosylation, and modified S-layer glycoprotein Asn residues only presented the first three pentasaccharide subunits, pointing to a role for AglQ in either preparing the third sugar for attachment of the fourth pentasaccharide subunit or processing the fourth sugar prior to its addition to the lipid-linked trisaccharide. To better define the precise role of AglQ, shown to be a soluble protein, bioinformatics tools were recruited to identify sequence or structural homologs of known function. Site-directed mutagenesis experiments guided by these predictions identified residues important for AglQ function. The results obtained point to AglQ acting as an isomerase in Hfx. volcanii N-glycosylation. <br/>
</details><br/>
---

*Haloferax volcanii archaeosortase is required for motility, mating, and C-terminal processing of the S-layer glycoprotein.*<br/>
Abdul Halim MF; Pfeiffer F; Zou J; Frisch A; Haft D; Wu S; Tolić N; Brewer H; Payne SH; Paša-Tolić L; Pohlschroder M<br/>
**2013**<br/>
Mol. Microbiol. 88(6)<br/>
10.1111/mmi.12248
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23651326)<br/>
[Citation]()<br/>
Cell surfaces are decorated by a variety of proteins that facilitate interactions with their environments and support cell stability. These secreted proteins are anchored to the cell by mechanisms that are diverse, and, in archaea, poorly understood. Recently published in silico data suggest that in some species a subset of secreted euryarchaeal proteins, which includes the S-layer glycoprotein, is processed and covalently linked to the cell membrane by enzymes referred to as archaeosortases. In silico work led to the proposal that an independent, sortase-like system for proteolysis-coupled, carboxy-terminal lipid modification exists in bacteria (exosortase) and archaea (archaeosortase). Here, we provide the first in vivo characterization of an archaeosortase in the haloarchaeal model organism Haloferax volcanii. Deletion of the artA gene (HVO_0915) resulted in multiple biological phenotypes: (a) poor growth, especially under low-salt conditions, (b) alterations in cell shape and the S-layer, (c) impaired motility, suppressors of which still exhibit poor growth, and (d) impaired conjugation. We studied one of the ArtA substrates, the S-layer glycoprotein, using detailed proteomic analysis. While the carboxy-terminal region of S-layer glycoproteins, consisting of a putative threonine-rich O-glycosylated region followed by a hydrophobic transmembrane helix, has been notoriously resistant to any proteomic peptide identification, we were able to identify two overlapping peptides from the transmembrane domain present in the ΔartA strain but not in the wild-type strain. This clearly shows that ArtA is involved in carboxy-terminal post-translational processing of the S-layer glycoprotein. As it is known from previous studies that a lipid is covalently attached to the carboxy-terminal region of the S-layer glycoprotein, our data strongly support the conclusion that archaeosortase functions analogously to sortase, mediating proteolysis-coupled, covalent cell surface attachment.<br/>
</details><br/>
---

*Effect of organic solvents on the activity and stability of halophilic alcohol dehydrogenase (ADH2) from Haloferax volcanii.*<br/>
Alsafadi D; Paradisi F<br/>
**2013**<br/>
Extremophiles 17(1)<br/>
10.1007/s00792-012-0498-0
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23179592)<br/>
[Citation]()<br/>
The effect of various organic solvents on the catalytic activity, stability and substrate specificity of alchohol dehydrogenase from Haloferax volcanii (HvADH2) was evaluated. The HvADH2 showed remarkable stability and catalysed the reaction in aqueous-organic medium containing dimethyl sulfoxide (DMSO) and methanol (MeOH). Tetrahydrofuran and acetonitrile were also investigated and adversely affected the stability of the enzyme. High concentration of salt, essential to maintain the enzymatic activity and structural integrity of the halophilic enzyme under standard conditions may be partially replaced by DMSO and MeOH. The presence of organic solvents did not induce gross changes in substrate specificity. DMSO offered a protective effect for the stability of the enzyme at nonoptimal pHs such as 6 and 10. Salt and solvent effects on the HvADH2 conformation and folding were examined through fluorescence spectroscopy. The fluorescence findings were consistent with the activity and stability results and corroborated the denaturing properties of some solvents. The intrinsic tolerance of this enzyme to organic solvent makes it highly attractive to industry.<br/>
</details><br/>
---

*DNA binding in high salt: analysing the salt dependence of replication protein A3 from the halophile Haloferax volcanii.*<br/>
Winter JA; Patoli B; Bunting KA<br/>
**2012**<br/>
Archaea 2012(None)<br/>
2012/719092
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22973163)<br/>
[Citation]()<br/>
Halophilic archaea maintain intracellular salt concentrations close to saturation to survive in high-salt environments and their cellular processes have adapted to function under these conditions. Little is known regarding halophilic adaptation of the DNA processing machinery, particularly intriguing since protein-DNA interactions are classically salt sensitive. To investigate such adaptation, we characterised the DNA-binding capabilities of recombinant RPA3 from Haloferax volcanii (HvRPA3). Under physiological salt conditions (3 M KCl), HvRPA3 is monomeric, binding 18 nucleotide ssDNA with nanomolar affinity, demonstrating that RPAs containing the single OB-fold/zinc finger architecture bind with broadly comparable affinity to two OB-fold/zinc finger RPAs. Reducing the salt concentration to 1 M KCl induces dimerisation of the protein, which retains its ability to bind DNA. On circular ssDNA, two concentration-dependent binding modes are observed. Conventionally, increased salt concentration adversely affects DNA binding but HvRPA3 does not bind DNA in 0.2 M KCl, although multimerisation may occlude the binding site. The single N-terminal OB-fold is competent to bind DNA in the absence of the C-terminal zinc finger, albeit with reduced affinity. This study represents the first quantitative characterisation of DNA binding in a halophilic protein in extreme salt concentrations.<br/>
</details><br/>
---

*Identification of the enzyme responsible for N1-methylation of pseudouridine 54 in archaeal tRNAs.*<br/>
Wurm JP; Griese M; Bahr U; Held M; Heckel A; Karas M; Soppa J; Wöhnert J<br/>
**2012**<br/>
RNA 18(3)<br/>
10.1261/rna.028498.111
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22274954)<br/>
[Citation]()<br/>
tRNAs from all three kingdoms of life contain a variety of modified nucleotides required for their stability, proper folding, and accurate decoding. One prominent example is the eponymous ribothymidine (rT) modification at position 54 in the T-arm of eukaryotic and bacterial tRNAs. In contrast, in most archaea this position is occupied by another hypermodified nucleotide: the isosteric N1-methylated pseudouridine. While the enzyme catalyzing pseudouridine formation at this position is known, the pseudouridine N1-specific methyltransferase responsible for this modification has not yet been experimentally identified. Here, we present biochemical and genetic evidence that the two homologous proteins, Mja_1640 (COG 1901, Pfam DUF358) and Hvo_1989 (Pfam DUF358) from Methanocaldococcus jannaschii and Haloferax volcanii, respectively, are representatives of the methyltransferase responsible for this modification. However, the in-frame deletion of the pseudouridine N1-methyltransferase gene in H. volcanii did not result in a discernable phenotype in line with similar observations for knockouts of other T-arm methylating enzymes.<br/>
</details><br/>
---

*Enhanced archaeal laccase production in recombinant Escherichia coli by modification of N-terminal propeptide and twin arginine translocation motifs.*<br/>
Uthandi S; Prunetti L; De Vera IM; Fanucci GE; Angerhofer A; Maupin-Furlow JA<br/>
**2012**<br/>
J. Ind. Microbiol. Biotechnol. 39(10)<br/>
10.1007/s10295-012-1152-7
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22752793)<br/>
[Citation]()<br/>
Laccases are multicopper oxidases that couple the oxidation of phenolic polymers to the reduction of molecular oxygen. While an archaeal laccase has only recently been described (LccA from the culture broth of Haloferax volcanii), this enzyme appears promising for biotechnology applications based on its robust bilirubin oxidase and laccase activities as well as its ability to withstand prolonged exposure to extreme conditions. To further optimize LccA productivity and develop an option for LccA purification from whole cells, the encoding gene was modified through deletion of the twin-arginine translocation motif and N-terminal propeptide, and the modified genes were expressed in Escherichia coli. With this approach, LccA was readily purified (overall yield up to 54 %) from the soluble fraction of E. coli as a 74-kDa monomer with syringaldazine oxidizing activity as high as 33 U mg(-1). LccA proteins prepared from H. volcanii culture broth and the soluble fraction of E. coli cells were compared by ICP-AES, EPR, DSC, CD, and UV-Vis spectroscopy and found to have a similar folding pattern with T (m) values and a rich β-sheet structure analogous to other multicopper oxidases. However, in contrast to the H. volcanii-purified LccA, which was loaded with copper, copper was not fully incorporated into the type-I Cu center of E. coli purified LccA, thus, providing insight into avenues for further optimization.<br/>
</details><br/>
---

*N-glycosylation of Haloferax volcanii flagellins requires known Agl proteins and is essential for biosynthesis of stable flagella.*<br/>
Tripepi M; You J; Temel S; Önder Ö; Brisson D; Pohlschröder M<br/>
**2012**<br/>
J. Bacteriol. 194(18)<br/>
10.1128/JB.00731-12
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22730124)<br/>
[Citation]()<br/>
N-glycosylation, a posttranslational modification required for the accurate folding and stability of many proteins, has been observed in organisms of all domains of life. Although the haloarchaeal S-layer glycoprotein was the first prokaryotic glycoprotein identified, little is known about the glycosylation of other haloarchaeal proteins. We demonstrate here that the glycosylation of Haloferax volcanii flagellins requires archaeal glycosylation (Agl) components involved in S-layer glycosylation and that the deletion of any Hfx. volcanii agl gene impairs its swimming motility to various extents. A comparison of proteins in CsCl density gradient centrifugation fractions from supernatants of wild-type Hfx. volcanii and deletion mutants lacking the oligosaccharyltransferase AglB suggests that when the Agl glycosylation pathway is disrupted, cells lack stable flagella, which purification studies indicate consist of a major flagellin, FlgA1, and a minor flagellin, FlgA2. Mass spectrometric analyses of FlgA1 confirm that its three predicted N-glycosylation sites are modified with covalently linked pentasaccharides having the same mass as that modifying its S-layer glycoprotein. Finally, the replacement of any of three predicted N-glycosylated asparagines of FlgA1 renders cells nonmotile, providing direct evidence for the first time that the N-glycosylation of archaeal flagellins is critical for motility. These results provide insight into the role that glycosylation plays in the assembly and function of Hfx. volcanii flagella and demonstrate that Hfx. volcanii flagellins are excellent reporter proteins for the study of haloarchaeal glycosylation processes.<br/>
</details><br/>
---

*Characterization of alcohol dehydrogenase (ADH12) from Haloarcula marismortui, an extreme halophile from the Dead Sea.*<br/>
Timpson LM; Alsafadi D; Mac Donnchadha C; Liddell S; Sharkey MA; Paradisi F<br/>
**2012**<br/>
Extremophiles 16(1)<br/>
10.1007/s00792-011-0405-0
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22015539)<br/>
[Citation]()<br/>
Haloarchaeal alcohol dehydrogenases are of increasing interest as biocatalysts in the field of white biotechnology. In this study, the gene adh12 from the extreme halophile Haloarcula marismortui (HmADH12), encoding a 384 residue protein, was cloned into two vectors: pRV1 and pTA963. The resulting constructs were used to transform host strains Haloferax volcanii (DS70) and (H1209), respectively. Overexpressed His-tagged recombinant HmADH12 was purified by immobilized metal-affinity chromatography (IMAC). The His-tagged protein was visualized by SDS-PAGE, with a subunit molecular mass of 41.6 kDa, and its identity was confirmed by mass spectrometry. Purified HmADH12 catalyzed the interconversion between alcohols and aldehydes and ketones, being optimally active in the presence of 2 M KCl. It was thermoactive, with maximum activity registered at 60°C. The NADP(H) dependent enzyme was haloalkaliphilic for the oxidative reaction with optimum activity at pH 10.0. It favored a slightly acidic pH of 6.0 for catalysis of the reductive reaction. HmADH12 was significantly more tolerant than mesophilic ADHs to selected organic solvents, making it a much more suitable biocatalyst for industrial application.<br/>
</details><br/>
---

*Genetic and Biochemical Identification of a Novel Single-Stranded DNA-Binding Complex in Haloferax volcanii.*<br/>
Stroud A; Liddell S; Allers T<br/>
**2012**<br/>
Front Microbiol 3(None)<br/>
10.3389/fmicb.2012.00224
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22719738)<br/>
[Citation]()<br/>
Single-stranded DNA (ssDNA)-binding proteins play an essential role in DNA replication and repair. They use oligonucleotide/oligosaccharide-binding (OB)-folds, a five-stranded β-sheet coiled into a closed barrel, to bind to ssDNA thereby protecting and stabilizing the DNA. In eukaryotes the ssDNA-binding protein (SSB) is known as replication protein A (RPA) and consists of three distinct subunits that function as a heterotrimer. The bacterial homolog is termed SSB and functions as a homotetramer. In the archaeon Haloferax volcanii there are three genes encoding homologs of RPA. Two of the rpa genes (rpa1 and rpa3) exist in operons with a novel gene specific to Euryarchaeota; this gene encodes a protein that we have termed RPA-associated protein (rpap). The rpap genes encode proteins belonging to COG3390 group and feature OB-folds, suggesting that they might cooperate with RPA in binding to ssDNA. Our genetic analysis showed that rpa1 and rpa3 deletion mutants have differing phenotypes; only Δrpa3 strains are hypersensitive to DNA damaging agents. Deletion of the rpa3-associated gene rpap3 led to similar levels of DNA damage sensitivity, as did deletion of the rpa3 operon, suggesting that RPA3 and RPAP3 function in the same pathway. Protein pull-downs involving recombinant hexahistidine-tagged RPAs showed that RPA3 co-purifies with RPAP3, and RPA1 co-purifies with RPAP1. This indicates that the RPAs interact only with their respective associated proteins; this was corroborated by the inability to construct rpa1 rpap3 and rpa3 rpap1 double mutants. This is the first report investigating the individual function of the archaeal COG3390 RPA-associated proteins (RPAPs). We have shown genetically and biochemically that the RPAPs interact with their respective RPAs, and have uncovered a novel single-stranded DNA-binding complex that is unique to Euryarchaeota.<br/>
</details><br/>
---

*Identification of essential and non-essential single-stranded DNA-binding proteins in a model archaeal organism.*<br/>
Skowyra A; MacNeill SA<br/>
**2012**<br/>
Nucleic Acids Res. 40(3)<br/>
nar/gkr838
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21976728)<br/>
[Citation]()<br/>
Single-stranded DNA-binding proteins (SSBs) play vital roles in all aspects of DNA metabolism in all three domains of life and are characterized by the presence of one or more OB fold ssDNA-binding domains. Here, using the genetically tractable euryarchaeon Haloferax volcanii as a model, we present the first genetic analysis of SSB function in the archaea. We show that genes encoding the OB fold and zinc finger-containing RpaA1 and RpaB1 proteins are individually non-essential for cell viability but share an essential function, whereas the gene encoding the triple OB fold RpaC protein is essential. Loss of RpaC function can however be rescued by elevated expression of RpaB, indicative of functional overlap between the two classes of haloarchaeal SSB. Deletion analysis is used to demonstrate important roles for individual OB folds in RpaC and to show that conserved N- and C-terminal domains are required for efficient repair of DNA damage. Consistent with a role for RpaC in DNA repair, elevated expression of this protein leads to enhanced resistance to DNA damage. Taken together, our results offer important insights into archaeal SSB function and establish the haloarchaea as a valuable model for further studies.<br/>
</details><br/>
---

*Autocatalytic maturation of the Tat-dependent halophilic subtilase Nep produced by the archaeon Natrialba magadii.*<br/>
Ruiz DM; Paggi RA; Giménez MI; De Castro RE<br/>
**2012**<br/>
J. Bacteriol. 194(14)<br/>
10.1128/JB.06792-11
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22582277)<br/>
[Citation]()<br/>
Halolysins are subtilisin-like extracellular proteases produced by haloarchaea that possess unique protein domains and are salt dependent for structural integrity and functionality. In contrast to bacterial subtilases, the maturation mechanism of halolysins has not been addressed. The halolysin Nep is secreted by the alkaliphilic haloarchaeon Natrialba magadii, and the recombinant active enzyme has been synthesized in Haloferax volcanii. Nep contains an N-terminal signal peptide with the typical Tat consensus motif (GRRSVL), an N-terminal propeptide, the protease domain, and a C-terminal domain. In this study, we used Nep as a model protease to examine the secretion and maturation of halolysins by using genetic and biochemical approaches. Mutant variants of Nep were constructed by site-directed mutagenesis and expressed in H. volcanii, which were then analyzed by protease activity and Western blotting. The Tat dependence of Nep secretion was demonstrated in Nep RR/KK variants containing double lysine (KK) in place of the twin arginines (RR), in which Nep remained cell associated and the extracellular activity was undetectable. High-molecular-mass Nep polypeptides without protease activity were detected as cell associated and extracellularly in the Nep S/A variant, in which the catalytic serine 352 had been changed by alanine, indicating that Nep protease activity was needed for precursor processing and activation. Nep NSN 1-2 containing a modification in two potential cleavage sites for signal peptidase I (ASA) was not efficiently processed and activated. This study examined for the first time the secretion and maturation of a Tat-dependent halophilic subtilase.<br/>
</details><br/>
---

*Fructose degradation in the haloarchaeon Haloferax volcanii involves a bacterial type phosphoenolpyruvate-dependent phosphotransferase system, fructose-1-phosphate kinase, and class II fructose-1,6-bisphosphate aldolase.*<br/>
Pickl A; Johnsen U; Schönheit P<br/>
**2012**<br/>
J. Bacteriol. 194(12)<br/>
10.1128/JB.00200-12
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22493022)<br/>
[Citation]()<br/>
The halophilic archaeon Haloferax volcanii utilizes fructose as a sole carbon and energy source. Genes and enzymes involved in fructose uptake and degradation were identified by transcriptional analyses, deletion mutant experiments, and enzyme characterization. During growth on fructose, the gene cluster HVO_1495 to HVO_1499, encoding homologs of the five bacterial phosphotransferase system (PTS) components enzyme IIB (EIIB), enzyme I (EI), histidine protein (HPr), EIIA, and EIIC, was highly upregulated as a cotranscript. The in-frame deletion of HVO_1499, designated ptfC (ptf stands for phosphotransferase system for fructose) and encoding the putative fructose-specific membrane component EIIC, resulted in a loss of growth on fructose, which could be recovered by complementation in trans. Transcripts of HVO_1500 (pfkB) and HVO_1494 (fba), encoding putative fructose-1-phosphate kinase (1-PFK) and fructose-1,6-bisphosphate aldolase (FBA), respectively, as well as 1-PFK and FBA activities were specifically upregulated in fructose-grown cells. pfkB and fba knockout mutants did not grow on fructose, whereas growth on glucose was not inhibited, indicating the functional involvement of both enzymes in fructose catabolism. Recombinant 1-PFK and FBA obtained after homologous overexpression were characterized as having kinetic properties indicative of functional 1-PFK and a class II type FBA. From these data, we conclude that fructose uptake in H. volcanii involves a fructose-specific PTS generating fructose-1-phosphate, which is further converted via fructose-1,6-bisphosphate to triose phosphates by 1-PFK and FBA. This is the first report of the functional involvement of a bacterial-like PTS and of class II FBA in the sugar metabolism of archaea.<br/>
</details><br/>
---

*A genetic investigation of the KEOPS complex in halophilic Archaea.*<br/>
Naor A; Thiaville PC; Altman-Price N; Cohen-Or I; Allers T; de Crécy-Lagard V; Gophna U<br/>
**2012**<br/>
PLoS ONE 7(8)<br/>
10.1371/journal.pone.0043013
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22927945)<br/>
[Citation]()<br/>
KEOPS is an important cellular complex conserved in Eukarya, with some subunits conserved in Archaea and Bacteria. This complex was recently found to play an essential role in formation of the tRNA modification threonylcarbamoyladenosine (t(6)A), and was previously associated with telomere length maintenance and transcription. KEOPS subunits are conserved in Archaea, especially in the Euryarchaea, where they had been studied in vitro. Here we attempted to delete the genes encoding the four conserved subunits of the KEOPS complex in the euryarchaeote Haloferax volcanii and study their phenotypes in vivo. The fused kae1-bud32 gene was shown to be essential as was cgi121, which is dispensable in yeast. In contrast, pcc1 (encoding the putative dimerizing unit of KEOPS) was not essential in H. volcanii. Deletion of pcc1 led to pleiotropic phenotypes, including decreased growth rate, reduced levels of t(6)A modification, and elevated levels of intra-cellular glycation products.<br/>
</details><br/>
---

*Low species barriers in halophilic archaea and the formation of recombinant hybrids.*<br/>
Naor A; Lapierre P; Mevarech M; Papke RT; Gophna U<br/>
**2012**<br/>
Curr. Biol. 22(15)<br/>
10.1016/j.cub.2012.05.056
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22748314)<br/>
[Citation]()<br/>
Speciation of sexually reproducing organisms requires reproductive barriers. Prokaryotes reproduce asexually but often exchange DNA by lateral gene transfer mechanisms and recombination [1], yet distinct lineages are still observed. Thus, barriers to gene flow such as geographic isolation, genetic incompatibility or a physiological inability to transfer DNA represent potential underlying mechanisms behind preferred exchange groups observed in prokaryotes [2-6]. In Bacteria, experimental evidence showed that sequence divergence impedes homologous recombination between bacterial species [7-11]. Here we study interspecies gene exchange in halophilic archaea that possess a parasexual mechanism of genetic exchange that is functional between species [12, 13]. In this process, cells fuse forming a diploid state containing the full genetic repertoire of both parental cells, which facilitates genetic exchange and recombination. Later, cells separate, occasionally resulting in hybrids of the parental strains [14]. We show high recombination frequencies between Haloferax volcanii and Haloferax mediterranei, two species that have an average nucleotide sequence identity of 86.6%. Whole genome sequencing of Haloferax interspecies hybrids revealed the exchange of chromosomal fragments ranging from 310Kb to 530Kb. These results show that recombination barriers may be more permissive in halophilic archaea than they are in bacteria.<br/>
</details><br/>
---

*A predicted geranylgeranyl reductase reduces the ω-position isoprene of dolichol phosphate in the halophilic archaeon, Haloferax volcanii.*<br/>
Naparstek S; Guan Z; Eichler J<br/>
**2012**<br/>
Biochim. Biophys. Acta 1821(6)<br/>
10.1016/j.bbalip.2012.03.002
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22469971)<br/>
[Citation]()<br/>
In N-glycosylation in both Eukarya and Archaea, N-linked oligosaccharides are assembled on dolichol phosphate prior to transfer of the glycan to the protein target. However, whereas only the α-position isoprene subunit is saturated in eukaryal dolichol phosphate, both the α- and ω-position isoprene subunits are reduced in the archaeal lipid. The agents responsible for dolichol phosphate saturation remain largely unknown. The present study sought to identify dolichol phosphate reductases in the halophilic archaeon, Haloferax volcanii. Homology-based searches recognize HVO_1799 as a geranylgeranyl reductase. Mass spectrometry revealed that cells deleted of HVO_1799 fail to fully reduce the isoprene chains of H. volcanii membrane phospholipids and glycolipids. Likewise, the absence of HVO_1799 led to a loss of saturation of the ω-position isoprene subunit of C(55) and C(60) dolichol phosphate, with the effect of HVO_1799 deletion being more pronounced with C(60) dolichol phosphate than with C(55) dolichol phosphate. Glycosylation of dolichol phosphate in the deletion strain occurred preferentially on that version of the lipid saturated at both the α- and ω-position isoprene subunits.<br/>
</details><br/>
---

*The immune system of halophilic archaea.*<br/>
Maier LK; Fischer S; Stoll B; Brendel J; Pfeiffer F; Dyall-Smith M; Marchfelder A<br/>
**2012**<br/>
Mob Genet Elements 2(5)<br/>
10.4161/mge.22530
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23446883)<br/>
[Citation]()<br/>
Prokaryotes have developed several strategies to defend themselves against foreign genetic elements. One of those defense mechanisms is the recently identified CRISPR/Cas system, which is used by approximately half of all bacterial and almost all archaeal organisms. The CRISPR/Cas system differs from the other defense strategies because it is adaptive, hereditary and it recognizes the invader by a sequence specific mechanism. To identify the invading foreign nucleic acid, a crRNA that matches the invader DNA is required, as well as a short sequence motif called protospacer adjacent motif (PAM). We recently identified the PAM sequences for the halophilic archaeon <br/>
</details><br/>
---

*A dual promoter region with overlapping activator sequences drives the expression of gas vesicle protein genes in haloarchaea.*<br/>
Marschaus L; Pfeifer F<br/>
**2012**<br/>
Microbiology (Reading, Engl.) 158(Pt 11)<br/>
10.1099/mic.0.060178-0
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22997463)<br/>
[Citation]()<br/>
Gas vesicle formation in haloarchaea involves 14 gas vesicle protein (gvp) genes. The strong promoter P(A) drives the expression of gvpACNO, which encodes the major gas vesicle structural proteins GvpA and GvpC, whereas the oppositely oriented promoter P(D) initiates the synthesis of the two regulator proteins, GvpD and GvpE. GvpE activates P(A) and P(D), and requires a 20 nt upstream activator sequence (UAS). UAS(A) and UAS(D) partially overlap in the centre of the 35 bp intergenic region. The basal and GvpE-induced activities of P(A) and P(D) were investigated in Haloferax volcanii transformants. Each UAS consists of two 8 nt portions (P(A), 1A+2A; P(D), 1D+2D), and mutations in the overlapping 1A and 1D portions affected the GvpE induction of both promoters. Substitution of one of the UAS portions by a nonsense sequence showed that a complete UAS is required for activation. The activation of P(A) was more efficient compared with P(D). Promoter P(A) with UAS(A) in configuration 1A+1A was still activated by GvpE, but P(D) was not inducible with UAS(D) in configuration 1D+1D. The TATA box and/or transcription factor B recognition element (BRE) were exchanged between P(A) and P(D). All elements of P(A) functioned well in the environment of 'P(D)' and transferred the stronger P(A) activity to 'P(D)'. In contrast, the respective 'P(A)' chimeras were less active, and BRE(D) was not functional in the environment of 'P(A)'. The relative strengths of the two promoters were substantially determined by the BRE. A 4 nt scanning mutagenesis uncovered an additional regulatory element in the region between TATA(D) and the transcriptional start site of gvpD.<br/>
</details><br/>
---

*The only exoribonuclease present in Haloferax volcanii has an unique response to temperature changes.*<br/>
Matos RG; López-Viñas E; Goméz-Puertas P; Arraiano CM<br/>
**2012**<br/>
Biochim. Biophys. Acta 1820(10)<br/>
10.1016/j.bbagen.2012.06.009
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22705677)<br/>
[Citation]()<br/>
BACKGROUND: Little is known regarding mRNA degradation mechanisms in archaea. In some of these single-cell organisms the existence of a complex of exoribonucleases called the exosome has been demonstrated. However, in halophilic archaea the RNase R homologue is essential since it is the only enzyme described with exoribonucleolytic activity.
METHODS: In this work we have characterized the mechanism of action of Haloferax volcanii RNase R and its implications for the RNA degradation process. We have determined the salt, pH and divalent ion preference, and set the best conditions for the activity assays. Furthermore, we have determined the activity of the protein at different temperatures using different substrates. The dissociation constants were also calculated by Surface Plasmon Resonance. Finally, we have built a model and compared it with the Escherichia coli counterparts.
RESULTS: The results obtained showed that at 37°C, in spite of being named RNase R, this protein behaves like an RNase II protein, halting when it reaches secondary structures, and releasing a 4 nt end-product. However, at 42°C, the optimum temperature of growth, this protein is able to degrade secondary structures, acting like RNase R.
GENERAL SIGNIFICANCE: This discovery has a great impact for RNA degradation, since this is the first case reported where a single enzyme has two different exoribonucleolytic activities according to the temperature. Furthermore, the results obtained are very important to help to decipher the RNA degradation mechanisms in H. volcanii, since RNase R is the only exoribonuclease involved in this process.<br/>
</details><br/>
---

*Overexpression in a non-native halophilic host and biotechnological potential of NAD+-dependent glutamate dehydrogenase from Halobacterium salinarum strain NRC-36014.*<br/>
Munawar N; Engel PC<br/>
**2012**<br/>
Extremophiles 16(3)<br/>
10.1007/s00792-012-0446-z
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22527040)<br/>
[Citation]()<br/>
Enzymes produced by halophilic archaea are generally heat resistant and organic solvent tolerant, and accordingly important for biocatalytic applications in 'green chemistry', frequently requiring a low-water environment. NAD(+)-dependent glutamate dehydrogenase from an extremely halophilic archaeon Halobacterium salinarum strain NRC-36014 was selected to explore the biotechnological potential of this enzyme and genetically engineered derivatives. Over-expression in a halophilic host Haloferax volcanii provided a soluble, active recombinant enzyme, not achievable in mesophilic Escherichia coli, and an efficient purification procedure was developed. pH and salt dependence, thermostability, organic solvent stability and kinetic parameters were explored. The enzyme is active up to 90 °C and fully stable up to 70 °C. It shows good tolerance of various miscible organic solvents. High concentrations of salt may be substituted with 30 % DMSO or betaine with good stability and activity. The robustness of this enzyme under a wide range of conditions offers a promising scaffold for protein engineering.<br/>
</details><br/>
---

*AglR is required for addition of the final mannose residue of the N-linked glycan decorating the Haloferax volcanii S-layer glycoprotein.*<br/>
Kaminski L; Guan Z; Abu-Qarn M; Konrad Z; Eichler J<br/>
**2012**<br/>
Biochim. Biophys. Acta 1820(10)<br/>
10.1016/j.bbagen.2012.06.014
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22750201)<br/>
[Citation]()<br/>
BACKGROUND: Recent studies of Haloferax volcanii have begun to elucidate the steps of N-glycosylation in Archaea, where this universal post-translational modification remains poorly described. In Hfx. volcanii, a series of Agl proteins catalyzes the assembly and attachment of a N-linked pentasaccharide to the S-layer glycoprotein. Although roles have been assigned to the majority of Agl proteins, others await description. In the following, the contribution of AglR to N-glycosylation was addressed.
METHODS: A combination of bioinformatics, gene deletion, mass spectrometry and metabolic radiolabeling served to show a role for AglR in archaeal N-glycosylation at both the dolichol phosphate and reporter glycoprotein levels.
RESULTS: The modified behavior of the S-layer glycoprotein isolated from cells lacking AglR points to an involvement of this protein in N-glycosylation. In cells lacking AglR, glycan-charged dolichol phosphate, including mannose-charged dolichol phosphate, accumulates. At the same time, the S-layer glycoprotein does not incorporate mannose, the final subunit of the N-linked pentasaccharide decorating this protein. AglR is a homologue of Wzx proteins, annotated as flippases responsible for delivering lipid-linked O-antigen precursor oligosaccharides across the bacterial plasma membrane during lipopolysaccharide biogenesis.
CONCLUSIONS: The effects resulting from aglR deletion are consistent with AglR interacting with dolichol phosphate-mannose, possibly acting as a dolichol phosphate-mannose flippase or contributing to such activity.
GENERAL SIGNIFICANCE: Little is known of how lipid-linked oligosaccharides are translocated across membrane during N-glycosylation. The possibility of Hfx. volcanii AglR mediating or contributing to flippase activity could help address this situation.<br/>
</details><br/>
---

*Chemical cross-linking, mass spectrometry, and in silico modeling of proteasomal 20S core particles of the haloarchaeon Haloferax volcanii.*<br/>
Karadzic I; Maupin-Furlow J; Humbard M; Prunetti L; Singh P; Goodlett DR<br/>
**2012**<br/>
Proteomics 12(11)<br/>
10.1002/pmic.201100260
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22623373)<br/>
[Citation]()<br/>
A fast and accurate method is reported to generate distance constraints between juxtaposited amino acids and to validate molecular models of halophilic protein complexes. Proteasomal 20S core particles (CPs) from the haloarchaeon Haloferax volcanii were used to investigate the quaternary structure of halophilic proteins based on their symmetrical, yet distinct subunit composition. Proteasomal CPs are cylindrical barrel-like structures of four-stacked homoheptameric rings of α- and β-type subunits organized in α(7)β(7) β(7)α(7) stoichiometry. The CPs of H. volcanii are formed from a single type of β subunit associated with α1 and/or α2 subunits. Tandem affinity chromatography and new genetic constructs were used to separately isolate α1(7)β(7)β(7)α1(7) and α2(7)β(7)β(7)α2(7) CPs from H. volcanii. Chemically cross-linked peptides of the H. volcanii CPs were analyzed by high-performance mass spectrometry and an open modification search strategy to first generate and then to interpret the resulting tandem mass spectrometric data. Distance constraints obtained by chemical cross-linking mass spectrometry, together with the available structural data of nonhalophilic CPs, facilitated the selection of accurate models of H. volcanii proteasomal CPs composed of α1-, α2-, and β-homoheptameric rings from several different possible structures from Protein Data Bank.<br/>
</details><br/>
---

*Diversity in prokaryotic glycosylation: an archaeal-derived N-linked glycan contains legionaminic acid.*<br/>
Kandiba L; Aitio O; Helin J; Guan Z; Permi P; Bamford DH; Eichler J; Roine E<br/>
**2012**<br/>
Mol. Microbiol. 84(3)<br/>
10.1111/j.1365-2958.2012.08045.x
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22435790)<br/>
[Citation]()<br/>
VP4, the major structural protein of the haloarchaeal pleomorphic virus, HRPV-1, is glycosylated. To define the glycan structure attached to this protein, oligosaccharides released by β-elimination were analysed by mass spectrometry and nuclear magnetic resonance spectroscopy. Such analyses showed that the major VP4-derived glycan is a pentasaccharide comprising glucose, glucuronic acid, mannose, sulphated glucuronic acid and a terminal 5-N-formyl-legionaminic acid residue. This is the first observation of legionaminic acid, a sialic acid-like sugar, in an archaeal-derived glycan structure. The importance of this residue for viral infection was demonstrated upon incubation with N-acetylneuraminic acid, a similar monosaccharide. Such treatment reduced progeny virus production by half 4 h post infection. LC-ESI/MS analysis confirmed the presence of pentasaccharide precursors on two different VP4-derived peptides bearing the N-glycosylation signal, NTT. The same sites modified by the native host, Halorubrum sp. strain PV6, were also recognized by the Haloferax volcanii N-glycosylation apparatus, as determined by LC-ESI/MS of heterologously expressed VP4. Here, however, the N-linked pentasaccharide was the same as shown to decorate the S-layer glycoprotein in this species. Hence, N-glycosylation of the haloarchaeal viral protein, VP4, is host-specific. These results thus present additional examples of archaeal N-glycosylation diversity and show the ability of Archaea to modify heterologously expressed proteins.<br/>
</details><br/>
---

*Archaeal JAB1/MPN/MOV34 metalloenzyme (HvJAMM1) cleaves ubiquitin-like small archaeal modifier proteins (SAMPs) from protein-conjugates.*<br/>
Hepowit NL; Uthandi S; Miranda HV; Toniutti M; Prunetti L; Olivarez O; De Vera IM; Fanucci GE; Chen S; Maupin-Furlow JA<br/>
**2012**<br/>
Mol. Microbiol. 86(4)<br/>
10.1111/mmi.12038
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22970855)<br/>
[Citation]()<br/>
Proteins with JAB1/MPN/MOV34 metalloenzyme (JAMM/MPN+) domains are widespread among all domains of life, yet poorly understood. Here we report the purification and characterization of an archaeal JAMM/MPN+ domain protein (HvJAMM1) from Haloferax volcanii that cleaves ubiquitin-like small archaeal modifier proteins (SAMP1/2) from protein conjugates. HvJAMM1 cleaved SAMP1/2 conjugates generated in H. volcanii as well as isopeptide- and linear-linked SAMP1-MoaE in purified form. Cleavage of linear linked SAMP1-MoaE was dependent on the presence of the SAMP domain and the C-terminal VSGG motif of this domain. While HvJAMM1 was inhibited by size exclusion chromatography and metal chelators, its activity could be restored by addition of excess ZnCl2 . HvJAMM1 residues (Glu31, His88, His90, Ser98 and Asp101) that were conserved with the JAMM/MPN+ active-site motif were required for enzyme activity. Together, these results provide the first example of a JAMM/MPN+ zinc metalloprotease that independently catalyses the cleavage of ubiquitin-like (isopeptide and linear) bonds from target proteins. In archaea, HvJAMM1 likely regulates sampylation and the pools of 'free' SAMP available for protein modification. HvJAMM1-type proteins are thought to release the SAMPs from proteins modified post-translationally as well as those synthesized as domain fusions.<br/>
</details><br/>
---

*High throughput sequencing reveals a plethora of small RNAs including tRNA derived fragments in Haloferax volcanii.*<br/>
Heyer R; Dörr M; Jellen-Ritter A; Späth B; Babski J; Jaschinski K; Soppa J; Marchfelder A<br/>
**2012**<br/>
RNA Biol 9(7)<br/>
10.4161/rna.20826
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22767255)<br/>
[Citation]()<br/>
To define the complete sRNA population of the halophilic archaeon Haloferax volcanii, we employed high throughput sequencing. cDNAs were generated from RNA ranging in size from 17 to 500 nucleotides isolated from cells grown at three different conditions to exponential and stationary phase, respectively. Altogether, 145 intergenic and 45 antisense sRNAs were identified. Comparison of the expression profile showed different numbers of reads at the six different conditions for the majority of sRNAs. A striking difference in the number of sRNA reads was observed between cells grown under standard vs. low salt conditions. Furthermore, the six highest numbers of reads were found for low salt conditions. In contrast, only slight differences between sRNA reads at different growth temperatures were detected. Attempts to delete four sRNA genes revealed that one sRNA gene is essential. The three viable sRNA gene deletion mutants possessed distinct phenotypes. According to microarray analyses, the removal of the sRNA gene resulted in a profound change of the transcriptome when compared with the wild type. High throughput sequencing also showed the presence of high concentrations of tRNA derived fragments in H. volcanii. These tRF molecules were shown to have different amounts of reads at the six conditions analyzed. Northern analysis was used to confirm the presence of the tRNA-derived fragments.<br/>
</details><br/>
---

*tRNA-like elements in Haloferax volcanii.*<br/>
Hölzle A; Stoll B; Schnattinger T; Schöning U; Tjaden B; Marchfelder A<br/>
**2012**<br/>
Biochimie 94(4)<br/>
10.1016/j.biochi.2011.12.002
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22178322)<br/>
[Citation]()<br/>
All functional RNAs are generated from precursor molecules by a plethora of processing steps. The generation of mature RNA molecules by processing is an important layer of gene expression regulation catalysed by ribonucleases. Here, we analysed 5S rRNA processing in the halophilic Archaeon Haloferax volcanii. Earlier experiments showed that the 5S rRNA is cleaved at its 5' end by the endonuclease tRNase Z. Interestingly, a tRNA-like structure was identified upstream of the 5S rRNA that might be used as a processing signal. Here, we show that this tRNA-like element is indeed recognised as a processing signal by tRNase Z. Substrates containing mutations in the tRNA-like sequence are no longer processed, whereas a substrate containing a deletion in the 5S rRNA sequence is still cleaved. Therefore, an intact 5S rRNA structure is not required for processing. Further, we used bioinformatics analyses to identify additional sequences in Haloferax containing tRNA-like structures. This search resulted in the identification of all tRNAs, the tRNA-like structure upstream of the 5S RNA and 47 new tRNA-like structural elements. However, the in vitro processing of selected examples showed no cleavage of these newly identified elements. Thus, tRNA-like elements are not a general processing signal in Haloferax.<br/>
</details><br/>
---

*tRNA-derived fragments target the ribosome and function as regulatory non-coding RNA in Haloferax volcanii.*<br/>
Gebetsberger J; Zywicki M; Künzi A; Polacek N<br/>
**2012**<br/>
Archaea 2012(None)<br/>
2012/260909
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23326205)<br/>
[Citation]()<br/>
Nonprotein coding RNA (ncRNA) molecules have been recognized recently as major contributors to regulatory networks in controlling gene expression in a highly efficient manner. These RNAs either originate from their individual transcription units or are processing products from longer precursor RNAs. For example, tRNA-derived fragments (tRFs) have been identified in all domains of life and represent a growing, yet functionally poorly understood, class of ncRNA candidates. Here we present evidence that tRFs from the halophilic archaeon Haloferax volcanii directly bind to ribosomes. In the presented genomic screen of the ribosome-associated RNome, a 26-residue-long fragment originating from the 5' part of valine tRNA was by far the most abundant tRF. The Val-tRF is processed in a stress-dependent manner and was found to primarily target the small ribosomal subunit in vitro and in vivo. As a consequence of ribosome binding, Val-tRF reduces protein synthesis by interfering with peptidyl transferase activity. Therefore this tRF functions as ribosome-bound small ncRNA capable of regulating gene expression in H. volcanii under environmental stress conditions probably by fine tuning the rate of protein production.<br/>
</details><br/>
---

*Protein glycosylation as an adaptive response in Archaea: growth at different salt concentrations leads to alterations in Haloferax volcanii S-layer glycoprotein N-glycosylation.*<br/>
Guan Z; Naparstek S; Calo D; Eichler J<br/>
**2012**<br/>
Environ. Microbiol. 14(3)<br/>
10.1111/j.1462-2920.2011.02625.x
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22029420)<br/>
[Citation]()<br/>
To cope with life in hypersaline environments, halophilic archaeal proteins are enriched in acidic amino acids. This strategy does not, however, offer a response to transient changes in salinity, as would post-translational modifications. To test this hypothesis, N-glycosylation of the Haloferax volcanii S-layer glycoprotein was compared in cells grown in high (3.4 M NaCl) and low (1.75 M NaCl) salt, as was the glycan bound to dolichol phosphate, the lipid upon which the N-linked glycan is assembled. In high salt, S-layer glycoprotein Asn-13 and Asn-83 are modified by a pentasaccharide, while dolichol phosphate is modified by a tetrasaccharide comprising the first four pentasaccharide residues. When the same targets were considered from cells grown in low salt, substantially less pentasaccharide was detected. At the same time, cells grown at low salinity contain dolichol phosphate modified by a distinct tetrasaccharide absent in cells grown at high salinity. The same tetrasaccharide modified S-layer glycoprotein Asn-498 in cells grown in low salt, whereas no glycan decorated this residue in cells grown in the high-salt medium. Thus, in response to changes in environmental salinity, Hfx. volcanii not only modulates the N-linked glycans decorating the S-layer glycoprotein but also the sites of such post-translational modification.<br/>
</details><br/>
---

*Biofilm formation by haloarchaea.*<br/>
Fröls S; Dyall-Smith M; Pfeifer F<br/>
**2012**<br/>
Environ. Microbiol. 14(12)<br/>
10.1111/j.1462-2920.2012.02895.x
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23057712)<br/>
[Citation]()<br/>
A fluorescence-based live-cell adhesion assay was used to examine biofilm formation by 20 different haloarchaea, including species of Halobacterium, Haloferax and Halorubrum, as well as novel natural isolates from an Antarctic salt lake. Thirteen of the 20 tested strains significantly adhered (P-value  < 0.05) to a plastic surface. Examination of adherent cell layers on glass surfaces by differential interference contrast, fluorescence and confocal microscopy showed two types of biofilm structures. Carpet-like, multi-layered biofilms containing micro- and macrocolonies (up to 50 μm in height) were formed by strains of Halobacterium salinarum and the Antarctic isolate t-ADL strain DL24. The second type of biofilm, characterized by large aggregates of cells adhering to surfaces, was formed by Haloferax volcanii DSM 3757T and Halorubrum lacusprofundi DL28. Staining of the biofilms formed by the strongly adhesive haloarchaeal strains revealed the presence of extracellular polymers, such as eDNA and glycoconjugates, substances previously shown to stabilize bacterial biofilms. For Hbt. salinarum DSM 3754T and Hfx. volcanii DSM 3757T , cells adhered within 1 day of culture and remained viable for at least 2 months in mature biofilms. Adherent cells of Hbt. salinarum DSM 3754T showed several types of cellular appendages that could be involved in the initial attachment. Our results show that biofilm formation occurs in a surprisingly wide variety of haloarchaeal species.<br/>
</details><br/>
---

*An archaeal immune system can detect multiple protospacer adjacent motifs (PAMs) to target invader DNA.*<br/>
Fischer S; Maier LK; Stoll B; Brendel J; Fischer E; Pfeiffer F; Dyall-Smith M; Marchfelder A<br/>
**2012**<br/>
J. Biol. Chem. 287(40)<br/>
10.1074/jbc.M112.377002
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22767603)<br/>
[Citation]()<br/>
The clustered regularly interspaced short palindromic repeat (CRISPR)/CRISPR-associated (Cas) system provides adaptive and heritable immunity against foreign genetic elements in most archaea and many bacteria. Although this system is widespread and diverse with many subtypes, only a few species have been investigated to elucidate the precise mechanisms for the defense of viruses or plasmids. Approximately 90% of all sequenced archaea encode CRISPR/Cas systems, but their molecular details have so far only been examined in three archaeal species: Sulfolobus solfataricus, Sulfolobus islandicus, and Pyrococcus furiosus. Here, we analyzed the CRISPR/Cas system of Haloferax volcanii using a plasmid-based invader assay. Haloferax encodes a type I-B CRISPR/Cas system with eight Cas proteins and three CRISPR loci for which the identity of protospacer adjacent motifs (PAMs) was unknown until now. We identified six different PAM sequences that are required upstream of the protospacer to permit target DNA recognition. This is only the second archaeon for which PAM sequences have been determined, and the first CRISPR group with such a high number of PAM sequences. Cells could survive the plasmid challenge if their CRISPR/Cas system was altered or defective, e.g. by deletion of the cas gene cassette. Experimental PAM data were supplemented with bioinformatics data on Haloferax and Haloquadratum.<br/>
</details><br/>
---

*Assigning a function to a conserved archaeal metallo-β-lactamase from Haloferax volcanii.*<br/>
Fischer S; John von Freyend S; Sabag-Daigle A; Daniels CJ; Allers T; Marchfelder A<br/>
**2012**<br/>
Extremophiles 16(2)<br/>
10.1007/s00792-012-0433-4
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22350204)<br/>
[Citation]()<br/>
The metallo-β-lactamase family of enzymes comprises a large group of proteins with diverse functions in the metabolism of the cell. Among others, this superfamily contains proteins which are involved in DNA and RNA metabolism, acting as nucleases in e.g. repair and maturation. Many proteins have been annotated in prokaryotic genomes as being potential metallo-β-lactamases, but very often the function has not been proven. The protein HVO_2763 from Haloferax volcanii is such a potential metallo-β-lactamase. HVO_2763 has sequence similarity to the metallo-β-lactamase tRNase Z, a tRNA 3' processing endonuclease. Here, we report the characterisation of this metallo-β-lactamase HVO_2763 in the halophilic archaeon Haloferax volcanii. Using different in vitro assays with the recombinant HVO_2763, we could show that the protein does not have tRNA 3' processing or exonuclease activity. According to transcriptome analyses of the HVO_2763 deletion strain, expression of proteins involved in membrane transport is downregulated in the mutant. Therefore, HVO_2763 might be involved directly or indirectly in membrane transport.<br/>
</details><br/>
---

*AglS, a novel component of the Haloferax volcanii N-glycosylation pathway, is a dolichol phosphate-mannose mannosyltransferase.*<br/>
Cohen-Rosenzweig C; Yurist-Doutsch S; Eichler J<br/>
**2012**<br/>
J. Bacteriol. 194(24)<br/>
10.1128/JB.01716-12
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23086206)<br/>
[Citation]()<br/>
In Haloferax volcanii, a series of Agl proteins mediates protein N-glycosylation. The genes encoding all but one of the Agl proteins are sequestered into a single gene island. The same region of the genome includes sequences also suspected but not yet verified as serving N-glycosylation roles, such as HVO_1526. In the following, HVO_1526, renamed AglS, is shown to be necessary for the addition of the final mannose subunit of the pentasaccharide N-linked to the surface (S)-layer glycoprotein, a convenient reporter of N-glycosylation in Hfx. volcanii. Relying on bioinformatics, topological analysis, gene deletion, mass spectrometry, and biochemical assays, AglS was shown to act as a dolichol phosphate-mannose mannosyltransferase, mediating the transfer of mannose from dolichol phosphate to the tetrasaccharide corresponding to the first four subunits of the pentasaccharide N-linked to the S-layer glycoprotein.<br/>
</details><br/>
---

*Prokaryotic sex: eukaryote-like qualities of recombination in an Archaean lineage.*<br/>
Cohan FM; Aracena S<br/>
**2012**<br/>
Curr. Biol. 22(15)<br/>
10.1016/j.cub.2012.06.052
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22877780)<br/>
[Citation]()<br/>
Genetic exchange within one Archaean lineage is a bit like sex in eukaryotes - cells fuse and huge segments of DNA are recombined - with consequences for the spread of adaptations across species.<br/>
</details><br/>
---

*The archaeal COG1901/DUF358 SPOUT-methyltransferase members, together with pseudouridine synthase Pus10, catalyze the formation of 1-methylpseudouridine at position 54 of tRNA.*<br/>
Chatterjee K; Blaby IK; Thiaville PC; Majumder M; Grosjean H; Yuan YA; Gupta R; de Crécy-Lagard V<br/>
**2012**<br/>
RNA 18(3)<br/>
10.1261/rna.030841.111
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22274953)<br/>
[Citation]()<br/>
The methylation of pseudouridine (Ψ) at position 54 of tRNA, producing m(1)Ψ, is a hallmark of many archaeal species, but the specific methylase involved in the formation of this modification had yet to be characterized. A comparative genomics analysis had previously identified COG1901 (DUF358), part of the SPOUT superfamily, as a candidate for this missing methylase family. To test this prediction, the COG1901 encoding gene, HVO_1989, was deleted from the Haloferax volcanii genome. Analyses of modified base contents indicated that while m(1)Ψ was present in tRNA extracted from the wild-type strain, it was absent from tRNA extracted from the mutant strain. Expression of the gene encoding COG1901 from Halobacterium sp. NRC-1, VNG1980C, complemented the m(1)Ψ minus phenotype of the ΔHVO_1989 strain. This in vivo validation was extended with in vitro tests. Using the COG1901 recombinant enzyme from Methanocaldococcus jannaschii (Mj1640), purified enzyme Pus10 from M. jannaschii and full-size tRNA transcripts or TΨ-arm (17-mer) fragments as substrates, the sequential pathway of m(1)Ψ54 formation in Archaea was reconstituted. The methylation reaction is AdoMet dependent. The efficiency of the methylase reaction depended on the identity of the residue at position 55 of the TΨ-loop. The presence of Ψ55 allowed the efficient conversion of Ψ54 to m(1)Ψ54, whereas in the presence of C55, the reaction was rather inefficient and no methylation reaction occurred if a purine was present at this position. These results led to renaming the Archaeal COG1901 members as TrmY proteins.<br/>
</details><br/>
---

*Chromatin is an ancient innovation conserved between Archaea and Eukarya.*<br/>
Ammar R; Torti D; Tsui K; Gebbia M; Durbic T; Bader GD; Giaever G; Nislow C<br/>
**2012**<br/>
Elife 1(None)<br/>
10.7554/eLife.00078
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/23240084)<br/>
[Citation]()<br/>
The eukaryotic nucleosome is the fundamental unit of chromatin, comprising a protein octamer that wraps ∼147 bp of DNA and has essential roles in DNA compaction, replication and gene expression. Nucleosomes and chromatin have historically been considered to be unique to eukaryotes, yet studies of select archaea have identified homologs of histone proteins that assemble into tetrameric nucleosomes. Here we report the first archaeal genome-wide nucleosome occupancy map, as observed in the halophile Haloferax volcanii. Nucleosome occupancy was compared with gene expression by compiling a comprehensive transcriptome of Hfx. volcanii. We found that archaeal transcripts possess hallmarks of eukaryotic chromatin structure: nucleosome-depleted regions at transcriptional start sites and conserved -1 and +1 promoter nucleosomes. Our observations demonstrate that histones and chromatin architecture evolved before the divergence of Archaea and Eukarya, suggesting that the fundamental role of chromatin in the regulation of gene expression is ancient.DOI:http://dx.doi.org/10.7554/eLife.00078.001.<br/>
</details><br/>
---

*A haloarchaeal ferredoxin electron donor that plays an essential role in nitrate assimilation.*<br/>
Zafrilla B; Martínez-Espinosa RM; Bonete MJ; Butt JN; Richardson DJ; Gates AJ<br/>
**2011**<br/>
Biochem. Soc. Trans. 39(6)<br/>
10.1042/BST20110709
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22103537)<br/>
[Citation]()<br/>
In the absence of ammonium, many organisms, including the halophilic archaeon Haloferax volcanii DS2 (DM3757), may assimilate inorganic nitrogen from nitrate or nitrite, using a ferredoxin-dependent assimilatory NO₃⁻/NO₂⁻ reductase pathway. The small acidic ferredoxin Hv-Fd plays an essential role in the electron transfer cascade required for assimilatory nitrate and nitrite reduction by the cytoplasmic NarB- and NirA-type reductases respectively. UV-visible absorbance and EPR spectroscopic characterization of purified Hv-Fd demonstrate that this protein binds a single [2Fe-2S] cluster, and potentiometric titration reveals that the cluster shares similar redox properties with those present in plant-type ferredoxins.<br/>
</details><br/>
---

*Functional insight into the C-terminal extension of halolysin SptA from haloarchaeon Natrinema sp. J7.*<br/>
Xu Z; Du X; Li T; Gan F; Tang B; Tang XF<br/>
**2011**<br/>
PLoS ONE 6(8)<br/>
10.1371/journal.pone.0023562
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21886797)<br/>
[Citation]()<br/>
Halolysin SptA from haloarchaeon Natrinema sp. J7 consists of a subtilisin-like catalytic domain and a C-terminal extension (CTE) containing two cysteine residues. In this report, we have investigated the function of the CTE using recombinant enzymes expressed in Haloferax volcanii WFD11. Deletion of the CTE greatly reduced but did not abolish protease activity, which suggests that the CTE is not essential for enzyme folding. Mutational analysis suggests that residues Cys303 and Cys338 within the CTE form a disulfide bond that make this domain resistant to autocleavage and proteolysis under hypotonic conditions. Characterization of full-length and CTE-truncation enzymes indicates the CTE not only confers extra stability to the enzyme but also assists enzyme activity on protein substrates by facilitating binding at high salinities. Interestingly, homology modeling of the CTE yields a β-jelly roll-like structure similar to those seen in Claudin-binding domain of Clostridium perfringens enterotoxin (clostridial C-CPE) and collagen binding domain (CBD), and the CTE also possesses collagen-binding activity, making it a potential candidate as an anchoring unit in drug delivery systems.<br/>
</details><br/>
---

*Functional genomic and advanced genetic studies reveal novel insights into the metabolism, regulation, and biology of Haloferax volcanii.*<br/>
Soppa J<br/>
**2011**<br/>
Archaea 2011(None)<br/>
2011/602408
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22190865)<br/>
[Citation]()<br/>
The genome sequence of Haloferax volcanii is available and several comparative genomic in silico studies were performed that yielded novel insight for example into protein export, RNA modifications, small non-coding RNAs, and ubiquitin-like Small Archaeal Modifier Proteins. The full range of functional genomic methods has been established and results from transcriptomic, proteomic and metabolomic studies are discussed. Notably, Hfx. volcanii is together with Halobacterium salinarum the only prokaryotic species for which a translatome analysis has been performed. The results revealed that the fraction of translationally-regulated genes in haloarchaea is as high as in eukaryotes. A highly efficient genetic system has been established that enables the application of libraries as well as the parallel generation of genomic deletion mutants. Facile mutant generation is complemented by the possibility to culture Hfx. volcanii in microtiter plates, allowing the phenotyping of mutant collections. Genetic approaches are currently used to study diverse biological questions-from replication to posttranslational modification-and selected results are discussed. Taken together, the wealth of functional genomic and genetic tools make Hfx. volcanii a bona fide archaeal model species, which has enabled the generation of important results in recent years and will most likely generate further breakthroughs in the future.<br/>
</details><br/>
---

*Activity and transcriptional regulation of bacterial protein-like glycerol-3-phosphate dehydrogenase of the haloarchaea in Haloferax volcanii.*<br/>
Rawls KS; Martin JH; Maupin-Furlow JA<br/>
**2011**<br/>
J. Bacteriol. 193(17)<br/>
10.1128/JB.00276-11
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21725010)<br/>
[Citation]()<br/>
Glycerol is a primary energy source for heterotrophic haloarchaea and a major component of "salty" biodiesel waste. Glycerol is catabolized solely by glycerol kinase (encoded by glpK) to glycerol-3-phosphate (G3P) in Haloferax volcanii. Here we characterized the next critical step of this metabolic pathway: the conversion of G3P to dihydroxyacetone phosphate by G3P dehydrogenase (G3PDH). H. volcanii harbors two putative G3PDH operons: (i) glpA1B1C1, located on the chromosome within the neighborhood of glpK, and (ii) glpA2B2C2, on megaplasmid pHV4. Analysis of knockout strains revealed that glpA1(and not glpA2) is required for growth on glycerol. However, both glpA1 and glpA2 could complement a glpA1 knockout strain (when expressed from a strong promoter in trans) and were required for the total G3PDH activity of cell lysates. The glpA1B1C1, glpK, glpF(encoding a putative glycerol facilitator), and ptsH2(encoding a homolog of the bacterial phosphotransferase system protein Hpr) genes were transcriptionally linked and appeared to be under the control of a strong, G3P-inducible promoter upstream of glpA1. Overall, this study provides fundamental insights into glycerol metabolism in H. volcanii and enhances our understanding of central metabolic pathways of haloarchaea.<br/>
</details><br/>
---

*Halophilic enzyme activation induced by salts.*<br/>
Ortega G; Laín A; Tadeo X; López-Méndez B; Castaño D; Millet O<br/>
**2011**<br/>
Sci Rep 1(None)<br/>
10.1038/srep00006
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22355525)<br/>
[Citation]()<br/>
Halophilic archea (halobacteriae) thrive in hypersaline environments, avoiding osmotic shock by increasing the ion concentration of their cytoplasm by up to 3-6 M. To remain folded and active, their constitutive proteins have evolved towards a biased amino acid composition. High salt concentration affects catalytic activity in an enzyme-dependent way and a unified molecular mechanism remains elusive. Here, we have investigated a DNA ligase from Haloferax volcanii (Hv LigN) to show that K(+) triggers catalytic activity by preferentially stabilising a specific conformation in the reaction coordinate. Sodium ions, in turn, do not populate such isoform and the enzyme remains inactive in the presence of this co-solute. Our results show that the halophilic amino acid signature enhances the enzyme's thermodynamic stability, with an indirect effect on its catalytic activity. This model has been successfully applied to reengineer Hv LigN into an enzyme that is catalytically active in the presence of NaCl.<br/>
</details><br/>
---

*Phenotyping in the archaea: optimization of growth parameters and analysis of mutants of Haloferax volcanii.*<br/>
Jantzer K; Zerulla K; Soppa J<br/>
**2011**<br/>
FEMS Microbiol. Lett. 322(2)<br/>
10.1111/j.1574-6968.2011.02341.x
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21692831)<br/>
[Citation]()<br/>
A method to grow the halophilic archaeon Haloferax volcanii in microtiter plates has been optimized and now allows the parallel generation of very reproducible growth curves. The doubling time in a synthetic medium with glucose is around 6 h. The method was used to optimize glucose and casamino acid concentrations, to clarify carbon source usage and to analyze vitamin dependence. The characterization of osmotolerance revealed that after a lag phase of 24 h, H. volcanii is able to grow at salt concentrations as low as 0.7 M NaCl, much lower than the 1.4 M NaCl described as the lowest concentration until now. The application of oxidative stresses showed that H. volcanii exhibits a reaction to paraquat that is delayed by about 10 h. Surprisingly, only one of two amino acid auxotrophic mutants could be fully supplemented by the addition of the respective amino acid. Analysis of eight sRNA gene deletion mutants exemplified that the method can be applied for bona fide phenotyping of mutant collections. This method for the parallel analysis of many cultures contributes towards making H. volcanii an archaeal model species for functional genomic approaches.<br/>
</details><br/>
---

*2'-O-methylation of the wobble residue of elongator pre-tRNA(Met) in Haloferax volcanii is guided by a box C/D RNA containing unique features.*<br/>
Joardar A; Malliahgari SR; Skariah G; Gupta R<br/>
**2011**<br/>
RNA Biol 8(5)<br/>
10.4161/rna.8.5.16015
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21654217)<br/>
[Citation]()<br/>
The wobble residue C34 of Haloferax volcanii elongator tRNA(Met) is 2'-O-methylated. Neither a protein enzyme nor a guide RNA for this modification has been described. In this study, we show that this methylation is guided by a box C/D RNA targeting the intron-containing precursor of the tRNA. This guide RNA is starkly different from its homologs. This unique RNA of approximately 75 bases, named sR-tMet, is encoded in the genomes of H. volcanii and several other haloarchaea. A unique feature of sR-tMet is that the mature RNA in H. volcanii is substantially larger than its predicted size, whereas those in other haloarchaea are as predicted. While the 5'-ends of all tested haloarchaeal sR-tMets are equivalent, H. volcanii sR-tMet possesses an additional 51-base extension at its 3' end. This extension is present in the precursor but not in the mature sR-tMet of Halobacterium sp., suggesting differential 3'-end processing of sR-tMet in these two closely related organisms. Archaeal box C/D RNAs mostly contain a K-loop at the C'/D' motif. Another unique feature of sR-tMet is that its C'/D' motif lacks either a conventional K-turn or a K-loop. Instead, it contains two tandem, sheared G•A base pairs and a pyrimidine-pyrimidine pair in the non-canonical stem; the latter may form an alternative K-turn. Gel shift assays indicate that the L7Ae protein can form a stable complex with this unusual C'/D' motif, suggesting a novel RNA structure for L7Ae interaction.<br/>
</details><br/>
---

*N-linked glycosylation in Archaea: two paths to the same glycan.*<br/>
Guerry P<br/>
**2011**<br/>
Mol. Microbiol. 81(5)<br/>
10.1111/j.1365-2958.2011.07782.x
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21848800)<br/>
[Citation]()<br/>
N-linked protein glycosylation occurs in all three branches of life, eukaryotes, bacteria and archaea. The simplest system is that of the bacterium, Campylobacter jejuni, in which a heptasaccharide glycan is added to multiple proteins from a single lipid carrier molecule. In the eukaryotic system a conserved tetradecasaccharide modification is first added to target proteins, but is then modified by trimming and addition of other glycans from additional carrier molecules resulting in a diverse array of glycans of distinct functionality. In the halophilic Archaea from the Dead Sea, Haloferax volcanii, the surface array or S-layer protein is glycosylated with a pentasaccharide. This glycan is synthesized from two separate carrier molecules, one that carries a tetrasaccharide and another that carries the terminal mannose, in a process that is analogous to that of eukaryotes. In this issue of Molecular Microbiology the glycosylation of the S-layer of another halophilic Archaea from the Dead Sea, Haloarcula marismortui is characterized (Calo et al., 2011). This S-layer is glycosylated with the same pentasaccharide as that of Hfx.  volcanii, but the intact pentasaccharide is synthesized on a single carrier molecule in Har. marismortui in a process that more closely resembles that of the bacterial N-linked system.<br/>
</details><br/>
---

*Bacterioopsin-mediated regulation of bacterioruberin biosynthesis in Halobacterium salinarum.*<br/>
Dummer AM; Bonsall JC; Cihla JB; Lawry SM; Johnson GC; Peck RF<br/>
**2011**<br/>
J. Bacteriol. 193(20)<br/>
10.1128/JB.05376-11
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21840984)<br/>
[Citation]()<br/>
Integral membrane protein complexes consisting of proteins and small molecules that act as cofactors have important functions in all organisms. To form functional complexes, cofactor biosynthesis must be coordinated with the production of corresponding apoproteins. To examine this coordination, we study bacteriorhodopsin (BR), a light-induced proton pump in the halophilic archaeon Halobacterium salinarum. This complex consists of a retinal cofactor and bacterioopsin (BO), the BR apoprotein. To examine possible novel regulatory mechanisms linking BO and retinal biosynthesis, we deleted bop, the gene that encodes BO. bop deletion resulted in a dramatic increase of bacterioruberins, carotenoid molecules that share biosynthetic precursors with retinal. Additional studies revealed that bacterioruberins accumulate in the absence of BO regardless of the presence of retinal or BR, suggesting that BO inhibits bacterioruberin biosynthesis to increase the availability of carotenoid precursors for retinal biosynthesis. To further examine this potential regulatory mechanism, we characterized an enzyme, encoded by the lye gene, that catalyzes bacterioruberin biosynthesis. BO-mediated inhibition of bacterioruberin synthesis appears to be specific to the H. salinarum lye-encoded enzyme, as expression of a lye homolog from Haloferax volcanii, a related archaeon that synthesizes bacterioruberins but lacks opsins, resulted in bacterioruberin synthesis that was not reduced in the presence of BO. Our results provide evidence for a novel regulatory mechanism in which biosynthesis of a cofactor is promoted by apoprotein-mediated inhibition of an alternate biochemical pathway. Specifically, BO accumulation promotes retinal production by inhibiting bacterioruberin biosynthesis.<br/>
</details><br/>
---

*Mauve assembly metrics.*<br/>
Darling AE; Tritt A; Eisen JA; Facciotti MT<br/>
**2011**<br/>
Bioinformatics 27(19)<br/>
bioinformatics/btr451
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21810901)<br/>
[Citation]()<br/>
SUMMARY: High-throughput DNA sequencing technologies have spurred the development of numerous novel methods for genome assembly. With few exceptions, these algorithms are heuristic and require one or more parameters to be manually set by the user. One approach to parameter tuning involves assembling data from an organism with an available high-quality reference genome, and measuring assembly accuracy using some metrics. We developed a system to measure assembly quality under several scoring metrics, and to compare assembly quality across a variety of assemblers, sequence data types, and parameter choices. When used in conjunction with training data such as a high-quality reference genome and sequence reads from the same organism, our program can be used to manually identify an optimal sequencing and assembly strategy for de novo sequencing of related organisms.
AVAILABILITY: GPL source code and a usage tutorial is at http://ngopt.googlecode.com
CONTACT: aarondarling@ucdavis.edu
SUPPLEMENTARY INFORMATION: Supplementary data is available at Bioinformatics online.<br/>
</details><br/>
---

*Different routes to the same ending: comparing the N-glycosylation processes of Haloferax volcanii and Haloarcula marismortui, two halophilic archaea from the Dead Sea.*<br/>
Calo D; Guan Z; Naparstek S; Eichler J<br/>
**2011**<br/>
Mol. Microbiol. 81(5)<br/>
10.1111/j.1365-2958.2011.07781.x
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21815949)<br/>
[Citation]()<br/>
Recent insight into the N-glycosylation pathway of the haloarchaeon, Haloferax volcanii, is helping to bridge the gap between our limited understanding of the archaeal version of this universal post-translational modification and the better-described eukaryal and bacterial processes. To delineate as yet undefined steps of the Hfx. volcanii N-glycosylation pathway, a comparative approach was taken with the initial characterization of N-glycosylation in Haloarcula marismortui, a second haloarchaeon also originating from the Dead Sea. While both species decorate the reporter glycoprotein, the S-layer glycoprotein, with the same N-linked pentasaccharide and employ dolichol phosphate as lipid glycan carrier, species-specific differences in the two N-glycosylation pathways exist. Specifically, Har. marismortui first assembles the complete pentasaccharide on dolichol phosphate and only then transfers the glycan to the target protein, as in the bacterial N-glycosylation pathway. In contrast, Hfx. volcanii initially transfers the first four pentasaccharide subunits from a common dolichol phosphate carrier to the target protein and only then delivers the final pentasaccharide subunit from a distinct dolichol phosphate to the N-linked tetrasaccharide, reminiscent of what occurs in eukaryal N-glycosylation. This study further indicates the extraordinary diversity of N-glycosylation pathways in Archaea, as compared with the relatively conserved parallel processes in Eukarya and Bacteria.<br/>
</details><br/>
---

*Bioinformatic prediction and experimental verification of sRNAs in the haloarchaeon Haloferax volcanii.*<br/>
Babski J; Tjaden B; Voss B; Jellen-Ritter A; Marchfelder A; Hess WR; Soppa J<br/>
**2011**<br/>
RNA Biol 8(5)<br/>
10.4161/rna.8.5.16039
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21712649)<br/>
[Citation]()<br/>
Recently a small-scale RNomics study led to the experimental identification of 21 intergenic and 18 antisense sRNA genes in the haloarchaeon Haloferax volcanii. To broaden the knowledge about sRNAs in haloarchaea, two bioinformatic approaches were used to predict sRNA genes in the genome of H. volcanii. More than 120 putative intergenic sRNA genes were identified by these comparative genomic approaches. The expression of 61 of the predicted genes was analyzed using DNA microarrays, and 37 were found to be expressed under at least one of three conditions tested. Using the results of Northern blot analyses and of a high throughput sequencing study the number of expressed genes was raised to 54 and the small size was verified for 26 predicted sRNAs. An analysis of the coding capacity revealed that the set of predicted sRNAs most likely does not encode proteins or peptides. In two cases it turned out that the predictions had not identified bona fide sRNAs but conserved regions in UTRs of large protein-encoding transcripts. Taken together, the combination of bioinformatic prediction and experimental verification has more than tripled the number of known haloarchaeal sRNAs, underscoring the importance of regulatory RNAs in the third domain of life, the archaea. Further analyses of the biological functions of selected sRNAs, including the construction of deletion mutants, are currently under way.<br/>
</details><br/>
---

*Comparative survival analysis of Deinococcus radiodurans and the haloarchaea Natrialba magadii and Haloferax volcanii exposed to vacuum ultraviolet irradiation.*<br/>
Abrevaya XC; Paulino-Lima IG; Galante D; Rodrigues F; Mauas PJ; Cortón E; Lage Cde A<br/>
**2011**<br/>
Astrobiology 11(10)<br/>
10.1089/ast.2011.0607
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/22165956)<br/>
[Citation]()<br/>
The haloarchaea Natrialba magadii and Haloferax volcanii, as well as the radiation-resistant bacterium Deinococcus radiodurans, were exposed to vacuum UV (VUV) radiation at the Brazilian Synchrotron Light Laboratory. Cell monolayers (containing 10(5) to 10(6) cells per sample) were prepared over polycarbonate filters and irradiated under high vacuum (10(-5) Pa) with polychromatic synchrotron radiation. N. magadii was remarkably resistant to high vacuum with a survival fraction of (3.77±0.76)×10(-2), which was larger than that of D. radiodurans (1.13±0.23)×10(-2). The survival fraction of the haloarchaea H. volcanii, of (3.60±1.80)×10(-4), was much smaller. Radiation resistance profiles were similar between the haloarchaea and D. radiodurans for fluences up to 150 J m(-2). For fluences larger than 150 J m(-2), there was a significant decrease in the survival of haloarchaea, and in particular H. volcanii did not survive. Survival for D. radiodurans was 1% after exposure to the higher VUV fluence (1350 J m(-2)), while N. magadii had a survival lower than 0.1%. Such survival fractions are discussed regarding the possibility of interplanetary transfer of viable microorganisms and the possible existence of microbial life in extraterrestrial salty environments such as the planet Mars and Jupiter's moon Europa. This is the first work to report survival of haloarchaea under simulated interplanetary conditions.<br/>
</details><br/>
---

*Archaea-based microbial fuel cell operating at high ionic strength conditions.*<br/>
Abrevaya XC; Sacco N; Mauas PJ; Cortón E<br/>
**2011**<br/>
Extremophiles 15(6)<br/>
10.1007/s00792-011-0394-z
<details>
<summary>Abstract and Links</summary>

[Link to Publication](https://ncbi.nlm.nih.gov/pubmed/21894491)<br/>
[Citation]()<br/>
In this work, two archaea microorganisms (Haloferax volcanii and Natrialba magadii) used as biocatalyst at a microbial fuel cell (MFC) anode were evaluated. Both archaea are able to grow at high salt concentrations. By increasing the media conductivity, the internal resistance was diminished, improving the MFC's performance. Without any added redox mediator, maximum power (P                         (max)) and current at P                         (max) were 11.87/4.57/0.12 μW cm(-2) and 49.67/22.03/0.59 μA cm(-2) for H. volcanii, N. magadii and E. coli, respectively. When neutral red was used as the redox mediator, P                         (max) was 50.98 and 5.39 μW cm(-2) for H. volcanii and N. magadii, respectively. In this paper, an archaea MFC is described and compared with other MFC systems; the high salt concentration assayed here, comparable with that used in Pt-catalyzed alkaline hydrogen fuel cells, will open new options when MFC scaling up is the objective necessary for practical applications.<br/>
</details><br/>
---

[Go to top of page](# )<br/>
----
